{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 117,
   "id": "b714c165-fcb2-4b00-aee6-96aa6299933a",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: bs4 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (0.0.1)\n",
      "Requirement already satisfied: beautifulsoup4 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from bs4) (4.10.0)\n",
      "Requirement already satisfied: soupsieve>1.2 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from beautifulsoup4->bs4) (2.3.1)\n",
      "Note: you may need to restart the kernel to use updated packages.\n",
      "Collecting requests-html\n",
      "  Downloading requests_html-0.10.0-py3-none-any.whl (13 kB)\n",
      "Collecting fake-useragent\n",
      "  Downloading fake-useragent-0.1.11.tar.gz (13 kB)\n",
      "Collecting w3lib\n",
      "  Downloading w3lib-1.22.0-py2.py3-none-any.whl (20 kB)\n",
      "Collecting pyquery\n",
      "  Downloading pyquery-1.4.3-py3-none-any.whl (22 kB)\n",
      "Collecting parse\n",
      "  Downloading parse-1.19.0.tar.gz (30 kB)\n",
      "Collecting pyppeteer>=0.0.14\n",
      "  Downloading pyppeteer-0.2.6-py3-none-any.whl (83 kB)\n",
      "\u001b[K     |████████████████████████████████| 83 kB 138 kB/s eta 0:00:01\n",
      "\u001b[?25hRequirement already satisfied: bs4 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from requests-html) (0.0.1)\n",
      "Requirement already satisfied: requests in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from requests-html) (2.26.0)\n",
      "Requirement already satisfied: six>=1.4.1 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from w3lib->requests-html) (1.16.0)\n",
      "Requirement already satisfied: lxml>=2.1 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from pyquery->requests-html) (4.6.3)\n",
      "Collecting cssselect>0.7.9\n",
      "  Downloading cssselect-1.1.0-py2.py3-none-any.whl (16 kB)\n",
      "Collecting pyee<9.0.0,>=8.1.0\n",
      "  Downloading pyee-8.2.2-py2.py3-none-any.whl (12 kB)\n",
      "Collecting tqdm<5.0.0,>=4.42.1\n",
      "  Downloading tqdm-4.62.3-py2.py3-none-any.whl (76 kB)\n",
      "\u001b[K     |████████████████████████████████| 76 kB 240 kB/s eta 0:00:01\n",
      "\u001b[?25hRequirement already satisfied: urllib3<2.0.0,>=1.25.8 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from pyppeteer>=0.0.14->requests-html) (1.26.6)\n",
      "Requirement already satisfied: appdirs<2.0.0,>=1.4.3 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from pyppeteer>=0.0.14->requests-html) (1.4.4)\n",
      "Collecting websockets<10.0,>=9.1\n",
      "  Downloading websockets-9.1-cp37-cp37m-manylinux2010_x86_64.whl (103 kB)\n",
      "\u001b[K     |████████████████████████████████| 103 kB 236 kB/s eta 0:00:01\n",
      "\u001b[?25hRequirement already satisfied: importlib-metadata>=1.4 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from pyppeteer>=0.0.14->requests-html) (4.8.1)\n",
      "Requirement already satisfied: beautifulsoup4 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from bs4->requests-html) (4.10.0)\n",
      "Requirement already satisfied: charset-normalizer~=2.0.0; python_version >= \"3\" in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from requests->requests-html) (2.0.4)\n",
      "Requirement already satisfied: idna<4,>=2.5; python_version >= \"3\" in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from requests->requests-html) (3.2)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from requests->requests-html) (2021.5.30)\n",
      "Requirement already satisfied: typing-extensions>=3.6.4; python_version < \"3.8\" in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from importlib-metadata>=1.4->pyppeteer>=0.0.14->requests-html) (3.10.0.2)\n",
      "Requirement already satisfied: zipp>=0.5 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from importlib-metadata>=1.4->pyppeteer>=0.0.14->requests-html) (3.5.0)\n",
      "Requirement already satisfied: soupsieve>1.2 in /home/ben/miniconda3/envs/PyRosetta.notebooks/lib/python3.7/site-packages (from beautifulsoup4->bs4->requests-html) (2.3.1)\n",
      "Building wheels for collected packages: fake-useragent, parse\n",
      "  Building wheel for fake-useragent (setup.py) ... \u001b[?25ldone\n",
      "\u001b[?25h  Created wheel for fake-useragent: filename=fake_useragent-0.1.11-py3-none-any.whl size=13502 sha256=c2536c9d6359915647af909f9ecf460f38733b7d72263254cd6d3bbac0106daf\n",
      "  Stored in directory: /home/ben/.cache/pip/wheels/ed/f7/62/50ab6c9a0b5567267ab76a9daa9d06315704209b2c5d032031\n",
      "  Building wheel for parse (setup.py) ... \u001b[?25ldone\n",
      "\u001b[?25h  Created wheel for parse: filename=parse-1.19.0-py3-none-any.whl size=24591 sha256=19e89789889bf481dd19bbebd135aab454e839edd11bdd1e96c13b5d10e9d720\n",
      "  Stored in directory: /home/ben/.cache/pip/wheels/9c/aa/cc/f2228050ccb40f22144b073f15a2c84f11204f29fc0dce028e\n",
      "Successfully built fake-useragent parse\n",
      "Installing collected packages: fake-useragent, w3lib, cssselect, pyquery, parse, pyee, tqdm, websockets, pyppeteer, requests-html\n",
      "Successfully installed cssselect-1.1.0 fake-useragent-0.1.11 parse-1.19.0 pyee-8.2.2 pyppeteer-0.2.6 pyquery-1.4.3 requests-html-0.10.0 tqdm-4.62.3 w3lib-1.22.0 websockets-9.1\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "%pip install bs4\n",
    "%pip install requests-html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "id": "e68695d5-adbd-4205-9af7-47a907c6a2ad",
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "from requests_html import AsyncHTMLSession\n",
    "from bs4 import BeautifulSoup"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "b66913ef-75e1-47ac-b442-b01ae429031c",
   "metadata": {},
   "outputs": [],
   "source": [
    "def replace_words(t):\n",
    "    replacers = [\"<p>\", \"<br>\", \"</p>\", \"</br>\", \"<br/>\", \":\", \",\", \".\", r'\\xao', \"\\n\", \"(\", \")\", \"/\", \"-\"]\n",
    "    for i in replacers:\n",
    "        t = t.replace(i, \" \")\n",
    "    return t.strip().split()\n",
    "\n",
    "def search_for_keywords(text):\n",
    "    keywords = {\"comput\":5, \"protein\":1, \"drug\":5, \"design\":1, \"immun\":1, \"algorithm\":1, \"machine\":1, \"deep\":1, \"therap\":1, \"molecular\":0, \"antibod\":1}\n",
    "    counter = 0\n",
    "    \n",
    "    wordlist = replace_words(text)\n",
    "    for word in keywords.keys():\n",
    "        for idx, tw in enumerate(wordlist):\n",
    "            if word in tw or word.capitalize() in tw:\n",
    "                nextword = \"\" \n",
    "                if idx+1 < len(wordlist):\n",
    "                    nextword = wordlist[idx+1].lower()\n",
    "                if word == \"protein\" and (nextword == \"design\" or nextword == \"engineering\"):\n",
    "                    counter += keywords[word]*10\n",
    "                elif word == \"machine\" and nextword == \"learning\":\n",
    "                    counter += keywords[word]*5\n",
    "                elif word == \"deep\" and nextword == \"learning\":\n",
    "                    counter += keywords[word]*5\n",
    "                elif word == \"molecular\" and (nextword == \"engineering\" or nextword == \"immunology\"):\n",
    "                    counter += 20\n",
    "                else:\n",
    "                    counter += keywords[word]\n",
    "    return counter\n",
    "\n",
    "def print_sorted_faculty(faculty_dict, top_twenty=True):\n",
    "    faculty_list = []\n",
    "    for f, text in faculty_dict.items():\n",
    "        num_hits = 0\n",
    "        try:\n",
    "            num_hits = search_for_keywords(text)\n",
    "        except (TypeError, IndexError):\n",
    "            pass\n",
    "        faculty_list.append((f, num_hits))\n",
    "\n",
    "    sorted_fac = sorted(faculty_list, key=lambda x: x[1], reverse=True) \n",
    "    print(f\"Found and scored {len(sorted_fac)} faculty\")\n",
    "    ctr = 0\n",
    "    for i in sorted_fac:\n",
    "        print(i[0][0], i[0][1], i[1])\n",
    "        if ctr == 19 and top_twenty:\n",
    "            break\n",
    "        ctr += 1"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "43efcb3c-29b5-40fe-900b-0a6d37fee2c9",
   "metadata": {},
   "source": [
    "# Stanford Faculty Analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "967f4110-f87b-460f-84e3-ba003cbe47af",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = \"https://med.stanford.edu/biophysics/faculty1.html\"\n",
    "response = requests.get(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "1ff98156-9235-4f79-a1aa-146a22ac9d16",
   "metadata": {},
   "outputs": [],
   "source": [
    "soup = BeautifulSoup(response.text, \"html.parser\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "70613a6d-ac26-4599-a2bf-a12865be9278",
   "metadata": {
    "scrolled": true,
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[<p>The Abu-Remaileh Lab is interested in identifying novel pathways that enable cellular and organismal adaptation to metabolic stress and changes in environmental conditions. We also study how these pathways go awry in human diseases such as cancer, neurodegeneration and metabolic syndrome, in order to engineer new therapeutic modalities.<br/><br/>To address these questions, our lab uses a multidisciplinary approach to study the biochemical functions of the lysosome in vitro and in vivo. Lysosomes are membrane-bound compartments that degrade macromolecules and clear damaged organelles to enable cellular adaptation to various metabolic states. Lysosomal function is critical for organismal homeostasis—mutations in genes encoding lysosomal proteins cause severe human disorders known as lysosomal storage diseases, and lysosome dysfunction is implicated in age-associated diseases including cancer, neurodegeneration and metabolic syndrome.<br/><br/>By developing novel tools and harnessing the power of metabolomics, proteomics and functional genomics, our lab will define 1) how the lysosome communicates with other cellular compartments to fulfill the metabolic demands of the cell under various metabolic states, 2) and how its dysfunction leads to rare and common human diseases. Using insights from our research, we will engineer novel therapies to modulate the pathways that govern human disease.</p>]\n",
      "\n",
      "<p>Our goal is to develop and clinically implement new technologies for high-precision and noninvasive intervention upon the nervous system. Every few millimeters of the brain is functionally distinct, and different parts of the brain may have counteracting responses to therapy. To better match our therapies to neuroscience, we develop techniques that allow intervention upon only the right part of the nervous system at the right time, using technologies like focused ultrasound and nanotechnology.</p>\n",
      "\n",
      "[<p>Russ Biagio Altman is the Kenneth Fong Professor of Bioengineering, Genetics, Medicine, Biomedical Data Science and (by courtesy) Computer Science) and past chairman of the Bioengineering Department at Stanford University. His primary research interests are in the application of computing (AI, data science and informatics) to problems relevant to medicine. He is particularly interested in methods for understanding drug action at molecular, cellular, organism and population levels. His lab studies how human genetic variation impacts drug response (e.g., http://www.pharmgkb.org/). Other work focuses on the analysis of biological molecules to understand the actions, interactions and adverse events of drugs (e.g., http://feature.stanford.edu/). He helps lead an FDA-supported Center of Excellence in Regulatory Science &amp; Innovation.<br/><br/>Dr. Altman holds an AB from Harvard College, and an MD from Stanford Medical School, and a PhD in Medical Information Sciences from Stanford. He received the U.S. Presidential Early Career Award for Scientists and Engineers and a National Science Foundation CAREER Award. He is a fellow of the American College of Physicians (ACP), the American College of Medical Informatics (ACMI), the American Institute of Medical and Biological Engineering (AIMBE), and the American Association for the Advancement of Science (AAAS). He is a member of the National Academy of Medicine (formerly the Institute of Medicine, IOM). He is a past-president, founding board member, and a fellow of the International Society for Computational Biology (ISCB), and a past-president of the American Society for Clinical Pharmacology &amp; Therapeutics (ASCPT). He has chaired the Science Board advising the FDA commissioner, and has served on the NIH Director’s Advisory Committee, and as cochair of the IOM Drug Forum. He is an organizer of the annual Pacific Symposium on Biocomputing, and a founder of Personalis (NASDAQ: PSNL). Dr. Altman is board certified in Internal Medicine and in Clinical Informatics. He received the Stanford Medical School graduate teaching award in 2000 and mentorship award in 2014.  He is the founding editor of the Annual Reviews of Biomedical Data Science, and hosts a SiriusXM radioshow (and podcast) entitled “The Future of Everything.”</p>]<p>I am interested in the application of computational technologies to problems in molecular biology of relevance to medicine. In particular, my laboratory focuses on drug response at the molecular level, working in three areas.  First, we are building a comprehensive pharmacogenomics knowledge base (http://www.pharmgkb.org/) that provides access to information relating genotype to phenotype (in particular, how variation in genetics leads to variation in response to drugs). We are interested in collaboratively discovering and applying new pharmacogenomics knowledge.  Second, we are interested in the analysis of three dimensional biological structures.  We have methods for analyzing protein structures to recognize and annotate active sites and binding sites, particularly in the context of interactions with small molecule drugs.   We are also interested in physics-based simulation of biological structures to understand how their dynamics impact their function (http://simbios.stanford.edu/).   Finally, we are interested in computational methods for analyzing functional genomics information.  We use natural language processing techniques for extracting and summarizing information in the literature, chemoinformatics methods for understanding small molecule function, and machine learning &amp;amp; data mining techniques to understand the molecular responses to drugs.</p>\n",
      "\n",
      "<p>My laboratory studies how bacteria colonize our bodies for long periods of time, and how interactions between bacteria and the epithelial surfaces of the gastrointestinal tract and skin may lead to disease. Epithelial surfaces are the first barrier against infection, but they also where our bodies meet and co-evolve with the microbial world.. Several of our studies have focused on the epithelial junctions as a target for bacterial pathogens. The host epithelium uses its epithelial junctions to form a tight but dynamic barrier with an external surface that is inhospitable to microbial attachment, secretes anti-microbial compounds, and has a rapid rate of self-renewal. The balance in the microbe-epithelial relationship results in silent commensalism or symbiosis; an imbalance results in diseases ranging from acute bacterial invasive disease to chronic ulcers or carcinoma.&lt;br/&gt;&lt;br/&gt;Our laboratory has developed novel microscopy applications such as quantitative 3D confocal microscopy, electron microscopy, time-lapse imaging, microinjection and micromanipulation to visualize the interaction of pathogens with epithelial cells in culture and in animal and human tissues. Many of out studies focus on the gastric pathogen Helicobacter pylori, but we have also expanded our investigations to include the intestinal pathogens Listeria monocytogenes and Salmonella enterica, and the skin pathogen and colonizer Staphylococcus aureus. I believe that elucidating how microbes communicate with and alter our epithelial cells at a molecular level will be important for finding novel therapeutic targets to control mucosal colonization and prevent invasive disease. &lt;br/&gt;&lt;br/&gt;Using this perspective, we have uncovered several novel concepts of how bacteria colonize and breach our epithelial surfaces. For example, we discovered that Helicobacter pylori target the intercellular junctions, and in particular that the virulence factor CagA affects junction assembly and cell polarity. This confers H. pylori the ability to extract nutrients and grow directly on the epithelial surface. We also found that these properties of CagA have consequences for cellular transformation of the epithelium. For instance, we showed that H. pylori affect the activity and state of epithelial stem cells in the stomach by colonizing the epithelial surface deep in the gastric glands. This gland-associated population is essential for pathological inflammation and hyperplasia in animal models, and confers significant colonization advantages to the bacteria. Our Listeria research uncovered a new mechanism and site where bacteria can breach the gastrointestinal epithelial barrier to invade. We found that Listeria find their receptor for invasion at sites of epithelial senescence, where the epithelial junctions undergo dynamic turnover. To study Salmonella and H. pylori we have developed a human organoid model to study their interactions with human gut epithelium in vitro. To study Staphylococcus aureus pathogenesis, we have developed methods to visualize infection at the scale of a single bacterial microcolony using an organoid culture system of human keratinocytes and fibroblasts that grow into a 3D skin-equivalent. We recently identified several proteins at the eptithelial junctions as host factors involved in the pathogenesis of one of Staphylococcus aureus major toxins.</p>\n",
      "\n",
      "\n",
      "\n",
      "[<p>Annelise E. Barron is the W.M. Keck Associate Professor of Bioengineering at Stanford University. <br/><br/>The broad theme of the Barron lab is the study and biomimicry of natural host defense peptides (antimicrobial peptides). We study the molecular biophysics and mechanisms of LL-37—a centrally important human host defense peptide—and its involvement in Alzheimer's dementia (via LL-37 dysregulation and degradation by pathogen virulence factors). Alzheimer's dementia can be caused by (or at least, accompanied by) cerebral infections, a phenomenon now receiving renewed attention given recent discoveries. We are also working to develop biostable peptoid mimics of LL-37 as therapeutics that can combat antibiotic-resistant infections. Finally, we are working to mimic lung surfactant proteins that facilitate the delivery of therapeutics to the lungs, treat bacterial and viral pneumonia, or prevent or treat ventilator-associated acute lung injury. <br/><br/>We are currently putting efforts into better understanding the pathogenic mechanisms of Covid-19, as relates to dysregulation of innate immunity; understanding why certain minority populations seem to be more strongly affected by Covid-19 infections; and developing therapeutic approaches to both preventing and treating severe Covid-19.<br/><br/>Dr. Barron was trained as a chemical engineer at the University of Washington (B.S.) and U.C. Berkeley (Ph.D., under the mentorship of Prof. Harvey W. Blanch), and was a Pharmaceutical Chemistry postdoc with Prof. Ken A. Dill (UCSF) and Dr. Ronald N. Zuckermann (Chiron Corp.). She has served on the faculty at Stanford since 2007, and prior to that, served on the Chemical &amp; Biological Engineering faculty of Northwestern University in Evanston, IL for 10 years (1997-2007). Dr. Barron has been awarded the NIH Pioneer Award (2020), the Presidential Early Career Award for Scientists &amp; Engineers (PECASE) through NIH / NHGRI (1999), the Beckman Young Investigator Award (1999), and the Camille Dreyfus Teacher-Scholar Award (1998), among other awards. Dr. Barron was the youngest scientist ever to serve on the Scientific Advisory Committee to the Director of the NIH, under Dr. Elias Zerhouni. She has more than 172 publications and a current H-index of 45 (Web of Science, All Databases), and serves on the advisory boards of several biotechnology companies. She is proud to be 1/4 Quechua (the Native American people of Bolivia), 1/4 Hispanic, 1/4 Swedish, 1/4 English, and 100% American.</p>]<p>According to our recent findings, innate immune responses in humans and other mammals involving infection-, injury-, or stress-related dynamic imbalances between particular, immunomodulatory host defense peptides we study, and pro-amyloid / fibrillogenic peptides including ABeta and IAPP, may play a role in the poorly understood etiology of chronic / progressive plaque diseases, including diabetes type II melittus, atherosclerosis and Alzheimers dementia. All of these diseases involve senescent/dystrophic cells, inflammation, and proteopathies with plaque accumulation; and can be complicated by infection.&lt;br/&gt;&lt;br/&gt;The latter disease, Alzheimers, is in need of a major breakthrough in fundamental understanding, more than almost any human disease currently under study. Of a total of 420+ clinical trials initiated by Pharma towards the development of Alzheimer's treatments over the past 15+ years, all of these trials have failed. There is no current effective treatment. Obviously, the most fundamental ideas for what drives Alzheimers must be flawed or incomplete.&lt;br/&gt;&lt;br/&gt;Until recently Alzheimers disease was believed to be the sixth leading cause of death in the United States, according to the Centers for Disease Control and Prevention (CDC). But in March 2014, new research published in Neurology suggested that Alzheimers may actually be responsible for as many deaths each year as heart disease or cancer – the two leading causes of death in the U.S. – due to issues, in hospitals, of improper prior determinations of underlying cause of death in the elderly.&lt;br/&gt;&lt;br/&gt;My lab is testing novel mechanistic hypotheses of Alzheimers etiology, based on recent, unique molecular biophysical observations of pro-amyloid and innate immune peptides. We are also looking at linkages to certain chronic infections.&lt;br/&gt;&lt;br/&gt;Increasing numbers of epidemiological and co-morbidity studies indicate that multiple, progressive degenerative diseases, all involving plaque deposition in various body compartments, are linked. For instance, some researchers have begun to refer to Alzheimers Disease as &amp;quot;Diabetes Melittus Type III&amp;quot;. We seek, with current projects, to sleuth out the shared molecular biophysical bases for these emerging linkages. &lt;br/&gt;&lt;br/&gt;(Note: Succinct, exemplary summaries of these fascinating epidemiological / comorbidity linkages are found, for instance, in the following papers: &amp;quot;The ‘psoriatic march’: a concept of how severe psoriasis may drive cardiovascular comorbidity&amp;quot;, Experimental Dermatology (2011) 20, 303–307; &amp;quot;Circle of Willis atherosclerosis: association with Alzheimer’s disease, neuritic plaques and neurofibrillary tangles&amp;quot;, Acta Neuropathol (2007) 113:13–21; &amp;quot;Increased prevalence of the metabolic syndrome in patients with moderate to severe psoriasis&amp;quot;, Arch Dermatol Res (2006) 298: 321–328; &amp;quot;Association of Alzheimer disease pathology with abnormal lipid metabolism&amp;quot;, Neurology (2011) 77;1068).</p>\n",
      "\n",
      "[<p>Lacra Bintu is an Assistant Professor in the Bioengineering Department at Stanford. Her lab performs single-cell and high-throughput measurements of chromatin and gene regulation dynamics, and uses these data to develop predictive models and improve mammalian cell engineering. <br/><br/>Lacra started working on the theory of gene regulation as an undergraduate with Jané Kondev from Brandeis University and Rob Phillips from Caltech. As a Physics PhD student in the lab of Carlos Bustamante at U.C. Berkeley, she used single-molecule methods to tease apart the molecular mechanisms of transcription through nucleosomes. She transitioned to studying the dynamics of epigenetic regulation in live cells during her postdoctoral fellowship with Michael Elowitz at Caltech.</p>]\n",
      "\n",
      "[<p>S.M. Block holds the Ascherman Chair in the Depts. of Applied Physics and Biology at Stanford. He’s best-known as a founder of the field known as “single molecule biophysics.” Block holds degrees from Oxford and Caltech, and served as faculty at the Rowland Institute and Harvard, then Princeton, prior to joining Stanford in 1999. Block is a member of the National Academy of Sciences, the American Academy of Arts &amp; Sciences, and is a Fellow of the AAAS, the APS, and the BPS. His research lies at the interface of physics and biology, particularly in the study of biomolecular motors, including kinesin and RNA polymerase, and the folding of nucleic acid-based structures. His group pioneered the use of laser-based optical traps, or ‘optical tweezers,’ to study the nanoscale motions of biomolecules. In what’s left of his spare time, he enjoys skiing and playing bluegrass music on the banjo and mandolin.</p>]<p>Single molecule biophysics using optical trapping and fluorescence</p>\n",
      "\n",
      "[<p>Alistair Boettiger is an Assistant Professor in the Department of Developmental Biology at the Stanford University School of Medicine and is a member of Bio-X and the Biophysics Program. As an undergraduate he studied Physics and Molecular Biology at Princeton University, where he had the opportunity to experience lab research under the mentorship of Stas Shvartsman, who introduced him to his abiding interest in quantitative imaging and animal development. He conducted his Ph.D. research under the guidance of Michael Levine at UC Berkeley, where he studied cis-regulatory sequences that modulate the precision and robustness of gene expression (in particular shadow enhancers and paused promoters). As a postdoc in Xiaowei Zhuang's single-molecule imaging group at Harvard University, he studied gene regulation through super-resolution microscopy, multiplexed, error-correcting imaging, and deep sequencing. Dr. Boettiger started his lab at Stanford in 2016.</p>]<p>My research aims to improve our understanding of how cells with the same genome can develop dramatically different behaviors.  For example, consider the mechanical abilities of a muscle cell compared to the electrical excitability of a neuron, or the industrious activity of bone building cells in a youthful person compared to an elderly one.  Each of these cells, (if taken from the same individual) has an identical genome — and yet each is “reading” a very distinct subset of that genome and consequently carrying out very different behaviors.  The choice of what to read and what to hide away is made during development.   An increasing body of data suggests this is accomplished by modifying the genome both in the nature of the proteins bound to different sequences and in the spatial organization of those sequences relative to each other.  The spatial organization or folding of the genome may be particularly important in complex multicellular organisms, since many of the sequences known to interact based on genetic data are nonetheless substantially separated from each other along the linear genome.  By regulating the folding of this linear sequence into a higher order structure, a cell might change which regulatory sequences have access to which genes, and achieve different behavioral states.&lt;br/&gt;&lt;br/&gt;So far we have little imaging data on how the genome is folded within a cell on the length scale of individual genes, or whether this folding is regulated in any way relevant to the behavior of the cell.   Our limited knowledge stems largely from want of a method that has both the resolution and specificity to visualize such genomic substructure.  Conventional fluorescent microscopy has developed excellent tools for coloring specific regions of DNA and particular DNA-associated proteins with uniquely colored dyes — but lacks the resolution to turn these colored blurs into structures.  Electron-microscopy has substantially greater resolution but lacks compatibility with specific labeling techniques to tell different gene clusters or different protein types apart.  Super-resolution imaging approaches promise to address this balance by allowing the use of fluorescent labels while simultaneously resolving structures on the nano-scale.  I have been adapting this approach to uncover the nano-scale structure of chromatin and determine to how this structure changes when bound by different types of nuclear proteins.  While individual gene clusters appear as quite diverse structures, there appear to be a few general features, for example, linking structure with the epigenetic state.</p>\n",
      "\n",
      "[<p>My laboratory investigates the structure and function of biological systems with a strong physical perspective. We invent experimental methods and develop theory as needed. We are pursuing several interconnected themes:<br/><br/>Excited State Dynamics in GFP &amp; Split GFP <br/><br/>Green Fluorescent Protein (GFP) is widely used as a probe to localize proteins in cells. Our lab was the first to demonstrate that the GFP chromophore exists in two protonation states, interconvertible by ultrafast excited state proton transfer. We and others have since developed this idea to generate novel GFP variants with diverse colors and sensitivities. Current work focuses on split GFP in which structural elements such as entire beta strands are replaced with synthetic ones. We have discovered that some split GFPs can be photodissociated, generating a peptide and a truncated protein; alternatively, in some conditions, light can be used to associate peptides with truncated proteins. Thus these split GFPs are optogenetic elements for manipulations inside cells, and we seek to understand how they work.<br/><br/>Electrostatics and Dynamics in Proteins<br/><br/>We study electrostatics in proteins and how electric fields affect function. Early test systems used mutants of myoglobin, which was first cloned and expressed in our lab. This led to probes whose sensitivity to electric fields can be calibrated by Stark spectroscopy — spectroscopy in electric fields — which we have developed into a broadly applicable method. Vibrational Stark experiments exploit molecular vibrations as local and directional probes to map electrostatic fields in proteins. Current work applies nitrile probes introduced into proteins on inhibitors. These can be used to probe electrostatics and hydration at the active sites of important drug targets. Recent work focuses on carbonyl probes to study enzymatic reactions. By combining the vibrational Stark effect, vibrational solvatochromism and MD simulations, we have developed a general method to measure the absolute field sensed by the carbonyl probe in proteins. This has been used to quantify the electrostatic contribution to the catalytic rate in several enzymes.<br/><br/>Model Membranes<br/><br/>Our group has developed supported lipid bilayers as mimics for cell surfaces and tools in biotechnology. A broad vision is to engineer interfaces between hard surfaces and soft materials, ultimately leading to sophisticated biocompatible interfaces that can be used to control, interrogate or organize complex living systems. We have developed methods to partition and manipulate elements of these unique self-assembled systems; these methods are now used in many laboratories.<br/><br/>Recent work addresses four interrelated areas: 1) characterization of membrane organization, domains and protein associations using a novel type of imaging mass spectrometry; 2) models for membrane fusion and investigations into the fusion of enveloped viruses to their target membrane; 3) development of tethered lipid bilayers as a platform to study membrane domains, junction topology, vesicle fusion and enveloped virus fusion; and 4) a membrane interferometer where a free-standing lipid bilayer is held within a few hundred nm of an atomically flat mirror, with the ultimate goal of measuring protein conformational changes optically with sub-nm precision in parallel with electrical measurements, e.g. in ion channels.<br/><br/>Energy and Electron Transfer in Photosynthesis<br/><br/>Light-driven long-distance electron transfer in photosynthetic reaction centers is one of the fastest known chemical reactions. We study this by femtosecond fluorescence and transient absorption spectroscopy, manipulation in electric fields, site-specific and global mutagenesis and some novel types of Stark spectroscopy. Current work probes alternate pathways of electron transfer in novel bacterial reaction centers that lack normal electron acceptors, and introduces non-canonical amino acids to perturb and probe pathways.</p>]<p>Please visit my website for complete information:&lt;br/&gt;http://www.stanford.edu/group/boxer/</p>\n",
      "\n",
      "<p>The Brandman Lab studies how cells sense and respond to stress. We employ an integrated set of techniques including single cell analysis, mathematical modeling, genomics, structural studies, and in vitro assays.</p>\n",
      "\n",
      "[<p>Dr. Brunet is interested in the molecular mechanisms of aging and longevity, with a particular emphasis on the nervous system. Her lab is interested in identifying pathways involved in delaying aging in response to external stimuli such as availability of nutrients and mates. She also seeks to understand the mechanisms that influence the rejuvenation of old stem cells. Finally, her lab has pioneered the naturally short-lived African killifish as a new model to explore the regulation of aging and age-related diseases.</p>]<p>The overarching goal of our lab is to understand the genetic mechanisms of aging and longevity. Aging is a highly plastic process regulated by a combination of genetic and environmental factors. &lt;br/&gt;&lt;br/&gt;We have a long-standing interest in the genetic pathway that connects insulin to FOXO transcription factors, a central pathway to regulate lifespan from worms to humans. We use a combination of genetic, molecular, and cellular approaches to analyze the regulation and importance of FOXO transcription factors, and more generally 'longevity genes' in mammals. We are particularly interested in the role of longevity genes in the maintenance of the pool of adult neural stem cells and intact cognitive function during aging. We also use ultra-high throughput sequencing technologies to study epigenetic changes and transcriptional networks during aging. &lt;br/&gt;&lt;br/&gt;In parallel, our goal is to identify novel  longevity genes  using short-lived animal models. Our lab uses unbiased approaches in the nematode C. elegans to identify novel pathways that control organismal longevity, particularly in response to dietary restriction. We are particularly interested in the role of chromatin modifiers in the regulation of lifespan and metabolism. &lt;br/&gt;&lt;br/&gt;Finally, we are developing the extremely short-lived African killifish N. furzeri as a new vertebrate model for aging studies. We are taking advantage of this fish to explore the genetic architecture of longevity in vertebrates.</p>\n",
      "\n",
      "[<p>Axel Brunger received his Physics Diploma at the University of Hamburg in 1980, and his Ph.D. degree from the Technical University of Munich in 1982 working with Klaus Schulten. He held a NATO postdoctoral fellowship and subsequently became a research associate with Martin Karplus at the Department of Chemistry, Harvard University. In 1987 he joined the faculty in the Department of Molecular Biophysics and Biochemistry at Yale University. In 2000, he moved to Stanford University where he is Professor and Chair of the Department of Molecular and Cellular Physiology. He also holds an appointment as Investigator in the Howard Hughes Medical Institute. In 1995 he was awarded the Röntgen Prize for Biosciences from the University of Würzburg. In 2003, he received the Gregori Aminoff Award of the Royal Swedish Academy.  In 2005 he was elected member of the National Academy of Sciences. In 2011 he received the DeLano Award of the American Society for Biochemistry and Molecular Biology, in 2014 he received the Bernard Katz Award of the Biophysical Society, and the Carl Hermann Medal of the German Crystallographic Society, and in 2016 he received the Trueblood Award of the American Crystallographic Association. In 2021 he was elected member of the American Academy of Arts &amp; Sciences.<br/><br/>Early in his career, Brunger began developing tools for interpreting x-ray crystallography diffraction data. Scientists use x-ray crystallography to determine molecular structures by crystallizing the molecules and then bombarding them with x-rays. From the data produced by the diffracted x-rays, scientists can calculate a three-dimensional model of the molecule. Brunger’s powerful computational methodology revolutionized structural calculation, accelerating its automation and making protein crystallography accessible to non-experts.<br/><br/>Brunger also developed a major computational tool called the “free R value,” to rate a molecular model’s quality and how likely it is to be correct. The free R value has since become a standard criterion for judging agreement between a crystallographic model and its experimental x-ray diffraction data.<br/><br/>Since the mid-1990s, Brunger has applied his expertise in structural biology to study the molecular mechanisms of synaptic proteins that enable nerve cell communication. <br/><br/>In 1998, Brunger and his team showed that the corkscrew-shaped SNARE proteins assemble into quartets of one syntaxin-1, one synaptobrevin, and two SNAP-25 helices. The proteins all lie in parallel, with their heads pointing in the same direction, to promote membrane fusion. <br/><br/>Since moving to Stanford University in 2000, Brunger and his collaborators have developed a reconstituted system that enables them to study synaptic fusion at greater level of detail than possible in neurons. The team studied the molecular mechanism of neuronal SNAREs, complexin, and synaptotagmin, as well as other factors involved in priming and pre-synaptic plasticity. <br/><br/>In 2015, Brunger’s team used electron cryo-microscopy to determine the structure of the supercomplex of SNAREs, the ATPase NSF, and the adapter protein α-SNAP. This subnanometer-resolution structure, along with functional studies, revealed first glimpses of the molecular mechanism of NSF-mediated SNARE complex disassembly, which allows SNARE to be recycled for the next round of synaptic vesicle fusion. <br/><br/>In 2017, the team determined atomic-resolution structures of the complex of the calcium sensor synaptotagmin-1, the regulator complexin, and the SNARE complex, revealing two essential interfaces that are essential for fast synchronous release of neurotransmitters. . These structures suggest an unlocking mechanism that is triggered by Ca2+binding to the synaptotagmin molecules, leading to SNARE complex zippering, and membrane fusion.</p>]<p>Nerve cells communicate by releasing the contents of neurotransmitter-bearing synaptic vesicles into the space between adjoining cells. This process depends on a handful of proteins that promote vesicle and nerve cell membrane fusion. The Brunger lab team uses structural and biophysical tools to capture this machinery at different stages of vesicle fusion. These structures (Figure 1) then provide the framework for further investigations, using microscopy and live neurons, into the functional and dynamic aspects of the system.&lt;br/&gt;&lt;br/&gt;SNARE proteins, found in both nerve cell and vesicle membranes, set the stage for fusion by zipping together into a parallel, four-helix bundle that juxtaposes the two membranes. Brunger and his collaborators determined the first x-ray crystal structure of the neuronal SNARE complex, as well as the structures of other key components of the synaptic release machinery. Recently, the Brunger’s team visualized the SNARE complex bound to the Ca2+-sensor synaptotagmin-1 and to the regulator complexin, revealing two interfaces that are essential for fast synchronous release of neurotransmitters. The structure of this three-part complex suggests that it is in a primed and locked state. Action-potential-driven Ca2+ ions bind to the synaptotagmin proteins, unlock the complex, and trigger membrane fusion on a sub-millisecond timescale. &lt;br/&gt;&lt;br/&gt;After fusion has occurred, SNARE complexes are recycled by the ATPase NSF, which breaks down the SNARE complex into its individual components. The Brunger team visualized this molecular machine at near-atomic level and obtained the first glimpses of how this SNARE-recycling machine works. The SNARE complex resembles a rope with a left-handed twist, and NSF uses adapter proteins called SNAPs to grasp the “rope” in multiple places. The SNAPs wrap around the SNARE complex with a right-handed twist, suggesting that the disassembly occurs via a simple unwinding motion that frees the zipped SNARE proteins.&lt;br/&gt;&lt;br/&gt;The Brunger team is also using structural and functional studies to explore other machinery relevant to neurotransmitter release, such as factors involved in priming and pre-synaptic plasticity. Their research may one day provide new possibilities for targeting therapeutics to control neurotransmitter release.</p>\n",
      "\n",
      "[<p>Molecular motors lie at the heart of biological processes from DNA replication to vesicle transport. My laboratory seeks to understand the physical mechanisms by which these nanoscale machines convert chemical energy into mechanical work. We use single molecule tracking and manipulation techniques to observe and perturb substeps in the mechanochemical cycles of individual motors. Protein engineering helps us to explore relationships between molecular structures and mechanical functions. Broad topics of current interest include torque generation by DNA-associated ATPases and mechanical adaptations of unconventional myosins.</p>]<p>Molecular motors lie at the heart of biological processes from DNA replication to vesicle transport. My laboratory seeks to understand the physical mechanisms by which these nanoscale machines convert chemical energy into mechanical work. We use single molecule tracking and manipulation techniques to observe and perturb substeps in the mechanochemical cycles of individual motors. Protein engineering helps us to explore relationships between molecular structures and mechanical functions. Broad topics of current interest include torque generation by DNA-associated ATPases and mechanical adaptations of unconventional myosins.</p>\n",
      "\n",
      "[<p>David B. Camarillo is Associate Professor of Bioengineering, (by courtesy) Mechanical Engineering and Neurosurgery at Stanford University. Dr. Camarillo holds a B.S.E in Mechanical and Aerospace Engineering from Princeton University, a Ph.D. in Mechanical Engineering from Stanford University and completed postdoctoral fellowships in Biophysics at the UCSF and Biodesign Innovation at Stanford. Dr. Camarillo worked in the surgical robotics industry at Intuitive Surgical and Hansen Medical, before launching his laboratory at Stanford in 2012. His current research focuses on precision human measurement for multiple clinical and physiological areas including the brain, heart, lungs, and reproductive system. Dr. Camarillo has been awarded the Hellman Fellowship, the Office of Naval Research Young Investigator Program award, among other honors including multiple best paper awards in brain injury and robotic surgery. His research has been funded by the NIH, NSF, DoD, as well as corporations and private philanthropy. His lab’s research has been featured on NPR, the New York Times, The Washington Post, Science News, ESPN, and TED.com as well as other media outlets aimed at education of the public.</p>]\n",
      "\n",
      "[<p>Associate Professor Lynette Cegelski's research is inspired by the challenge and importance of elucidating chemical structure and function in biological systems and the need for new and unconventional approaches to solve outstanding problems in biology and medicine. Lynette completed her undergraduate studies in Chemistry at SUNY-Binghamton, New York (B.S. summa cum laude and Phi Beta Kappa 1998), where she participated in research to determine the microtubule-bound conformation of the anti-cancer drug Taxol by REDOR solid-state NMR. This formative experience motivated her move to Washington University to conduct her PhD training in the laboratory of Professor Jacob Schaefer, where she trained as a solid-state NMR spectroscopist (Ph.D. Chemistry 2004). She investigated bacterial and plant cell-wall and whole-cell systems. She examined the mode of action of vancomycin and oritavancin and also examined photosynthesis and photorespiration in intact leaf NMR experiments. She trained in Microbiology and Infectious Disease research as a postdoctoral fellow in Molecular Microbiology at the Washington University School of Medicine, working with Professor Scott Hultgren. There, she introduced the first small-molecule inhibitors of functional amyloid assembly in bacteria. She joined the faculty of the Stanford Chemistry Department in 2008. The Cegelski Research program integrates chemistry, biology, and physics to investigate the assembly and function of macromolecular and whole-cell systems. They are revealing new bacterial structures, uncovering fundamental parameters of chemical composition and architecture in complex biofilm assemblies, and identifying new anti-infective strategies. Cegelski's work has garnered early career awards, including the Burroughs Wellcome Career Award at the Scientific Interface, the 2010 NIH Director’s New Innovator Award, the National Science Foundation CAREER Award, and the Presidential Early Career Award for Scientists and Engineers (PECASE).</p>]<p>Our research program integrates chemistry, biology, and physics to investigate the assembly and function of macromolecular and whole-cell systems. The genomics and proteomics revolutions have been enormously successful in generating crucial &amp;quot;parts lists&amp;quot; for biological systems. Yet, for many fascinating systems, formidable challenges exist in building complete descriptions of how the parts function and assemble into macromolecular complexes and whole-cell factories. We are inspired by the need for new and unconventional approaches to solve these outstanding problems and to drive the discovery of new therapeutics for human disease.&lt;br/&gt;&lt;br/&gt;Our approach is different from the more conventional protein-structure determinations of structural biology. We employ biophysical and biochemical tools, and are designing new strategies using solid-state NMR spectroscopy to examine assemblies such as amyloid fibers, bacterial cell walls, whole cells, and biofilms. We would like to understand at a molecular and atomic level how bacteria self-assemble extracellular structures, including functional amyloid fibers termed curli, and how bacteria use such building blocks to construct organized biofilm architectures. We also employ a chemical genetics approach to recruit small molecules as tools to interrupt and interrogate the temporal and spatial events during assembly processes and to develop new strategies to prevent and treat infectious diseases. Overall, our approach is multi-pronged and provides training opportunities for students interested in research at the chemistry-biology interface.</p>\n",
      "\n",
      "[<p>Our group's research is focused at the intersection of mechanics and biology. We are interested in elucidating the underlying molecular mechanisms that give rise to the complex mechanical properties of cells, extracellular matrices, and tissues . Conversely, we are investigating how complex mechanical cues influence important biological processes such as cell division, differentiation, or cancer progression. Our approaches involve using force measurement instrumentation, such as atomic force microscopy, to exert and measure forces on materials and cells at the nanoscale, and the development of material systems for 3D cell culture that allow precise and independent manipulation of mechanical properties.</p>]\n",
      "\n",
      "<p>Research in my laboratory is aimed at understanding how eukaryotes replicate their DNA despite numerous challenges (collectively known as replication stress), and more generally – how eukaryotic cells safeguard genome integrity. Specifically, we are investigating: (i) mechanisms that regulate the activity of the replicative helicase during replication stress, (ii) mechanisms that control the inheritance of epigenetic information during replication, and (iii) mechanisms of ubiquitin-mediated regulation of genome maintenance. We utilize single-molecule microscopy to directly image fluorescently-labeled replication factors and track them in real time in Xenopus egg extracts. I developed this system as a postdoctoral fellow, and used it to monitor how the eukaryotic replicative helicase copes with DNA damage. We plan to further extend the capabilities of this platform to directly visualize other essential replication factors, nucleosomes, and regulatory post-translational modifications like ubiquitin chains. By elucidating molecular mechanisms responsible for maintaining genome stability, we aim to better understand the link between genome instability and cancer, and how these mechanisms can be harnessed to improve disease treatment.</p>\n",
      "\n",
      "[<p>Wah Chiu received his BA in Physics and PhD in Biophysics from the University of California, Berkeley. He is the Wallenberg-Bienenstock Professor and a professor in the Department of Bioengineering, Department of Microbiology and Immunology and the SLAC National Accelerator Laboratory at Stanford University. He is a pioneer in methodology development for electron cryo-microscopy (cryo-EM). His work has made multiple transformational contributions in developing single particle cryo-EM as a tool for the structural determination of molecular machines at atomic resolution.<br/><br/>For three decades, Dr. Chiu directs a NIH funded 3DEM Resource Center. He has solved many cryo-EM structures including viruses, chaperonins, membrane proteins, ion channels, antigen-antibody complexes, protein-RNA complexes and RNA in collaboration with many scientists around the world. His lab continues to establish high standard testing and characterization protocols for cryo-EM instrumentation and to develop new image processing and modeling algorithms for cryo-EM structure determination. <br/><br/>Dr. Chiu’s research, collaboration and training efforts have been recognized by his elected membership to the Academia Sinica, Taiwan (2008), and the United States National Academy of Sciences (2012), in addition to several honors including the Distinguished Science Award from the Microscopy Society of America (2014) and the Honorary Doctorate of Philosophy from the University of Helsinki, Finland (2014).</p>]<p>My research includes methodology improvements in single particle cryo-EM for atomic resolution structure determination of molecules and molecular machines, as well as in cryo-ET of cells and organelles towards subnanometer resolutions. We collaborate with many researchers around the country and outside the USA on understanding biological processes such as protein folding, virus assembly and disassembly, pathogen-host interactions, signal transduction, and transport across cytosol and membranes.</p>\n",
      "\n",
      "<p>After shuttering the wet lab, we have focused on: a point-of-care device to measure blood ammonia and prevent brain damage; a human protein complex that juxtaposes and joins DNA ends for repair and V(D)J recombination; and strategies for teaching students and for reducing selection bias in educational programs.&lt;br/&gt;&lt;br/&gt;To understand how cells repair DNA damaged by ultraviolet radiation (UV), we identified UV-damaged DNA binding activity, which enhances nucleotide excision repair and is mutated in xeroderma pigmentosum group E.&lt;br/&gt;&lt;br/&gt;To understand how cells repair DNA damaged by ionizing radiation, we identified and studied the proteins required for non-homologous end joining, which repairs double-strand breaks produced by ionizing radiation and V(D)J recombination. &lt;br/&gt;&lt;br/&gt;To interpret the massive data-sets in gene expression profiles, we invented the algorithms SAM (Significance Analysis of Microarrays) and PAM (Prediction Analysis of Microarrays). &lt;br/&gt;&lt;br/&gt;In an interdisciplinary collaboration, we invented a point-of-care device to measure blood ammonia for diagnosis and management of patients with inborn errors of metabolism, liver disease, and cancer-associated chemobrain.</p>\n",
      "\n",
      "[<p>Steven Chu is the William R. Kenan, Jr., Professor of Physics and Professor of Molecular &amp; Cellular Physiology in the Medical School at Stanford University. He has published over 280 papers in atomic and polymer physics, biophysics, biology, bio-imaging, batteries, and other energy technologies.  He holds 15 patents, and an additional 9 patent disclosures or filings since 2015.<br/> <br/>Dr. Chu was the 12th U.S. Secretary of Energy from January 2009 until the end of April 2013. As the first scientist to hold a Cabinet position and the longest serving Energy Secretary, he recruited outstanding scientists and engineers into the Department of Energy. He began several initiatives including ARPA-E (Advanced Research Projects Agency – Energy), the Energy Innovation Hubs, and was personally tasked by President Obama to assist BP in stopping the Deepwater Horizon oil leak.<br/> <br/>Prior to his cabinet post, he was director of the Lawrence Berkeley National Laboratory, where he was active in pursuit of alternative and renewable energy technologies, and Professor of Physics and Applied Physics at Stanford University, where he helped launch Bio-X, a multi-disciplinary institute combining the physical and biological sciences with medicine and engineering. Previously he was head of the Quantum Electronics Research Department at AT&amp;T Bell Laboratories.<br/> <br/>Dr. Chu is the co-recipient of the 1997 Nobel Prize in Physics for his contributions to laser cooling and atom trapping, and has received numerous other awards. He is a member of the National Academy of Sciences, the American Philosophical Society, the American Academy of Arts and Sciences, the Academia Sinica, and is a foreign member of the Royal Society, the Royal Academy of Engineering, the Chinese Academy of Sciences, the Korean Academy of Sciences and Technology and the National Academy of Sciences, Belarus. He is the President Elect of the American Association for the Advancement of Science. He received an A.B. degree in mathematics and a B.S. degree in physics from the University of Rochester, and a Ph.D. in physics from the University of California, Berkeley, as well as 32 honorary degrees.</p>]<p>Synthesis, functionalization and applications of nanoparticle bioprobes for molecular cellular in vivo imaging in biology and biomedicine. Linear and nonlinear difference frequency mixing ultrasound imaging. Lithium metal-sulfur batteries, new approaches to electrochemical splitting of water. CO2 reduction, lithium extraction from salt water</p>\n",
      "\n",
      "[<p>Jennifer Cochran is the Shriram Chair of the Department of Bioengineering at Stanford University. She is a Professor of Bioengineering and, by courtesy, Chemical Engineering and a member of the Cancer Biology, Biophysics, and Immunology graduate programs. Dr. Cochran also serves as the Director of the Stanford/NIH Biotechnology pre-doctoral training program. Her research group uses interdisciplinary approaches in chemistry, engineering, and biophysics to study complex biological systems and to develop new tools for basic science and biomedical applications. Dr. Cochran's translational interests span protein-based drug discovery and development for applications in oncology and regenerative medicine, and development of new technologies for high-throughput protein analysis and engineering. Dr. Cochran obtained her Ph.D. in Biological Chemistry from the Massachusetts Institute of Technology, where she also completed a postdoctoral fellowship in Biological Engineering.</p>]<p>The Cochran laboratory uses interdisciplinary approaches in chemistry, engineering, and biophysics to study complex biological systems. Our main goals are to develop new technologies for basic science and biomedical applications. Clinical applications of our research involves wound healing, cardiac tissue regeneration, ocular disease, and cancer imaging and therapy. Our research is driven by the philosophy that in order to control physiological processes it is necessary to understand the molecular mechanisms that drive these processes. We are interested in elucidating molecular details of receptor-mediated cell signaling events; at the same time developing protein and peptide-based tools that will allow us to manipulate cellular processes on a molecular level. For biomedical applications, we are combining rational design and combinatorial methods to create designer protein therapeutics and diagnostic agents. &lt;br/&gt;&lt;br/&gt;Examples of our work are highlighted here: &lt;br/&gt;- http://news.stanford.edu/news/2014/september/metastasis-protein-therapy-092114.html&lt;br/&gt;- https://news.stanford.edu/2015/12/07/proteins-scale-extraction-120715/&lt;br/&gt;- https://news.stanford.edu/2016/06/16/stanford-scientists-create-guided-chemotherapy-missiles-target-cancer-cells-spare-healthy-ones/</p>\n",
      "\n",
      "[<p>Dr. Bianxiao Cui is the Job and Gertrud Tamaki Professor of Chemistry and a fellow of the Wu Tsai Stanford Neuroscience Institute at Stanford University. She holds a Ph.D. degree in Chemistry from the University of Chicago and a BS degree from the University of Science and Technology of China. Dr. Cui develops new tools to study the nano-bio interface, electrophysiology, and signal transduction in cells at normal and disease conditions. As a scientist and a teacher, she enjoys working with young scientists to explore the natural world with scientific innovations. Research in her group spans the disciplines of physical chemistry, material science, nanotechnology, and neurobiology. Her awards and distinctions include Barany Award from Biophysical Society, NIH New Innovator Award, NSF CAREER award, NSF Inspire award, Packard Fellowships in Science and Engineering, Hellman Scholar, Searle Scholar Award and Dreyfus New Faculty Award.</p>]<p>Our objective is to develop new biophysical methods to advance current understandings of cellular machinery in the complicated environment of living cells. Creative applications of biophysical technologies such as crystallography, nuclear magnetic resonance and single-molecule fluorescence imaging have significantly advanced our understanding of biomolecular interactions and functions. However, traditional approaches that analyze purified biomolecules only yield insights that are removed from the cellular context. Our approach is to develop tools that precisely manipulate and measure biomolecular functions in live cells. Currently, we are focusing on four research areas: (1) Membrane curvature at the nano-bio interface; (2) Nanoelectrode arrays (NEAs) for scalable intracellular electrophysiology; (3) Electrochromic optical recording (ECORE) for neuroscience; and (4) Optical control of neurotrophin receptor tyrosine kinases. &lt;br/&gt;&lt;br/&gt;https://cuilab.stanford.edu/</p>\n",
      "\n",
      "[<p>Professor Dai’s research spans chemistry, physics, and materials and biomedical sciences, leading to materials with properties useful in electronics, energy storage and biomedicine. Recent developments include near-infrared-II fluorescence imaging, ultra-sensitive diagnostic assays, a fast-charging aluminum battery and inexpensive electrocatalysts that split water into oxygen and hydrogen fuels.<br/><br/>Born in 1966 in Shaoyang, China, Hongjie Dai began his formal studies in physics at Tsinghua U. (B.S. 1989) and applied sciences at Columbia U. (M.S. 1991). He obtained his Ph.D. from Harvard U and performed postdoctoral research with Dr. Richard Smalley. He joined the Stanford faculty in 1997, and in 2007 was named Jackson–Wood Professor of Chemistry. Among many awards, he has been recognized with the ACS Pure Chemistry Award, APS McGroddy Prize for New Materials, Julius Springer Prize for Applied Physics and Materials Research Society Mid-Career Award. He has been elected to the American Academy of Arts and Sciences, National Academy of Sciences (NAS), National Academy of Medicine (NAM) and Foreign Member of Chinese Academy of Sciences.<br/><br/>The Dai Laboratory has advanced the synthesis and basic understanding of carbon nanomaterials and applications in nanoelectronics, nanomedicine, energy storage and electrocatalysis.<br/><br/>Nanomaterials<br/>The Dai Lab pioneered some of the now-widespread uses of chemical vapor deposition for carbon nanotube (CNT) growth, including vertically aligned nanotubes and patterned growth of single-walled CNTs on wafer substrates, facilitating fundamental studies of their intrinsic properties. The group developed the synthesis of graphene nanoribbons, and of nanocrystals and nanoparticles on CNTs and graphene with controlled degrees of oxidation, producing a class of strongly coupled hybrid materials with advanced properties for electrochemistry, electrocatalysis and photocatalysis. The lab’s synthesis of a novel plasmonic gold film has enhanced near-infrared fluorescence up to 100-fold, enabling ultra-sensitive assays of disease biomarkers.<br/><br/>Nanoscale Physics and Electronics<br/>High quality nanotubes from his group’s synthesis are widely used to investigate the electrical, mechanical, optical, electro-mechanical and thermal properties of quasi-one-dimensional systems. Lab members have studied ballistic electron transport in nanotubes and demonstrated nanotube-based nanosensors, Pd ohmic contacts and ballistic field effect transistors with integrated high-kappa dielectrics.<br/><br/>Nanomedicine and NIR-II Imaging<br/>Advancing biological research with CNTs and nano-graphene, group members have developed π–π stacking non-covalent functionalization chemistry, molecular cellular delivery (drugs, proteins and siRNA), in vivo anti-cancer drug delivery and in vivo photothermal ablation of cancer. Using nanotubes as novel contrast agents, lab collaborations have developed in vitro and in vivo Raman, photoacoustic and fluorescence imaging. Lab members have exploited the physics of reduced light scattering in the near-infrared-II (1000-1700nm) window and pioneered NIR-II fluorescence imaging to increase tissue penetration depth in vivo. Video-rate NIR-II imaging can measure blood flow in single vessels in real time. The lab has developed novel NIR-II fluorescence agents, including CNTs, quantum dots, conjugated polymers and small organic dyes with promise for clinical translation.<br/><br/>Electrocatalysis and Batteries<br/>The Dai group’s nanocarbon–inorganic particle hybrid materials have opened new directions in energy research. Advances include electrocatalysts for oxygen reduction and water splitting catalysts including NiFe layered-double-hydroxide for oxygen evolution. Recently, the group also demonstrated an aluminum ion battery with graphite cathodes and ionic liquid electrolytes, a substantial breakthrough in battery science.</p>]\n",
      "\n",
      "[<p>Dr. Das is an Associate Professor of Biochemistry at Stanford University School of Medicine. After training in particle physics and cosmology at Harvard, Cambridge, University College London, and Stanford, Dr. Das did postdoctoral research in computational protein folding at the University of Washington with David Baker. On returning to Stanford, Dr. Das set up his lab to focus on computer modeling and design of RNA molecules, which underlie important molecular machines in biology and medicine. As a core part of this research, Dr. Das leads Eterna, an open science platform that crowdsources intractable RNA design problems to 250,000 players of an online videogame and provides scoring feedback based on actual wet-lab experiments. Dr. Das’s research has been recognized by the Burroughs-Wellcome Career Award at the Interface of Science, the OpenEye/American Chemical Society Outstanding Junior Faculty Award, and the Stanford Medicine Endowed Faculty Scholar award. In response to the COVID-19 pandemic, Dr. Das and Eterna have launched the OpenVaccine challenge to develop refrigerator-stable mRNA vaccines needed for world-wide immunization.</p>]<p>Our lab strives to predict and design how biopolymer sequences define and regulate biopolymer structure/function, focusing on medically important RNA and RNA/protein complexes.&lt;br/&gt;&lt;br/&gt;We develop algorithms to predict the structures and energetics of RNAs and RNA/protein interfaces at high resolution, with an increasing focus on ribosomes and viruses. We test these ideas through community-wide blind trials and by solving molecule structures and structure ensembles with chemical mapping, NMR, crystallographic, and cryoelectron microscopy data. Notable achievements include top models in the majority of RNA-Puzzles blind structure prediction challenges and first experimental structures of several historically and biomedically important RNA molecules, such as the Tetrahymena ribozyme.&lt;br/&gt;&lt;br/&gt;Complementary to this computational research, we are developing information-rich biochemical methods to model the myriad structures of non-coding RNAs that remain unknown. Current efforts focus on probing the extent and biological impact of RNA structure and conformational change in fundamental processes like splicing and mRNA transport in brain cells and viruses.&lt;br/&gt;&lt;br/&gt;In addition to modeling RNAs, we aim to design new ones for basic science, diagnostics, therapeutics, and vaccines. Our videogame project Eterna seeks missing rules and novel molecules for medicine by giving citizen scientists access to high-throughput wet-lab experiments. Notable achievements include the first algorithm for automated 3D RNA design, development of the current community benchmark for RNA design, discovery of optimal 'zero-energy' switches made of RNA, and invention of RNA calculators for point-of-care diagnostics responsive to complex gene signatures for active tuberculosis. This project has also given rise to several firsts in citizen science, including the first papers written by videogame players as lead authors and as sole authors.</p>\n",
      "\n",
      "[<p>Laura Dassama is a chemical biologist who uses principles from chemistry and physics to understand complex biological phenomenal, and to leverage that understanding for the modulation of biological processes. Her current research focuses on deciphering the molecular recognition mechanisms of multidrug transporters implicated in drug resistance, rational engineering and repurposing of natural products, and control of transcription factors relevant to sickle cell disease.</p>]\n",
      "\n",
      "<p>We are interested in the molecular basis of T and B lymphocyte recognition, as well as the control of differentiation and functional responses in these cells.These studies have ranged from  analyzing the inherent diversity of these highly diverse molecules and relating it to their function and specificity (Davis and Bjorkman 1988; Xu et al 2000) to basic aspects of TCR biochemistry and cell biology (Huppa et al 2010, Lillemeier et al, 2010). We also developed peptide-MHC tetramers which are useful for staining and isolating specific T cells in both basic science and clinical applications (Altman et al 1996; Newell et al., 2012, 2013).&lt;br/&gt;We have also been very much involved in trying to relate what we have learned in basic immunology using mouse models to understanding the human immune system. Here we have employed systems biology approaches to understand vaccine responses (Furman et al., 2013), twin studies to understand the relative influence of environment versus genetics (ongoing) and  T cell repertoire studies to understand self vs non-self capabilities and the origin of memory T cell responses (Su et al 2013). In this last paper, what is particularly interesting is that we have found that healthy adults have abundant numbers of pathogen-specific memory T cells even for viruses that they have never been exposed to, whereas newborns do not. This suggests that there is an unexpected source of protective lymphocytes in adults that likely impact disease resistance and conversely, indicates why young children are so vulnerable to infectious diseases. In general we have found that analyzing the human immune system has presented us with many surprises and complements what we have learned in mice.</p>\n",
      "\n",
      "<p>Molecular imaging technologies for studying cancer biology in vivo</p>\n",
      "\n",
      "[<p>José Dinneny received his BS from UC Berkeley in Plant Biology and Genetics in the Department of Plant and Microbial Biology and PhD from UC San Diego working with Detlef Weigel at the Salk Institute for Biological Science and Martin Yanofsky in the Division of Biology, UCSD.  His work focused on the cloning and characterization of JAGGED and NUBBIN in flower and fruit development.  He then went to Duke University to do his post-doctoral studies with Philip Benfey.  There he utilized Fluorescence Activated Cell Sorting (FACS) to develop the first tissue-specific map of transcriptional changes occurring during abiotic stress.  José established his independent lab at the Temasek Lifesciences Laboratory (TLL) in Singapore with a joint appointment at the National University of Singapore, Department of Biological Sciences. He was an inaugural fellow of the National Research Foundation, Singapore.  José moved his lab in 2011 to the Carnegie Institution for Science, Department of Plant Biology. In 2018 José joined the faculty at Stanford University in the Biology Department as an Associate Professor.  In the Biology Department he serves as Director of Graduate Studies and chair of the Graduate Studies Committee.<br/><br/>He is an HHMI-Simons Faculty Scholar and was a National Research Foundation of Singapore fellow, an NIH Ruth Kirschstein post-doctoral fellow and an HHMI predoctoral fellow.  He was recognized in 2017 by Science News magazine’s 2017 SN 10: Scientists to Watch list.</p>]<p>In the next 50 years, one of the greatest advances we can make for global human health is the realization of a society that is fully sustainable.  My research aims to improve agricultural sustainability by using a holistic approach that integrates across genetic, cellular and organismal scales to understand how plants survive stressful environments (Dinneny, 2015a; 2019). Prior research has explored water-stress responses at unparalleled spatial and temporal resolution, and identified the endodermal tissue layer as a critical signaling center for controlling growth and tissue differentiation in roots (Duan et al., 2013; Geng et al., 2013; Dinneny et al., 2008).  The discovery of novel adaptive mechanisms used by roots to capture water established potential targets for breeding to improve water use efficiency (Bao et al., 2014; Sebastian et al., 2016).  The invention of imaging methods enabled multidimensional studies of plant acclimation and illuminated our understanding of organ system growth from germination to senescence (Rellán-Álvarez et al., 2015; Sebastian et al., 2016). Physiological and molecular insight has been gained in understanding how plants sense water availability through computational modeling of tissue hydraulics (Robbins and Dinneny, 2015, 2018).  Additionally, fine-scale biomechanical measurements identified a novel mechanism by which salinity damages cells through its effects on cell-wall integrity (Feng et al., 2018).  I have paired my research with a personal passion for improving the education of young plant scientists, engaging lawmakers through science policy, and by being a vocal advocate for the broad deployment of agricultural biotechnology (Fahlgren et al., 2016, Friesner et al., 2021).</p>\n",
      "\n",
      "[<p>How is the function of biomolecules in living systems related to their atomic structure?<br/><br/>Professor Doniach's research group uses scattering of synchotron X-rays from electron storage rings at SLAC and at the Argonne National Laboratory to study changes in the conformation of molecules as their solvent environments are changed. The research also involves computer simulations of the dynamics and energetic of the resulting changes.<br/><br/>recent Advances in the biology of DNA have shown that a very large part of the genome in eukaryotes codes for small RNA molecules that appear to be centralto the way the genes (coding for proteins) are put together. Doniach's group is currently studying structural changes that occur when some small functional RNA's turn on and off gene expression (riboswitches) without needing to involve protein transcription factors. Understanding RNA control mechanisms is central to our ability to intervene in biological functions such as generation of biofuels by bacteria or of intervention when cells start to go cancerous.<br/><br/>The Doniach group's bio-simulation work involves new ways to represent changes in molecular structure, in which the entire trajectory for a change of conformation is represented in a large number of CPU's where each time slice of the trajectory is managed by one of the CPU's. In this way, a representationof changes involving thousands of degrees of freedom may be obtained at atomic reslution. This method has recently been applied to look at protein misfolding. Another project involves using a highly simplified normal mode representationto represent large scale conformational changes in molecular motor molecules and DNS polymerase.<br/><br/>The group is also working on ways to improve the methods of computing the statistical mechanics of counter-ion shielding of the very large Coulomb forces endangered by the phosphate backbones of DNA and RNA. Software has been developed that modifies the solving of the Poisson Boltzmann equation to include the effects of finite ion size. Further modifications are being worked in to include effects of ion-on correlations.<br/><br/>Current Area of Focus:<br/>- Membrane Proteins</p>]<p>Study of changes in conformation of proteins and RNA using x-ray scattering</p>\n",
      "\n",
      "[<p>Ron Dror is an Associate Professor of Computer Science in the Stanford Artificial Intelligence Lab. Dr. Dror leads a research group that uses molecular simulation and machine learning to elucidate biomolecular structure, dynamics, and function, and to guide the development of more effective medicines. He collaborates extensively with experimentalists in both academia and industry.<br/><br/>Before moving to Stanford, Dr. Dror served as second-in-command of D. E. Shaw Research, a hundred-person company, having joined as its first hire. He designed computer hardware, software, and algorithms that accelerate molecular dynamics simulations by orders of magnitude, and applied these simulations to the study of protein function, protein folding, and protein-drug interactions.<br/><br/>Dr. Dror earned a PhD in Electrical Engineering and Computer Science at MIT, where he developed machine learning methods for computer vision and genomics. He earned an MPhil in Biological Sciences as a Churchill Scholar at the University of Cambridge, as well as undergraduate degrees in Mathematics and in Electrical and Computer Engineering at Rice University, summa cum laude. He has been awarded a Fulbright Scholarship and fellowships from the National Science Foundation, the Department of Defense, and the Whitaker Foundation, as well as two Gordon Bell Prizes and several Best Paper awards. His work has been highlighted by Science as a top-10 breakthrough of the year.</p>]<p>My lab’s research focuses on computational biology, with an emphasis on 3D molecular structure. We combine two approaches: (1) Bottom-up: given the basic physics governing atomic interactions, use simulations to predict molecular behavior; (2) Top-down: given experimental data, use machine learning to predict molecular structures and properties. We collaborate closely with experimentalists and apply our methods to the discovery of safer, more effective drugs.</p>\n",
      "\n",
      "<p>Our research goal is to understand how dynamics in neuronal circuits relate and constrain the representation of information and computations upon it. We adopt three synergistic strategies: First, we analyze neural circuit population recordings  to better understand the relation between neural dynamics and behavior, Second, we theoretically explore the types of dynamics that could be associated with particular network computations. Third, we analyze the structural properties of neural circuits.</p>\n",
      "\n",
      "[<p>Alex Dunn is an Associate Professor in the Department of Chemical Engineering at Stanford University. His research focuses on understanding how living cells sense mechanical stimuli, with particular interests in stem cell biology and tissue engineering. Dr. Dunn worked as a postdoctoral scholar with James Spudich in the Department of Biochemistry at the Stanford University School of Medicine. He received his Ph.D. at the California Institute of Technology under the direction of Harry Gray, where his work focused on understanding the catalytic mechanism selective C-H bond oxidation by cytochrome P450 enzymes. His work has been recognized with numerous awards, including the Hertz Fellowship, the Burroughs Wellcome Career Award at the Scientific Interface, the NIH Director’s New Innovator Award, and the HHMI Faculty Scholar Award.</p>]<p>The goal of our laboratory is to determine how molecular-scale information encodes the shape and physical properties of cells, tissues, and whole organisms. To do so, we use a combination of sophisticated microscopy, single-molecule biophysics, and theoretical modeling to explore how information propagates upwards across biological length scales. Specific questions we are currently investigating include: 1) How do molecular-scale asymmetries encoded in individual proteins give rise to the emergent physical properties of the cell; and 2) How do cells coordinate their actions to shape organs and tissues? In helping to answer these general questions we hope to understand the physical principles that underlie the construction of complex, multicellular life. We anticipate that this knowledge will be highly relevant to the development of stem-cell-based therapies and to engineering complex, three-dimensional tissues in the laboratory.</p>\n",
      "\n",
      "<p>Biological membranes act as selective barriers that separate the interior of cells from their outside environment. Membrane proteins play crucial roles in a wide range of biological and physiological processes and are targeted by a large number of pharmacologically active compounds including ~ 50% of the drugs in use today. Lack of high-resolution structural information has become a bottleneck for the mechanistic understanding of membrane proteins and hinders potential drug development. Our research interest lies primarily in understanding the mechanism and regulation of these dynamic membrane proteins, and developing small molecule modulators based on their structures and functions. We are taking a multi-disciplinary approach, employing structural methods to capture the high-resolution picture of different states, functional assays and biophysical methods to dissect the information inferred from the structure and computational approaches to understand their dynamics and kinetics.</p>\n",
      "\n",
      "<p>My lab has two main goals: to understand the regulation of mitosis and to understand the systems-level logic of simple signaling circuits.  We often make use of Xenopus laevis oocytes, eggs, and cell-free extracts for both sorts of study.  We also carry out single-cell fluorescence imaging studies on mammalian cell lines.  Our experimental work is complemented by computational and theoretical studies aimed at understanding the design principles and recurring themes of regulatory circuits.</p>\n",
      "\n",
      "<p>Evolutionary &amp;amp; ecological dynamics &amp;amp; diversity, microbial,  expt'l,  &amp;amp; cancer</p>\n",
      "\n",
      "[<p>Polly Fordyce is an Assistant Professor of Genetics and Bioengineering and fellow of the ChEM-H Institute at Stanford, where her laboratory focuses on developing and applying new microfluidic platforms for quantitative, high-throughput biophysics and biochemistry and single-cell genomics. She graduated from the University of Colorado at Boulder with undergraduate degrees in physics and biology before moving to Stanford University, where she earned a Ph.D. in physics for work with Professor Steve Block developing instrumentation and assays for single-molecule studies of kinesin motor proteins. For her postdoctoral research, she worked with Professor Joe DeRisi to develop a new microfluidic platform for understanding how transcription factors recognize and bind their DNA targets as well as a new technology for bead-based multiplexing. She is the recipient of a number of awards, including an NIH New Innovator Award and an Alfred P. Sloan Foundation Research Fellowship, and is a Chan Zuckerberg Biohub Investigator.</p>]<p>Cellular function and organismal homeostasis are governed by molecular interactions. Protein-DNA binding interactions are essential for regulating gene transcription and translation, dense networks of protein-protein and protein-peptide interactions further regulate cellular function, and enzymes make possible all of the chemical transformations essential to metabolism and signaling.  Our goal is to understand, and eventually engineer, these complex processes by building and testing biophysical models of how the molecules that drive these processes work. To do so, an essential first step is to obtain the necessary quantitative measurements of the fundamental kinetic and thermodynamic constants of these molecular interactions and catalytic processes—the “universal language” needed to describe and ultimately predict function. In our lab, we use microfluidics and extensive hardware automation to perform these quantitative measurements at an unprecedented scale via 3 main platforms:&lt;br/&gt;&lt;br/&gt;1. Array-based multiplexing experiments (MITOMI and HT-MEK) employ microfluidic devices containing 1,568 valved reaction chambers aligned to printed DNA arrays. We are currently using these devices to better understand how transcription factors find and bind their genomic targets to regulate gene expression, as well as to understand how enzymes achieve their extraordinary catalytic efficiency and substrate specificity.&lt;br/&gt;&lt;br/&gt;2. MRBLEs (Microspheres with Ratiometric Barcode Lanthanide Encoding) rely on spectral multiplexing to track analytes throughout an experiment. We can create microspheres containing &amp;gt; 1,000 distinct ratios of lanthanide nanophosphors that can be uniquely identified via imaging alone, and are now using these MRBLEs in a variety of downstream assays.&lt;br/&gt;&lt;br/&gt;3. Dropception is a microfluidic platform for creating double emulsion (water-in-oil-in-water) droplets that can be sorted in high-throughput using standard flow cytometers (FACS machines). We recently demonstrated the ability to generate and sort double emulsion droplets without breakage, isolate individual rare droplets of interest in wells of a multiwell plate, and recover all encapsulated nucleic acids, enabling a wide range of novel single-cell multi-omic techniques.</p>\n",
      "\n",
      "<p>The long term goal  of our research is to understand how proteins fold in living cells. My lab uses a multidisciplinary approach to address fundamental questions about molecular chaperones, protein folding and degradation. In addition to basic mechanistic principles, we aim to define how impairment of cellular folding and quality control are linked to disease, including cancer and neurodegenerative diseases and examine whether  reengineering chaperone networks can provide therapeutic strategies.</p>\n",
      "\n",
      "[<p>How do we design biological systems as “smart medicine” that sense patients’ states, process the information, and respond accordingly? To realize this vision, we will tackle fundamental challenges across different levels of complexity, such as (1) protein components that minimize their crosstalk with human cells and immunogenicity, (2) biomolecular circuits that function robustly in different cells and are easy to deliver, (3) multicellular consortia that communicate through scalable channels, and (4) therapeutic modules that interface with physiological inputs/outputs. Our engineering targets include biomolecules, molecular circuits, viruses, and cells, and our approach combines quantitative experimental analysis with computational simulation. The molecular tools we build will be applied to diverse fields such as neurobiology and cancer therapy.</p>]\n",
      "\n",
      "<p>My laboratory studies the structural and functional basis of receptor/ligand interactions in systems which are relevant to human health and disease. Our investigations are aimed at understanding the molecular recognition properties governing the interactions of receptors with their ligands, and the subsequent molecular events which couple ligand recognition to receptor activation. Many of the systems we are studying in the laboratory are related to the interaction of the host with the environment. The structural studies are complemented by functional approaches using molecular biology and protein engineering to dissect the structural information, design new or altered proteins with modified specificities and activities, and ultimately contribute to the development of proteins or molecules with therapeutic potential. Molecules currently under study include receptors of the immune system involved in autoimmune disorders (T cell receptors, co-receptors, MHC, cytokines), proteins involved in host-pathogen interactions and molecular mimicry (CMV and Toxoplasma surface antigens), proteins of nervous system (peptide hormone receptors, neural guidance proteins), and membrane proteins (chemokine receptors).  An emerging focus of our research is to develop. using combinatorial biology approaches, novel ligands for receptors, which may have altered activities, that may serve as therapeutic starting points.</p>\n",
      "\n",
      "<p>My lab focuses on biomedical data fusion: the development of machine learning methods for biomedical decision support using multi-scale biomedical data. Previously we pioneered data fusion work using Bayesian and kernel methods studying breast and ovarian cancer. Additionally, we developed computational algorithms for the identification of driver genes using multi-omics data. Furthermore, we are working on multi-scale biomedical data fusion methods, bridging the molecular using omics data, cellular using pathology data and tissue using medical imaging data.</p>\n",
      "\n",
      "[<p>My research interests encompass the physics and mathematics of imaging with Magnetic Resonance (MR). My research is directed in part towards exploration of rapid MRI scanning methods using spiral and other non-Cartesian k-space trajectories for dynamic imaging of function. Using spiral techniques, we have developed MRI pulse sequences and processing methods for mapping cortical brain function by imaging the metabolic response to various stimuli, with applications in the basic neurosciences as well as for clinical applications. These methods develop differential image contrast from hemodynamically driven increases in oxygen content in the vascular bed of activated cortex (Blood Oxygen Level Dependent, or BOLD contrast), using pulse sequences sensitive to the paramagnetic behavior of deoxyhemoglobin or to the blood flow changes. Other interests include multimodal imaging using fMRI in conjunction with EEG, fPET, fNIRS, and neuromodulation with tDCS, tACS, TMS and HiFU.  Investigating viscoelasticity of human brain using MR Elastography is of interest as an alternative to BOLD contrast for depicting brain activation.</p>]<p>My research is devoted to the advancement of imaging sciences for applications in diagnostic radiology. We collaborate closely with departmental clinicians and with others in the school of medicine, humanities, and the engineering sciences. &lt;br/&gt;&lt;br/&gt;Presently my research is directed in part towards exploration of rapid scanning methods using spiral and other non-Cartesian k-space trajectories. Using spiral techniques, we have developed MRI pulse sequences and processing methods for mapping cortical brain function by imaging the metabolic response to various stimuli, with applications in the basic neurosciences as well as for clinical applications. These methods develop differential image contrast from hemodynamically driven increases in oxygen content in the vascular bed of activated cortex, using pulse sequences sensitive to the paramagnetic behavior of deoxyhemoglobin or to the blood flow changes. Multimodal imaging and neuromodulation combines fMRI with EEG, fPET, fNIRS, TES (TACS, TDCS, TMS) and HIFU.</p>\n",
      "\n",
      "[<p>Benjamin Good is a theoretical biophysicist with a background in experimental evolution and population genetics. He is interested in the short-term evolutionary dynamics that emerge in rapidly evolving microbial populations like the gut microbiome. Technological advances are revolutionizing our ability to peer into these evolving ecosystems, providing us with an increasingly detailed catalog of their component species, genes, and pathways. Yet a vast gap still remains in understanding the population-level processes that control their emergent structure and function. Our group uses tools from statistical physics, population genetics, and computational biology to understand how microscopic growth processes and genome dynamics at the single cell level give rise to the collective behaviors that can be observed at the population level. Projects range from basic theoretical investigations of non-equilibrium processes in microbial evolution and ecology, to the development of new computational tools for measuring these processes in situ in both natural and experimental microbial communities. Through these specific examples, we seek to uncover unifying theoretical principles that could help us understand, forecast, and eventually control the ecological and evolutionary dynamics that take place in these diverse scenarios.</p>]\n",
      "\n",
      "<p>We study the molecular events that give rise to the sensation of touch and temperature using C. elegans nematodes as a model system. To do this, we use a combination of quantitative behavioral analysis, genetics, in vivo electrophysiology, and heterologous expression of ion channels.  We also collaborate with Pruitt's group in Mechanical Engineering (http://microsystems.stanford.edu) to develop and fabricate novel devices for the study of sensory transduction.</p>\n",
      "\n",
      "[<p>William Greenleaf is an Associate Professor in the Genetics Department at Stanford University School of Medicine, with a courtesy appointment in the Applied Physics Department. He is a member of Bio-X, the Biophysics Program, the Biomedical Informatics Program, and the Cancer Center. He received an A.B. in physics from Harvard University (summa cum laude) in 2002, and received a Gates Fellowship to study computer science for one year in Trinity College, Cambridge, UK (with distinction). After this experience abroad, he returned to Stanford to carry out his Ph.D. in Applied Physics in the laboratory of Steven Block, where he investigated, at the single molecule level, the chemo-mechanics of RNA polymerase and the folding of RNA transcripts. He conducted postdoctoral work in the laboratory of X. Sunney Xie in the Chemistry and Chemical Biology Department at Harvard University, where he was awarded a Damon Runyon Cancer Research Foundation Fellowship, and developed new fluorescence-based high-throughput sequencing methodologies. He moved to Stanford as an Assistant Professor in November 2011. Since beginning his lab, he has been named a Rita Allen Foundation Young Scholar, an Ellison Foundation Young Scholar in Aging (declined), a Baxter Foundation Scholar, and a Chan-Zuckerberg Investigator. His highly interdisciplinary research links molecular biology, computer science, bioengineering, and genomics a to understand how the physical state of the human genome controls gene regulation and biological state. Efforts in his lab are split between building new tools to leverage the power of high-throughput sequencing and cutting-edge microscopies, and bringing these new technologies to bear against basic biological questions of genomic and epigenomic variation. His long-term goal is to unlock an understanding of the physical “regulome” — i.e. the factors that control how the genetic information is read into biological instructions — profoundly impacting our understanding of how cells maintain, or fail to maintain, their state in health and disease.</p>]<p>Our lab focuses on developing methods to probe both the structure and function of molecules encoded by the genome, as well as the physical compaction and folding of the genome itself. Our efforts are split between building new tools to leverage the power of high-throughput sequencing technologies and cutting-edge optical microscopies, and bringing these technologies to bear against basic biological questions by linking DNA sequence, structure, and function.</p>\n",
      "\n",
      "<p>Scientific breakthroughs often come on the heels of technological advances; advances that expose hidden truths of nature, and provide tools for engineering the world around us. Examples include the telescope (heliocentrism), the Michelson interferometer (relativity) and recombinant DNA (molecular evolution). Our lab explores innovative experimental approaches to problems in molecular biochemistry, focusing on technologies with the potential for broad impact.</p>\n",
      "\n",
      "<p>Our laboratory studies the mechanisms by which highly complex behaviors are mediated at the neuronal level, mainly focusing on the example of dynamic social interactions and the neural circuits that drive them. From dyadic interactions to group dynamics and collective decision making, the lab seeks a mechanistic understanding for the fundamental building blocks of societies, such as cooperation, empathy, fairness and reciprocity.&lt;br/&gt;&lt;br/&gt;The computations underlying social interactions are highly distributed across many brain areas. Our lab is interested in which specific areas are involved in a particular function, why such an architecture arises and how activity in multiple networks is coordinated. Our goal is to develop a roadmap of the social brain and use it for guiding restorative treatments for conditions in which social behavior is impaired, such as Autism Spectrum Disorders and Schizophrenia.&lt;br/&gt;&lt;br/&gt;%%&lt;br/&gt;&lt;br/&gt;We have open positions for currently enrolled graduate students (through available rotation opportunities) and prospective postdoctoral fellows (supported by federal funding) for studying the neuronal basis of dynamic social interactions. We perform cutting-edge large-scale recordings and targeted perturbation in rich social tasks based on game-theory combined with advanced analytical approaches, brain-computer interface, machine vision and deep learning to understand the single neuron and population level underpinnings of complex social computations. We are looking to recruit bright, highly motivated, and friendly individuals with strong interest (for students) or background (for postdoctoral applicants) in neurophysiology and/or computation. Individuals from groups under-represented in STEM are especially encouraged to apply. For more details, please email Keren Haroush directly at kharoush@stanford.edu.</p>\n",
      "\n",
      "[<p>The overarching goal of the Herschlag Lab is to understand the fundamental behavior of RNA and proteins and, in turn, how these behaviors determine and impact biology more broadly. We are particularly interested in questions of how enzymes work, how RNA folds, how proteins recognize RNA, and the roles of RNA/protein interactions in regulation and control, and the evolution of molecules and molecular interactions. The lab takes an interdisciplinary approach, spanning and integrating physics, chemistry and biology, and employing a wide range of techniques.</p>]<p>The overarching goal of the Herschlag Lab is to understand the fundamental behavior of RNA and proteins and, in turn, how these behaviors determine and impact biology more broadly. We are particularly interested in questions of how enzymes work, how RNA folds, how proteins recognize RNA, and the roles of RNA/protein interactions in regulation and control, and the evolution of molecules and molecular interactions. The lab takes an interdisciplinary approach, spanning and integrating physics, chemistry and biology, and employing a wide range of techniques.</p>\n",
      "\n",
      "[<p>Combining inorganic, biophysical and structural chemistry, Professor Keith Hodgson investigates how structure at molecular and macromolecular levels relates to function. Studies in the Hodgson lab have pioneered the use of synchrotron x-radiation to probe the electronic and structural environment of biomolecules. Recent efforts focus on the applications of x-ray diffraction, scattering and absorption spectroscopy to examine metalloproteins that are important in Earth’s biosphere, such as those that convert nitrogen to ammonia or methane to methanol.<br/><br/>Keith O. Hodgson was born in Virginia in 1947. He studied chemistry at the University of Virginia (B.S. 1969) and University of California, Berkeley (Ph.D. 1972), with a postdoctoral year at the ETH in Zurich. He joined the Stanford Chemistry Department faculty in 1973, starting up a program of fundamental research into the use of x-rays to study chemical and biological structure that made use of the unique capabilities of the Stanford Synchrotron Radiation Lightsource (SSRL).  His lab carried out pioneering x-ray absorption and x-ray crystallographic studies of proteins, laying the foundation for a new field now in broad use worldwide. In the early eighties, he began development of one of the world's first synchrotron-based structural molecular biology research and user programs, centered at SSRL. He served as SSRL Director from 1998 to 2005, and SLAC National Accelerator Laboratory (SLAC) Deputy Director (2005-2007) and Associate Laboratory Director for Photon Science (2007-2011).<br/><br/>Today the Hodgson research group investigates how molecular structure at different organizational levels relates to biological and chemical function, using a variety of x-ray absorption, diffraction and scattering techniques. Typical of these molecular structural studies are investigations of metal ions as active sites of biomolecules. His research group develops and utilizes techniques such as x-ray absorption and emission spectroscopy (XAS and XES) to study the electronic and metrical details of a given metal ion in the biomolecule under a variety of natural conditions.<br/><br/>A major area of focus over many years, the active site of the enzyme nitrogenase is responsible for conversion of atmospheric di-nitrogen to ammonia. Using XAS studies at the S, Fe and Mo edge, the Hodgson group has worked to understand the electronic structure as a function of redox in this cluster. They have developed new methods to study long distances in the cluster within and outside the protein. Studies are ongoing to learn how this cluster functions during catalysis and interacts with substrates and inhibitors. Other components of the protein are also under active study.<br/><br/>Additional projects include the study of iron in dioxygen activation and oxidation within the binuclear iron-containing enzyme methane monooxygenase and in cytochrome oxidase. Lab members are also investigating the role of copper in electron transport and in dioxygen activation. Other studies include the electronic structure of iron-sulfur clusters in models and enzymes.<br/><br/>The research group is also focusing on using the next generation of x-ray light sources, the free electron laser. Such a light source, called the LCLS, is also located at SLAC.  They are also developing new approaches using x-ray free electron laser radiation to image noncrystalline biomolecules and study chemical reactivity on ultrafast time scales.</p>]\n",
      "\n",
      "[<p>My laboratory employs diverse interdisciplinary methods of inquiry to understand the relationships among cell shape detection, determination, and maintenance in bacteria. Cell shape plays a critical role in regulating many physiological functions, yet little is known about how the wide variety of cell shapes are determined and maintained. Inside the cell, many proteins organize and segregate, but how they detect and respond to the cellular morphology to end up at the right place at the right time is also largely mysterious. The group uses a combination of analytical, computational, and experimental approaches to probe physical mechanisms of shape-related self-organization in protein networks, membranes, and the cell wall. Current topics of interest are (i) cell-wall biosynthesis, (ii) the regulation and mechanics of cell division, (iii) membrane organization, and (iv) membrane-mediated protein interactions. Ultimately, the manipulation of cell shape may provide a direct tool for engineering complex cellular behaviors.</p>]<p>We primarily focus on bacteria, in which the exquisite patterning of the interior in both space and time is critical for a wide variety of cellular functions. The wide variety of shapes and sizes that bacteria take on can be used as synthetic environment for studying the establishment of intracellular organization and the cellular response to perturbations in morphology. Ultimately, the manipulation of cell shape may provide a direct tool for engineering complex cellular behaviors. &lt;br/&gt;&lt;br/&gt;Currently, we are interested in (i) the role of the cell wall in cell-shape determination, (ii) the regulation and mechanics of the cell cycle and cell division, (iii) the spatial and temporal organization of the membrane, (iv) the role of the membrane in transmembrane-protein interactions and ion channel gating, and (v) collective behavior in bacteria.</p>\n",
      "\n",
      "<p>Protein design: molecular engineering, method development and novel therapeutics</p>\n",
      "\n",
      "[<p>Christine Jacobs-Wagner is a Dennis Cunningham Professor in the Department of Biology and the ChEM-H Institute at Stanford University. She is interested in understanding the fundamental mechanisms and principles by which cells, and, in particular, bacterial cells, are able to multiple. She received her PhD in Biochemistry in 1996 from the University of Liège, Belgium where she unraveled a molecular mechanism by which some bacterial pathogens sense and respond to antibiotics attack to achieve resistance. For this work, she received multiple awards including the 1997 GE &amp; Science Prize for Young Life Scientists. During her postdoctoral work at Stanford Medical School, she demonstrated that bacteria can localize regulatory proteins to specific intracellular regions to control signal transduction and the cell cycle, uncovering a new, unsuspected level of bacterial regulation.<br/><br/>She started her own lab at Yale University in 2001. Over the years, her group made major contributions in the emerging field of bacterial cell biology and provided key molecular insights into the temporal and spatial mechanisms involved in cell morphogenesis, cell polarization, chromosome segregation and cell cycle control. For her distinguished work, she received the Pew Scholars award from the Pew Charitable Trust, the Woman in Cell Biology Junior award from the American Society of Cell Biology and the Eli Lilly award from the American Society of Microbiology. She held the Maxine F. Singer and William H. Fleming professor chairs at Yale. She was elected to the Connecticut academy of Science, the American Academy of Microbiology and the National Academy of Sciences. She has been an investigator of the Howard Hughes Medical Institute since 2008. <br/><br/>Her lab moved to Stanford in 2019. Current research examines the general principles and spatiotemporal mechanisms by which bacterial cells replicate, using Caulobacter crescentus and Escherichia coli as models.  Recently, the Jacobs-Wagner lab expanded their interests to the Lyme disease agent Borrelia burgdorferi, revealing unsuspected ways by which this pathogen grows and causes disease</p>]\n",
      "\n",
      "<p>The Jardetzky laboratory is studying the structures and mechanisms of macromolecular complexes important in viral pathogenesis, allergic hypersensitivities and the regulation of cellular growth and differentiation, with an interest in uncovering novel conceptual approaches to intervening in disease processes.  Ongoing research projects include studies of paramyxovirus and herpesvirus entry mechanisms, IgE-receptor structure and function and TGF-beta ligand signaling pathways.</p>\n",
      "\n",
      "[<p>Starting fall of 2021, I am winding down a term as chair of physics and then taking an extended sabbatical/leave.  My focus during this period will be on updating my background and competence in rapidly growing new areas of interest including machine learning and its application to problems involving large datasets.<br/><br/>My recent research interests have included mathematical and computational studies of evolutionary dynamics; field theoretic condensed matter physics, including study of non-Fermi liquids and fracton phases; and mathematical aspects of string theory.  I would characterize my research programs in these three areas as being in the fledgling stage, relatively recently established, and well developed, respectively.<br/><br/>It is hard to know what the future holds, but you can get some idea of the kinds of things I work on by looking at my past.  Highlights of my past research include:<br/><br/>- The discovery of string dualities with 4d N=2 supersymmetry, and their use to find exact solutions of gauge theories (with Cumrun Vafa)<br/>- The construction of the first examples of AdS/CFT duality with reduced supersymmetry (with Eva Silverstein)<br/>- Foundational papers on string compactification in the presence of background fluxes (with Steve Giddings and Joe Polchinski)<br/>- Basic models of cosmic acceleration in string theory (with Renata Kallosh, Andrei Linde, and Sandip Trivedi)<br/>- First computation of the non-Gaussianity in general single field inflation (with Xingang Chen, Min-xin Huang, and Gary Shiu)<br/>- Developing the framework underlying holography for non-relativistic field theories, relevant for modeling quantum matter at finite density (with Xiao Liu and Michael Mulligan)<br/>- Simple and tractable models of non-Fermi liquids (with Liam Fitzpatrick, Jared Kaplan, and Sri Raghu)<br/>- Studies of adaptive trade-offs in evolutionary dynamics of organisms exposed to a varying environment (with Daniel Fisher and Mikhail Tikhonov)<br/>- Developing a new approach — using string duality — to find precise expressions for Ricci flat metrics on K3 surfaces (with Arnav Tripathy and Max Zimet)<br/><br/><br/>For details about my present and former students, please see the “Research and Scholarship” link in my full Stanford profile.</p>]<p>Starting fall of 2021, I will be taking an extended sabbatical/leave.  Much of my effort during this period will be spent developing my skills in rapidly developing areas like machine learning, with a view towards the science emerging from analysis of large datasets.&lt;br/&gt;&lt;br/&gt;My recent research has involved three directions:&lt;br/&gt;&lt;br/&gt;— Mathematical aspects of string theory, with a focus on interplay between BPS state counts, physics of black holes, geometry, and number theory.&lt;br/&gt;&lt;br/&gt;— Quantum field theory and condensed matter physics, with a particular interest in the emergence of non-Fermi liquid states of matter and of fracton phases.&lt;br/&gt;&lt;br/&gt;— Theoretical biology, with a focus on evolution and ecology.  This is a new initiative for me.  My first paper in this area appeared on bioRxiv in 2019, and I have started accepting students with a primary interest in this area.&lt;br/&gt;&lt;br/&gt;I typically work with a small number of graduate students and SITP postdocs.  I’ve also taken undergraduate researchers for summer or thesis projects.  My students have gone on to various interesting places after leaving Stanford.&lt;br/&gt;&lt;br/&gt;Present and former Graduate Advisees:&lt;br/&gt;&lt;br/&gt;Anita Kulkarni (current student)&lt;br/&gt;Brandon Rayhaun (current student)&lt;br/&gt;Milind Shyani (PhD, 2021, now working on AI)&lt;br/&gt;Richard Nally (now a Klarman Fellow at Cornell)&lt;br/&gt;Xinghe Li (M.A., 2021)&lt;br/&gt;Max Zimet (now a Fellow in the Black Hole Initiative at Harvard)&lt;br/&gt;Nathan Benjamin (to Princeton PCTS; now at Caltech)&lt;br/&gt;Natalie Paquette (to Caltech postdoc, now a faculty member at UW - Seattle)&lt;br/&gt;Dan Whalen (now at Renaissance Technologies)&lt;br/&gt;Huajia Wang (to Illinois postdoc, faculty at Kavli Institute for Theoretical Sciences, Beijing)&lt;br/&gt;Sarah Harrison (to Harvard postdoc, joint math/physics faculty at McGill)&lt;br/&gt;Dusan Simic (to KITP postdoc)&lt;br/&gt;Mike Mulligan (to MIT postdoc, now faculty at UC Riverside)&lt;br/&gt;Wu-yen Chuang (to Rutgers postdoc, now mathematics faculty at National Taiwan University)&lt;br/&gt;Alexander Giryavets (now Vice-President at Morgan Stanley)&lt;br/&gt;Xiao Liu (to Perimeter Institute postdoc, now faculty at UESTC)&lt;br/&gt;Liam McAllister (to Princeton postdoc, now Cornell faculty)&lt;br/&gt;John McGreevy (to Princeton postdoc, now UCSD faculty)&lt;br/&gt;Michael Schulz (to Caltech postdoc, now Bryn Mawr faculty)&lt;br/&gt;&lt;br/&gt;Present and former undergraduate research students:&lt;br/&gt;&lt;br/&gt;Anna Biggs (Harvard undergrad), summer 2019 (to graduate school at Princeton)&lt;br/&gt;Inigo Lombera, summer 2019&lt;br/&gt;Jordi Montana-Lopez, summer 2019&lt;br/&gt;Sandra Nair (UCSC undergrad), summer 2019 (to math graduate school, Michigan)&lt;br/&gt;Sandip Roy, 2019-20 (to graduate school at Princeton)&lt;br/&gt;Lark Wang, 2019-20 (to graduate school at UC Berkeley)&lt;br/&gt;George Hulsey, 2018-19 (to graduate school at UCSB)&lt;br/&gt;Sungyeon Yang, 2018-19 (now in graduate school at Stanford)&lt;br/&gt;Newton Cheng, 2017-18 (to graduate school at UC Berkeley)&lt;br/&gt;Tudor Ciobanu, 2017-18 (to graduate school at Stony Brook)&lt;br/&gt;Ethan Sussman, 2016-18 (to graduate school in mathematics at MIT)&lt;br/&gt;Preethi Pallegar, 2015-16 (to graduate school at Princeton)&lt;br/&gt;Marc Robbins, 2015 (to graduate school at Illinois)&lt;br/&gt;Zhiming Wang, 2015 (to graduate school at Princeton)&lt;br/&gt;Temple He, 2009-10 (to graduate school at Harvard)&lt;br/&gt;Daniel Balick, 2005-06 (UCSB PhD in theoretical biology, now at Harvard)&lt;br/&gt;Christopher Beem, 2005-06 (Berkeley PhD in string theory, now faculty at Oxford)&lt;br/&gt;Dan Wohns, 2005-06 (Cornell PhD in string theory, now at Perimeter)</p>\n",
      "\n",
      "[<p>Dr. Kapiloff earned his Ph.D. in Biomedical Sciences in 1991 at the University of California, San Diego, in the laboratory of Dr. M. Geoffrey Rosenfeld, a HHMI investigator and member of the National Academy of Sciences. He graduated from UCSD with a Doctorate of Medicine in 1992 and completed a residency in General Pediatrics at the University of Utah in 1995. In 1997, Dr. Kapiloff became a Research Assistant Professor, performing research in the laboratory of HHMI Investigator Dr. John Scott at the Vollum Institute in Portland, OR. From 1999 to 2007, Dr. Kapiloff was an Assistant Professor of Pediatrics at the Oregon Health and Science University in Portland. From 2007 to 2017, he was Director of the Cardiac Signal Transduction and Cellular Biology Laboratory at the University of Miami Miller School of Medicine in Miami, Florida, where he was, as of 2013, a tenured Professor of Pediatrics and Medicine in the Division of Cardiology.  Since 2011, Dr. Kapiloff has also been studying signal transduction pathways in the eye. Dr. Kapiloff was recruited to Stanford in July, 2017 in a joint effort by the Department of Ophthalmology and the Stanford Cardiovascular Institute in recognition of his work in both fields.</p>]<p>Dr. Michael S. Kapiloff is a faculty member in the Departments of Ophthalmology and Medicine (Cardiovascular Medicine) and a member of the Stanford Cardiovascular Institute. Although Dr. Kapiloff was at one time a Board-Certified General Pediatrician, he is currently involved in full-time basic science and translational research. His laboratory studies the basic molecular mechanisms underlying  the response of the retinal ganglion cell and cardiac myocyte to disease. The longstanding interest of his laboratory is the role in intracellular signal transduction of multimolecular complexes organized by scaffold proteins. Recently, his lab has also been involved in the translation of these concepts into new therapies, including the development of new AAV gene therapy biologics for the prevention and treatment of heart failure and for neuroprotection in the eye.&lt;br/&gt;&lt;br/&gt;URL to NCBI listing of all published works:&lt;br/&gt;http://www.ncbi.nlm.nih.gov/sites/myncbi/michael.kapiloff.1/bibliography/40252285/public/?sort=date&amp;amp;direction=descending&lt;br/&gt;&lt;br/&gt;For more information see Dr. Kapiloff's lab website: http://med.stanford.edu/kapilofflab.html</p>\n",
      "\n",
      "<p>We are studying the mechanism of viral membrane fusion and its inhibition by drugs and antibodies. We use the HIV envelope protein (gp120/gp41) as a model system. Some of our studies are aimed at creating an HIV vaccine that elicits antibodies against a transient, but vulnerable, intermediate in the membrane-fusion process, called the pre-hairpin intermediate.  &lt;br/&gt;&lt;br/&gt;     We are also interested in protein surfaces that are referred to as &amp;quot;non-druggable&amp;quot;.  These surfaces are defined empirically based on failure to identify small, drug-like molecules that bind to them with high affinity and specificity.  Some of our efforts are aimed at characterizing select non-druggable targets.  We are also developing methods to identify ligands for non-druggable protein surfaces.</p>\n",
      "\n",
      "<p>My laboratory is involved in studying several aspects of adrenergic receptor biology. Adrenergic receptors form the interface between the sympathetic nervous system and the cardiovascular system and play a critical role in the regulation of cardiovascular function. Specific projects include:&lt;br/&gt;&lt;br/&gt;1- RECEPTOR STRUCTURE: We are interested in understanding the three dimensional structure of adrenergic receptors and learning about the conformational changes that mediate signal transduction. We are taking several experimental approaches including mutagenesis, biochemical, and biophysical studies. &lt;br/&gt;&lt;br/&gt;2-INTRACELLULAR TRAFFICKING OF ADRENERGIC RECEPTORS: The function of receptors can be modulated by changes in receptor structure (phosphorylation) and by changes in subcellular localization. We are using immunocytochemical approaches to study the targeting of receptors to specific subcellular domains and agonist mediated redistribution of receptors. Our goal is to determine the functional significance of differences in targeting and trafficking that we have observed in several adrenergic receptors, and to identify cellular proteins that mediate receptor trafficking.&lt;br/&gt;&lt;br/&gt;3-PHYSIOLOGIC RELEVANCE OF ADRENERGIC RECEPTOR SUBTYPE DIVERSITY: Multiple closely related subtypes of adrenergic receptors have been identified through cloning studies. We are using targeted gene modification in mice to study the physiologic role of these closely related subtypes. We have disrupted the genes for five adrenergic receptors (alpha 2a, alpha 2b, alpha 2c, beta 1, and beta2) and are investigating the consequence of these disruptions on neural and cardiovascular physiology.</p>\n",
      "\n",
      "[<p>Eric Kool received his Ph.D. in Chemistry from Columbia University and did postdoctoral work in nucleic acids chemistry at Caltech. He started his career at the University of Rochester before moving to Stanford in 1999, where he is the George and Hilda Daubert Professor of Chemistry. He teaches Organic Chemistry and Chemical Biology to undergraduate and graduate students.<br/><br/>The Kool lab uses the tools of chemistry to study the structures, interactions and biological activities of nucleic acids and the enzymes that process them. Molecular design and synthesis play a major role in this work, followed by analysis of structure and function, both in test tubes and in living systems. These studies are aimed at gaining a better basic understanding of biology, and applying this knowledge to practical applications in biomedicine.<br/><br/>As part of this research, members of the group synthesize designer nucleobases and nucleotides, with unusual properties such as fluorescence, enzyme reactivity, or altered shape and H-bonding ability. We use these as tools to study DNA polymerase enzymes, DNA repair pathways, and RNA modifying enzymes. This work is leading to new probes for diagnosis of cancer, useful fluorescent tags for biology, and fluorescent sensors of many species such as cancer metabolites and toxic metals.</p>]<p>Our lab uses the tools of molecular design and chemical synthesis, combined with modern molecular biology and genomics techniques, to study the biology of nucleic acids. We have a general interest in the design of small-molecule probes and reagents for the study of RNA and DNA in the cell, and of enzymes that modify them. For example, we are designing cell-permeable reagents that can be used to map structure and contacts of RNAs in living systems. We are also developing novel tools for labeling and caging RNAs, and methods for profiling transcriptome interactions. We are using these tools to uncover new knowledge about the functions of noncoding RNAs in the cell, and to study the potential of new anticancer targets in the transcriptome.&lt;br/&gt;&lt;br/&gt;Our lab is also studying DNA repair enzymes, with a focus on development of tools that will help us measure, and potentially treat, cancer and inflammation. We design enzyme mechanism-specific fluorescent probes of DNA base excision repair, and employ them in cellular and animal models of disease. We also use these probes to discover and develop small molecule inhibitors of these enzymes, to be used in translational models of disease. We collaborate with biomedical research groups in translational studies to test our hypotheses regarding the connections of DNA repair to disease.</p>\n",
      "\n",
      "<p>The ER is the &amp;quot;port of entry&amp;quot; for proteins destined for the cell surface and beyond.  The vast majority of proteins entering the secretory pathway are synthesized on ribosomes docked at ER translocons and are co-transationally translocated into the ER lumen. Proteins synthesized at the ER are subject to covalent modifications that include N- and O-glycosylation, disulfide bond formation, and in some cases, proline and lysine hydroxylation. Membrane proteins must be threaded co-translocationally into the lipid bilayer to become membrane-integrated, often with complex topologies and typically form hetero- or homo- oligomers. This highly complex &amp;quot;protein biogenesis&amp;quot; process is assisted by a diverse network of folding catalysts and protein-modifying enzymes and is scrutinized by molecular chaperones and other &amp;quot;quality control&amp;quot; factors which ensure that only correctly folded and assembled proteins exit the ER and proceed to distal compartments of the secretory pathway.&lt;br/&gt; &lt;br/&gt;The Kopito laboratory seeks a molecular understanding of how cells maintain the fidelity of their proteomes. Unlike DNA, which can be repaired if damaged or incorrectly made, proteins cannot be mended. Instead, damaged or incorrectly synthesized proteins must be rapidly and efficiently destroyed lest they form toxic aggregates. &lt;br/&gt;&lt;br/&gt;Our goal is to elucidate the functional networks that coordinate protein synthesis and quality control in the early secretory pathway. Currently the lab is focused on two specific systems: ERAD and ribosome UFMylation.</p>\n",
      "\n",
      "<p>Our goal is to elucidate the fundamental basis of gene regulation. We study the control of transcription, the first step in the pathway of gene expression. Current work focuses on discovery of the molecular machines involved in transcription, reconstitution of the process with purified components, structure determination of the transcription machinery, and structure-function relationships in chromatin, the natural DNA template for transcription.&lt;br/&gt;&lt;br/&gt;Highlights of work from the past three years include:&lt;br/&gt;&lt;br/&gt;1. Discovery of a human homolog of the 20-protein yeast Mediator complex. Mediator is the central processing unit of gene regulation, receiving both positive and negative inputs and transducing the information to the transcription machinery.&lt;br/&gt;&lt;br/&gt;2. Structure determination of 10-subunit, half million Dalton RNA polymerase II, in the act of transcription, with template DNA and protduct RNA, by X-ray crystallography at atomic resolution.&lt;br/&gt;&lt;br/&gt;3. Structure determination of the entire transcription initiation complex by two-dimensional protein crystallography.&lt;br/&gt;&lt;br/&gt;4. Discovery of a stably altered nucleosome produced by a purified chromatin-remodeling complex.&lt;br/&gt;&lt;br/&gt;Current work is directed towards the structure of the entire transcription apparatus at atomic resolution and the mechanism of transcription control in living cells.</p>\n",
      "\n",
      "[<p>Anshul Kundaje is an Assistant Professor of Genetics and Computer Science at Stanford University. His primary research area is large-scale computational regulatory genomics. The Kundaje lab specializes in developing statistical and machine learning methods for large-scale integrative analysis of heterogeneous, high-throughput functional genomic and genetic data to decipher regulatory elements and long-range regulatory interactions, learn predictive regulatory network models across individuals, cell-types and species and improve detection and interpretation of natural and disease-associated genetic variation. Previously as a postdoc at Stanford and Research Scientist at MIT, Anshul was the lead computational analyst of the ENCODE Project and the Roadmap Epigenomics Project. Anshul is also a recipient of the 2016 NIH Director's New Innovator Award and the 2014 Alfred Sloan Fellowship.</p>]<p>Our research focusses on development of statistical and machine learning methods for integrative analysis of diverse functional genomic and genetic data to learn models of gene regulation. We have led the analysis efforts of the Encyclopedia of DNA Elements (ENCODE) and The Roadmap Epigenomics Projects with the development of novel methods for &lt;br/&gt;1. Adaptive thresholding and normalization of massive collections of functional genomic data (e.g. ChIP-seq and DNase-seq)&lt;br/&gt;2. Dissecting combinatorial transcription factor co-occupancy within and across cell-types&lt;br/&gt;3. Predicting cell-type specific enhancers from chromatin state profiles&lt;br/&gt;4. Exploiting expression and chromatin co-dynamics with to predict enhancer-target gene links&lt;br/&gt;5. Jointly modeling sequence grammars at regulatory elements and their chromatin state dynamics, expression changes of regulators and functional interaction data to learn unified multi-scale gene regulation programs&lt;br/&gt;6. Elucidating the heterogeneity of chromatin architecture at regulatory elements&lt;br/&gt;7. Improving the detection and interpretation of potentially causal disease-associated variants from Genome-wide association studies&lt;br/&gt;More recently, we have also been developing methods to &lt;br/&gt;1. Decipher the functional heterogeneity of transcription factor binding&lt;br/&gt;2. Learn long-range, three-dimensional regulatory interactions&lt;br/&gt;3. Infer causal regulatory mechansisms by integrating diverse functional genomic data from temporal (e.g. differentiation/reprogramming) and perturbation (e.g. drug response, knockdown, genome-editing) experiments&lt;br/&gt;4. Model the complex relationships between genetic variation, regulatory chromatin variation and expression variation in healthy and diseased individuals&lt;br/&gt;5. Deep learning frameworks for genomics</p>\n",
      "\n",
      "[<p>Dr. Craig S. Levin is a Professor of Radiology and, by Courtesy, of Physics, Electrical Engineering, and Bioengineering at Stanford.  He is a founding member of the Molecular Imaging Program at Stanford, and Principal Investigator and Director of the NIH-NCI funded T32 Stanford Molecular Imaging Scholars postdoctoral training program. He received his M.S., M.Phil, and Ph.D. degrees in Physics from Yale University. An internationally recognized researcher in the field of molecular imaging he has nearly 200 peer-reviewed publications and 40 patents awarded or pending. He directs a 20-member laboratory that explores new concepts in instrumentation and software algorithms for molecular imaging, introduces these new tools into clinical and pre-clinical imaging studies of cancer, heart disease and neurological disorders, and partners with industry to disseminate some of these technologies into products used for patient care throughout the world.  To support his research, he has generated substantial NIH funding as Principal Investigator in addition to numerous grants from other government, industry, and private institutions. He lectured in a Nobel symposium in 2007, and was elected into the American Institute for Medical and Biological Engineering’s College of Fellows and was given the Academy of Radiology Research Distinguished Investigator Recognition Award.</p>]<p>Molecular Imaging Instrumentation Laboratory &lt;br/&gt;&lt;br/&gt;Our research interests involve the development of novel instrumentation and software algorithms for in vivo imaging of cellular and molecular signatures of disease in humans and small laboratory animal subjects. These new cameras efficiently image radiation emissions in the form of positrons, annihilation photons, gamma rays, and light from molecular probes developed to target molecular signals from deep within tissue of live subjects. The goals of the instrumentation projects are to push the sensitivity and spatial, spectral, and/or temporal resolutions as far as physically possible. The algorithm goals are to understand the physical system comprising the subject tissues, radiation transport, and imaging system, and to provide the best available image quality and quantitative accuracy. The work involves computer modeling, position sensitive sensors, readout electronics, data acquisition, image formation, image processing, and data/image analysis algorithms, and incorporating these innovations into practical imaging devices. The ultimate goal is to introduce these new imaging tools into studies of molecular mechanisms and treatments of disease within living subjects.</p>\n",
      "\n",
      "[<p>The world-wide COVID-19 corona virus pandemic has hijacked all our academic attention.  Please see Presentations.<br/><br/>Is it possible to understand the molecular structure and function of proteins and nucleic acids in enough detail to make accurate predictions about structure and function? We are mounting a two-pronged attack on this problem using both molecular dynamics simulation and molecular modeling. (i) Simulation attempts to reproduce the structural, thermodynamic and dynamic properties of a macromolecule in as accurate a way as possible. Starting with simple but realistic expressions for the interactions between atoms and classical laws of motion, we calculate a trajectory that specifies the position and velocity of every atom as a function of time. The time-step between calculated structures is small at 10-15 seconds, and we need to reduce hundreds of thousands of sets of atomic coordinates into a simple coherent description. We have simulated with reasonable fidelity the measurable static and dynamic properties of the several different proteins surrounded by thousands of water molecules. Simulation at different temperatures has allowed exploration of the pathways of protein denaturation of entire proteins and small fragments of protein secondary structure (alpha-helices and beta-hairpins). Companion studies of DNA double-helix segments in solution preserve the classical double helix while still showing a wide repertoire of interesting motions. (ii) Molecular modeling attempts to build a model of a macromolecule using known three-dimensional structures and energy minimization as complementary guidelines. Specific examples of this work include the automatic modeling of antibody variable domains, the general modeling of homologous proteins and studies of DNA base-pair mismatches. Questions we are trying to answer include: How can a protein be stabilized by a single amino acid change? How does the sequence of DNA cause local variations of double-helix conformation and stability? Extensive use is made of sophisticated programming, sequence and structural data bases, and computer graphics.</p>]<p>I pioneered of computational biology setting up the conceptual and theoretical framework for a field that I am still actively involved in at all levels.  More specifically, I still write and maintain computer programs of all types including large simulation packages and molecular graphics interfaces.  I have also developed a high-level of expertise in Perl scripting, as well as in the advanced use of the Office Suite of programs (Word, Excel and PowerPoint), which is more important and rare than it may seem.  My research focuses on three different but inter-related areas of research.  First, we are interested in predicting the folding of a polypeptide chain into a protein with a unique native-structure with particular emphasis on how the hydrophobic forces affect the pathway. We expect hydrophobic interactions to energetically favor structure that are more native-like.  In this way, the same stabilizing interactions that exist in the final folded state the search tractable.  Second we are interested in predicting protein structure from sequence without regard for the process of folding.  Such prediction relies on the well-established paradigms that similar protein sequences imply similar three-dimensional structures.  We have focused on the hardest problem in homology modeling: the refinement of a near-native structure to make it more precisely like the actual native structure of protein.  We have also focused on how the general similarity of all protein sequences resulting from their evolution from common ancestor sequence affects the nature of the protein universe.  Third, we are focusing on mesoscale modeling of large macromolecular complexes such as RNA polymerase and the mammalian chaperonin.  In this work, done in close collaboration with experimentalists, we use new morphing strategies combined with normal mode analysis in torsion angle space to overcome problems caused by the size and complexity of these critical, biomedically important systems.  All this work depends on the way a molecular structure is represented in terms of the force-field that allows calculation of the potential energy of the system.  We employ a very wide variety of such energy functions that extend from knowledge-based statistical potentials for a single interaction center per residue to quantum-mechanical force-fields that include inductive effects as well as polarization.</p>\n",
      "\n",
      "<p>Research in our laboratory is focused on the molecular mechanisms of calcium signaling through store-operated Ca2+ channels (SOCs).  This class of ion channels is regulated in a unique way, by the depletion of Ca2+ from the lumen of the endoplasmic reticulum (ER) which normally occurs following stimulation of cell surface receptors that generate IP3.  They are expressed in practically all cells, where they contribute to diverse functions including secretion, gene expression, and cell differentiation.  A SOC called the Ca2+ release-activated Ca2+ channel, or CRAC channel, is particularly important in T cells, where it generates sustained Ca2+ signals that are essential for triggering T cells to proliferate and carry out immune functions.  Loss of function of the CRAC channel by a single mutation in its structural gene leads to a devastating severe combined immunodeficiency (SCID) syndrome in humans.  &lt;br/&gt;&lt;br/&gt;A major effort in the lab is to understand how the depletion of Ca2+ from the ER triggers the opening of CRAC channels in the plasma membrane.  Since the recent discovery of genes encoding the ER Ca2+ sensor (STIM1) and the CRAC channel (Orai1), we have made rapid progress in revealing the choreographic nature of this process.  Following store depletion, STIM1 moves from locations throughout the ER to accumulate in ER subregions positioned within 10-25 nm of the plasma membrane (PM).  Simultaneously, the CRAC channel gathers at corresponding sites in the PM directly opposite STIM1, where it is opened through an interaction with STIM1.  This is an unprecedented mechanism for channel activation, in which a stimulus acts to bring a channel and its activator/sensor together for interaction across apposed membrane compartments.  We are currently studying how changes in ER Ca2+ drive the redistribution of STIM1 and the mechanism by which STIM1 and Orai1 interact across the ER-PM gap.  For these studies we combine protein engineering with patch-clamp electrophysiology and live-cell imaging using total internal reflection fluorescence (TIRF), confocal, and single-molecule microscopy.  &lt;br/&gt;&lt;br/&gt;A second major area of interest is how specific information is encoded in Ca2+ signals.  Specificity is an acute problem for pluripotent messengers like Ca2+ that are involved in multiple signaling pathways.  We have shown that transcriptional specificity in T cells can be achieved through the amplitude and dynamics of Ca2+ signals generated by CRAC channels.  We are now studying how these features contribute to cell fate decisions during T cell development.  In the thymus, self-reactive thymocytes are deleted through negative selection, while cells with the appropriate avidity for  self  are allowed to mature into T cells and populate the periphery (positive selection).  To study the role of Ca2+ in this choice, we have developed a novel thymic slice preparation in which we use two-photon microscopy to track and record Ca2+ signals in single thymocytes as they migrate through tissue engineered to provide defined selection signals.  We have found that positive selection is associated with Ca2+ oscillations, which immobilize the cells at locations of self-antigen recognition to promote gene activation.  We are currently comparing signaling during positive and negative selection to determine how the Ca2+ signal  signature  helps a T cell decide whether to live and prosper or die.</p>\n",
      "\n",
      "<p>The Li Lab is primarily interested in RNA editing mediated by ADAR enzymes. We co-discovered that the major function of RNA editing is to label endogenous dsRNAs as &amp;quot;self&amp;quot; to avoid being recognized as &amp;quot;non-self&amp;quot; by MDA5, a host innate immune dsRNA sensor, leading us to pursue therapeutic applications in cancer, autoimmune diseases, and viral infection. The other major direction of the lab is to develop technologies to harness endogenous ADAR enzymes for site-specific transcriptome engineering.</p>\n",
      "\n",
      "[<p>Dr. Li is an assistant professor in the Biochemistry Department and ChEM-H Institute at Stanford since 2015. Her lab works on understanding biochemical mechanisms of innate immunity and harnessing it to treat cancer. She majored in chemistry at University of Science and Technology of China and graduated with a B. En in 2003. She then trained with Dr. Laura Kiessling, a pioneer in chemical biology, at University of Wisconsin-Madison and graduated with a Ph.D in chemistry in 2010. She obtained her postdoctoral training with Dr. Timothy Mitchison at Harvard Medical School, who introduced her to the field of chemical immunology.</p>]\n",
      "\n",
      "\n",
      "\n",
      "[<p>Dr. Jonathan Long is an Assistant Professor of Pathology and an Institute Scholar of Stanford ChEM-H (Chemistry, Engineering &amp; Medicine for Human Health). Prior to arriving to Stanford in 2018, Dr. Long completed his Ph.D. in Chemistry at Scripps Research with Benjamin F. Cravatt and his postdoctoral work at Harvard Medical School/Dana-Farber Cancer Institute with Bruce M. Spiegelman. His contributions in the areas of lipid biochemistry and energy homeostasis have been recognized by numerous awards from the National Institutes of Health and the American Diabetes Association. <br/><br/>At Stanford, the Long laboratory studies signaling pathways in control mammalian energy metabolism. The lab focuses on metabolic hormones and other circulating hormone-like molecules. The long-term goal of this work is to discover new endocrine pathway of energy metabolism that can be translated into therapeutic opportunities for obesity, metabolic disease, and other age-associated chronic diseases.</p>]<p>Our laboratory focuses on the endocrine hormones and other circulating hormone-like molecules that regulate mammalian energy metabolism. With modern mass spectrometry, it is now recognized that blood plasma likely contains many more bioactive factors than previously recognized, secreted by cell types that were not previously considered to have endocrine functions. What are the identities of these molecules? What energy stressors do they respond to? Where are they made? What cell types or tissues do they act on? We use chemical biology and mass spectrometry-based technologies as discovery tools. We combine these tools with classical biochemical and genetic approaches in cell and animal models. Our goal is to uncover new endocrine pathways of organismal energy metabolism. Recent studies from our laboratory have identified a family of cold-regulated circulating lipids that stimulate mitochondrial respiration as well as an exercise-stimulated thermogenic polypeptide hormone. We suspect that many more remain to be discovered. We anticipate that our approach will uncover fundamental mechanisms that control mammalian energy homeostasis. In the long term, we hope to translate our discoveries into therapeutic opportunities that matter for metabolic and other age-associated chronic diseases.</p>\n",
      "\n",
      "[<p>Sharon Long received her undergraduate degree from Caltech, and carried out her PhD studies at Yale, working with Ian Sussex on plant development.   She was a postdoc with Fred Ausubel where she began study of rhizobia-legume symbioses.   She joined the Stanford faculty in 1982.</p>]<p>Biochemistry, genetics and cell biology of plant-bacterial symbiosis</p>\n",
      "\n",
      "[<p>Crystal L Mackall MD is the Ernest and Amelia Gallo Family Professor of Pediatrics and Internal Medicine at Stanford University.  She serves as Founding Director of the Stanford Center for Cancer Cell Therapy, Associate Director of Stanford Cancer Institute, Leader of the Cancer Immunology and Immunotherapy Program and Director of the Parker Institute for Cancer Immunotherapy at Stanford.  During a 27 year tenure culminating as Chief of the Pediatric Oncology Branch, NCI, and now through the Mackall Lab at Stanford (https://med.stanford.edu/mackalllab.html), she has led an internationally recognized basic and translational research program focused on immunooncology. Her work is credited with identifying an essential role for the thymus in human T cell regeneration and discovering IL-7 as the master regulator of T cell homeostasis.  She has conducted numerous early phase and first-in-human and first-in-child clinical trials spanning dendritic cell vaccines, cytokines, and adoptive immunotherapy using NK cells and genetically modified T cells. Her group was among the first to demonstrate impressive activity of CD19-CAR in pediatric leukemia, developed a novel CD22-CAR with impressive activity in leukemia and lymphoma refractory to CD19 targeting, and is leading exciting work focusing on CAR T cell therapy for brain tumors. Her group has identified T cell exhaustion as a major feature limiting the activity of CAR T cells and has developed novel approaches to prevent and reverse human T cell exhaustion. She is the recipient of numerous awards, including the AACR-St.Baldrick's Award for Outstanding Achievement in Pediatric Cancer Research, the ASCO Pediatric Oncology Award and Lecture, is a member of the American Society of Clinical Investigation, and the American Academy of Physicians. She serves in numerous national leadership positions, including co-PI on the NCI Pediatric Cancer Immunotherapy Network (U54), Leader of the NCI Pediatric Cancer Immunotherapy Trials Network, and co-Leader of the St. Baldrick’s-StandUp2Cancer Pediatric Dream Team.  She is Board Certified in Pediatrics, Pediatric Hematology-Oncology and Internal Medicine.</p>]<p>We are impressed by the potency of  immune responses for the treatment of cancer and our work focuses on enhancing the effectiveness of T cell based immunotherapies for cancer. Our approach is to simultaneously conduct basic studies alongside clinical trials, leveraging an iterative bench-to-bedside-bench rotation to efficiently optimize clinically relevant cancer immunotherapies.  Our laboratory, in parallel with the Center for Cancer Cell Therapy seeks to develop novel therapies for early phase testing in clinical trials, and conducts intensive studies on clinical samples obtained from patients treated on immunotherapy trials. We also seek to enhance fundamental understanding of human T cell biology.&lt;br/&gt;&lt;br/&gt;We focus primarily on using genetically engineered T cells to treat cancer, with an emphasis on chimeric antigen receptors (CARs). CARs are non-natural receptors, created using synthetic biology, that endow T cells with the capacity for antigen-specific, MHC-unrestricted killing.  Some clinical results using CAR based therapies have been impressive, but we believe that further progress will emerge as a result of focus on these three major areas:&lt;br/&gt;&lt;br/&gt;1.T cell exhaustion, a state whereby continued T cell activation leads to diminished functionality, is a fundamental barrier limiting the efficacy of many cancer immunotherapies.  Our laboratory is focused on using high dimensional, single cell analyses to better define human T cell exhaustion and to enhance understanding of the biological mechanisms responsible for this phenomenon.  We believe that enhanced biological understanding of T cell exhaustion will delivery novel approaches to prevent or reverse this phenomenon in the context of cancer immunotherapy.&lt;br/&gt;&lt;br/&gt;2.Effective immunotherapies require a therapeutic window whereby immune cell preferentially or exclusively attack the neoplastic cell while sparing non-neoplastic, vital tissues.  Our laboratory is focused on identifying novel targets for T cell based immunotherapies and on enhancing understanding of the quantitative thresholds for antigen recognition of these novel therapeutics.  Armed with this understanding, we seek to tune CAR T cells to differential antigen thresholds to enable safe recognition orfcancer cells while sparing normal tissues.  We are also interested in using novel approaches for combinatorial recognition, both to diminish the risk for tumor escape due to loss of antigen expression, and to allow targeting of tumor antigens that pose a risk due to co-expression on healthy, vital tissues.&lt;br/&gt;&lt;br/&gt;3.The tumor microenvironment is potently immunosuppressive and can prevent antigen specific immune responses from effectively mediating antitumor effects.  Our laboratory focuses on enhancing understanding of the immunosuppressive tumor microenvironment and on developing novel approaches to diminish the ability of the tumor microenvironment to limit the efficacy of T cell based immunotherapies.</p>\n",
      "\n",
      "<p>Ion transport across the hydrophobic barrier of the cell membrane is a primary challenge faced by all cells. Such transport sets up and exploits ion gradients, thus providing the basic energy and signaling events that are the foundation of life. My laboratory studies the molecular mechanisms of ion channels and transporters, the proteins that catalyze this transport. A major research focus is on the chloride-selective CLC family, which contains both types ion-transport protein. CLC proteins are expressed ubiquitously and perform diverse physiological functions in cardiovascular, neuronal, muscular, and epithelial function. We use a combination of biophysical methods to investigate membrane-protein structure and dynamics together with electrophysiological analyses to directly measure function. We additionally apply our expertise on ion-transport mechanisms to interdisciplinary collaborations, described below, to develop novel chemical tools and to understand the mechanism by which ultrasound modulates ion-transport to effect neuromodulation.</p>\n",
      "\n",
      "[<p>Theoretical chemist Todd Martínez develops and applies new methods that predict and explain how atoms move in molecules. These methods are used both to design new molecules and to understand the behavior of those that already exist. His research group studies the response of molecules to light (photochemistry) and external force (mechanochemistry). Photochemistry is a critical part of human vision, single-molecule spectroscopy, harnessing solar energy (either to make fuels or electricity), and even organic synthesis. Mechanochemistry represents a novel scheme to promote unusual reactions and potentially to create self-healing materials that resist degradation. The underlying tools embody the full gamut of quantum mechanical effects governing molecules, from chemical bond breaking/formation to electron/proton transfer and electronic excited states.<br/><br/>Professor Martínez was born in Amityville, New York, but spent most of his childhood in Central America and the Caribbean. His chemical curiosity benefitted tremendously from the relaxed safety standards in Central American chemical supply houses, giving him unfettered access to strong acids and bases. When he also became interested in computation, limited or nonexistent computer access forced him to write and debug computer programs on paper. Today, Prof. Martínez combines these interests by working toward theoretical and computational modeling and design of molecules. Martínez received his PhD in chemistry from UCLA in 1994. After postdoctoral study at UCLA and the Hebrew University in Jerusalem, he joined the faculty at the University of Illinois in 1996. In 2009, he joined the faculty at Stanford, where he is now the Ehrsam and Franklin Professor of Chemistry and Professor of Photon Science at SLAC National Accelerator Laboratory. He has received numerous awards for his contributions, including a MacArthur Fellowship (commonly known as the “genius award”). He is co-editor of Annual Reviews in Physical Chemistry, associate editor of The Journal of Chemical Physics, and an elected fellow of the American Academy of Arts and Sciences.<br/><br/>Current research in the Martínez lab aims to make molecular modeling both predictive and routine. New approaches to interactive molecular simulation are being developed, in which users interact with a virtual-reality based molecular modeling kit that fully understands quantum mechanics. New techniques to discover heretofore unknown chemical reactions are being developed and tested, exploiting the many efficient methods that the Martínez group has introduced for solving quantum mechanical problems quickly, using a combination of physical/chemical insights and commodity videogaming hardware. For more details, please visit http://mtzweb.stanford.edu.</p>]\n",
      "\n",
      "<p>I seek to understand how mammalian cells process information and make decisions. This is a fundamental open question as cells are controlled by multiple signaling pathways with tens of signaling proteins, second messengers and chromatin modifiers connected to each other on time-scales of seconds to days by positive and negative feedbacks. Understanding how signaling circuits control cell proliferation, migration and other outputs is important for identifying optimal drug targets and to facilitate the development of combination therapies. Much of my work is built on the premise that genetic and biochemical methods can be used to identify and characterize components of signaling circuits, but that single-cell microscopy, live-cell signaling reporters, and rapid perturbations are needed to understand the design principles of signaling circuits. My laboratory has pioneered the development and use of molecular tools and quantitative microscopy methods to understand feedback-connected signaling circuits and made key contributions to our understanding of the spatial and temporal control of calcium, lipid second messenger, small GTPase, and protein kinase signaling processes. Our current research identifies general control principles and specific mechanisms how cells integrate receptor, cell contact and stress inputs to decide between quiescence, proliferation and senescence, how they switch metabolic states, and how they trigger polarization and decide to move. We are investigating these signaling circuits by combining high-resolution live-cell analysis of signal transduction and local chromatin activity with optogenetic perturbations, single-cell RNAseq and computational modeling.</p>\n",
      "\n",
      "[<p>W. E. (William Esco) Moerner, the Harry S. Mosher Professor of Chemistry and Professor by courtesy of Applied Physics, has conducted research in physical chemistry, biophysics, and the optical properties of single molecules, and is actively involved in the development of 2D and 3D super-resolution imaging for cell biology. Imaging studies include protein superstructures in bacteria, structure of proteins in cells, studies of chromatin organization, and dynamics of regulatory proteins in the primary cilium. Using powerful microscopes optimized for tracking of single objects in cells, the motions of proteins, DNA, and RNA are being measured in three dimensions in real time to understand processing and binding interactions. A related research area concerns precise analysis of photodynamics of single trapped biomolecules in solution, with applications to photosynthesis, protein-protein interactions, and transport measurements.<br/><br/>Born on June 24, 1953 at Parks Air Force Base in Pleasanton, California, Professor Moerner was raised in San Antonio, Texas. He attended Washington University as a Langsdorf Engineering Fellow, graduating in 1975 with degrees in Physics and Electrical Engineering (both B.S. with top honors), and Mathematics (A.B. summa cum laude). His doctoral research in physics at Cornell University (M.S. 1978, Ph.D. 1982) employed tunable infrared lasers to explore infrared vibrational modes of impurities in crystals. In 1982, he moved from New York to San Jose, California to join the IBM Research Division developing spectral holeburning for frequency domain optical storage and photorefractivity for dynamic hologram formation. After 13 years at IBM, Dr. Moerner accepted a position as Distinguished Professor of Physical Chemistry at UC San Diego, where he broadened his research to include biological systems and biophysics. Recruited to the Stanford Chemistry Department faculty in 1997, he served as Chair of the department from 2011 to 2014.<br/><br/>Professor Moerner’s scientific contributions were recognized with the 2014 Nobel Prize in Chemistry \"for the development of super-resolved fluorescence microscopy.\" One method to surpass the optical diffraction limit (PALM/STORM) uses single-molecule imaging combined with a control mechanism to keep the concentration of emitting molecules at a very low level, followed by sequential localization to reconstruct the underlying structure. The fundamentals of this idea came from early work in the Moerner lab: optical detection and imaging of single molecules (1989) combined with blinking and switching at low temperature, as well as the discovery of optical control of single copies of green fluorescent protein at room temperature (1997). Among many other honors and awards, Professor Moerner was elected fellow of the American Physical Society, Optical Society of America, American Association for the Advancement of Science, American Academy of Arts and Sciences; and member of the National Academy of Sciences.<br/><br/>Today, the Moerner Laboratory uses laser spectroscopy and microscopy of single molecules to probe biological processes, one molecule at a time. Primary thrusts include development and application of fluorescence microscopy far beyond the optical diffraction limit by PALM/STORM and STED approaches, single-molecule tracking in complex cellular environments, invention and validation of methods for precise and accurate 3D optical microscopy in cells, and trapping of single photosynthetic biomolecules in solution for extended study. Through a variety of collaborations, these approaches are applied to explore protein and oligonucleotide localization patterns in bacteria, measure structures of amyloid aggregates in cells, define the behavior of signaling proteins in the primary cilium, quantify photodynamics for photosynthetic proteins and enzymes, and observe the dynamics of DNA and RNA in cells and viruses.<br/><br/>Please visit the Moerner Lab home page for more information.</p>]<p>Most biophysical or chemical experiments in condensed matter measure the average behavior of a huge number, N, of molecules, where N may range from millions to billions to Avogadro's Number. At the same time, most theoretical models are intended to describe the behavior of a single molecule interacting with its surroundings, and averaging over the number of molecules N is normally required to compute an observable. Using precision laser spectroscopic techniques, we have been detecting and probing the detailed properties of individual impurity molecules hidden deep inside a cell, in a protein, or even in a liquid, i.e., the ultimate limit of N=1. This was first done in the Moerner Lab in 1989, and has since expanded dramatically to include many groups around the world. A key reason for doing this is to explore heterogeneity that is normally obscured by ensemble averaging. &lt;br/&gt;&lt;br/&gt;Studying one individual molecule in a solid means we are working with an extremely small number of moles of material. You might be aware that the international standards organization, IUPAC, has defined several new prefixes: zepto- for 1E-21, and yocto- for 1E-24. Thus 1 molecule is equivalent to 1.66 yoctomoles. But we think this is unwieldy. Thus we define a new prefix guaca- so that (with apologies to Prof. Avogadro) &lt;br/&gt;&lt;br/&gt;1 guacamole = 1 / ( Avocado's Number) of moles. &lt;br/&gt;&lt;br/&gt;More seriously, it is worth recalling that each molecule we are probing is only 1 or 2 nanometers in size. This means that when we use a laser to select one probe molecule, we can sense details of the immediate local environment of a truly nanoscopic probe. &lt;br/&gt;&lt;br/&gt;To achieve this extreme reduction of the concentration and reach the single-molecule level, we use either (a) extremely low concentrations and diffraction-limited confocal, TIRF, or far-field microscopy, or (b) near field optical excitation to pump sample volumes much smaller than the diffraction limit, or (c) superresolution imaging by single-molecule active control. By studying a large number of individual molecules one at a time, we are able not only to observe how the usual ensemble average behavior is formed, but also to see unexpected, surprising behavior normally hidden by the usual ensemble averaging. &lt;br/&gt;&lt;br/&gt;The phenomena under study include protein localization patternd in bacteria, chaperonin proteins, and new fluorophores for active-control superresolution imaging.  By dispersing the emitted light, even the vibrational mode spectrum of a single molecule may be measured! By measuring correlations in the emitted photon stream, fast dynamics including environmental fluctuations, or the purely quantum-mechanical behavior termed photon antibunching may be probed. In biomolecules, we observe fascinating differences in behavior due to conformational states, local environments, or enzymatic cycle, all of which are obscured in large N experiments.&lt;br/&gt;&lt;br/&gt;Importantly, a single molecule can be viewed as a probe of its immediate local nanoenvironment on the scale on the order of the molecular size (~1 nm). Because single molecules are nanoscale emitters, when active control is used to turn molecules on and off, it is possible to build up a super-resolution image of the sample, far beyond the optical diffraction limit, typically on the 40 nm scale.  Several advanced optical techniques for obtaining thee-dimensional information from single-molecule photoswitching are underdevelopment, and we apply these methods to imaging a variety of cellular structures in bacteria and in mammalian cells and to tracking of RNA in living yeast.</p>\n",
      "\n",
      "<p>Our research interests are to elucidate the contribution of chromatin to mechanisms that promote genomic integrity.  The regulation of chromatin is a crucial component of DNA metabolism and processing in eukaryotic organisms.  Chromatin-remodeling complexes, modified histones, and higher order chromatin structure are all factors influencing genome stability.  We utilize an integrated approach of genetic, biochemical, and molecular techniques, in both yeast and mammalian systems, to examine the involvement of chromatin in processes that prevent genome instability and the pathogenesis of disease.</p>\n",
      "\n",
      "[<p>Dr. Palacios seek to provide statistically rigorous answers to concrete, data driven questions in evolutionary genetics and public health . My research involves probabilistic modeling of evolutionary forces and the development of computationally tractable methods that are applicable to big data problems. Past and current research relies heavily on the theory of stochastic processes, Bayesian nonparametrics and recent developments in machine learning and statistical theory for big data.</p>]\n",
      "\n",
      "[<p>Vijay Pande, Henry Dreyfus Professor of Chemistry and, by courtesy, of Structural Biology and Computer Science, also currently directs of the Stanford Program in Biophysics and the Folding@home Distribtued Computing project. His research centers on novel cloud computing simulation techniques to address problems in chemical biology. In particular, he has pioneered distributed computing methodology to break fundamental barriers in the simulation of protein and nucleic acid kinetics and thermodynamics. As director of the Folding@home project (http://folding.stanford.edu), Prof. Pande has, for the first time, directly simulated protein folding dynamics, making quantitative comparisons with experimental results, often considered a “holy grail” of computational biology. His current research also includes novel computational methods for drug design, especially in the area of protein misfolding and associated diseases such as Alzheimer’s and Huntington’s Disease.<br/><br/>Professor Pande studied physics at Princeton University (B.A. 1992), where he was first introduced to biophysical questions, especially in undergraduate research with Nobel Laureate P. Anderson. His doctoral research in physics under Profs. T. Tanaka and A. Grosberg at MIT (Ph.D. 1995) centered on statistical mechanical models of protein folding, suggesting new ways to design protein sequences for stability and folding properties. As a Miller Fellow under Prof. D. Rokhsar at UC Berkeley, Prof. Pande extended this methodology to examine atomistic protein models, laying the foundations for his work at Stanford University. Among numerous awards, Prof. Pande has received the Biophysical Society’s Bárány Award for Young Investigators and Protein Society’s Irving Sigal Young Investigator Award, and was named to MIT’s TR100 and elected a Fellow of the American Physical Society.<br/><br/>The Pande research group develops and applies new theoretical methods to understand the physical properties of biological molecules such as proteins, nucleic acids and lipid membranes, using this understanding to design synthetic systems including small-molecule therapeutics. In particular, the group examines the self-assembly properties of biomolecules. For example, how do protein and RNA molecules fold? How do proteins misfold and aggregate? How can we use this understanding to tackle misfolding related degeneration and develop small molecules to inhibit disease processes?<br/><br/>As these phenomena are complex, spanning molecular to mesoscopic lengths and nanosecond to millisecond timescales, the lab employs a variety of methods, including statistical mechanical analytic models, Markov State Models, and statistical and informatic methods. Other tools include Monte Carlo, Langevin dynamics, and molecular dynamics computer simulations on workstations and massively parallel supercomputers, superclusters, and worldwide distributed computing. The group has also done extensive work in the application of machine learning, pioneering traditional and deep learning approaches to cheminformatics, biophysics and drug design.<br/><br/>For example, simulations in all-atom detail on experimentally relevant timescales (milliseconds to seconds) have produced specific predictions of the structural and physical chemical nature of protein aggregation involved in Alzheimer’s and Huntington’s diseases. These results have fed into computational small molecule drug design methods, yielding interesting new chemical entities.<br/><br/>Since such problems are extremely computationally demanding, the group developed a distributed computing project for protein folding dynamics. Since its launch in October 2000, Folding@Home has attracted more than 4,000,000 PCs, and today is recognized as the most powerful supercluster in the world. Such enormous computational resources have allowed simulations of unprecedented folding timescales and statistical precision and accuracy. For more details, please visit http://pande.stanford.edu.</p>]\n",
      "\n",
      "<p>Evolution of genomes and population genomics of adaptation and variation</p>\n",
      "\n",
      "[<p>We use interdisciplinary approaches including theory and experiments to understand how computation is embodied in biological matter. Examples include cognition in single cell protists and morphological computing in animals with no neurons and origins of complex behavior in multi-cellular systems. Broadly, we invent new tools for studying non-model organisms with significant focus on life in the ocean - addressing fundamental questions such as how do cells sense pressure or gravity? Finally, we are dedicated towards inventing and distributing “frugal science” tools to democratize access to science (previous inventions used worldwide: Foldscope, Abuzz), diagnostics of deadly diseases like malaria and convening global citizen science communities to tackle planetary scale environmental challenges such as mosquito surveillance or plankton surveillance by citizen sailors mapping the ocean in the age of Anthropocene.</p>]\n",
      "\n",
      "<p>The Puglisi group investigates the role of RNA in cellular processes and disease.  Our goal is to understand RNA function in terms of molecular structure and dynamics using a variety of biophysical and biological tools.  We use nuclear magnetic resonance (NMR) spectroscopy to determine structures of biological molecules, and integrate structural understanding into further mechanistic and functional studies.  We investigate dynamics using single-molecule approaches.  Our goal is a unified picture of structure, dynamics and function. We are currently focused on the mechanism and regulation of translation, and the role of RNA in viral infections.  A long-term goal is to target processes involving RNA with novel therapeutic strategies.</p>\n",
      "\n",
      "[<p>Stephen Quake is the Lee Otterson Professor of Bioengineering and Professor of Applied Physics at Stanford University and is co-President of the Chan Zuckerberg Biohub. He received a B.S. in Physics and M.S. in Mathematics from Stanford University in 1991 and a doctorate in Theoretical Physics from the University of Oxford in 1994. Quake has invented many measurement tools for biology, including new DNA sequencing technologies that have enabled rapid analysis of the human genome and microfluidic automation that allows scientists to efficiently isolate individual cells and decipher their genetic code. Quake is also well known for inventing new diagnostic tools, including the first non-invasive prenatal test for Down syndrome and other aneuploidies. His test is rapidly replacing risky invasive approaches such as amniocentesis, and millions of women each year now benefit from this approach. His innovations have helped to radically accelerate the pace of biology and have made medicine safer by replacing invasive biopsies with simple blood tests.</p>]<p>Single molecule biophysics, precision force measurement, micro and nano fabrication with soft materials, integrated microfluidics and large scale biological automation.</p>\n",
      "\n",
      "<p>Probe chemistry and nanotechnology for molecular imaging and diagnostics</p>\n",
      "\n",
      "<p>Cardiovascular developmental biology</p>\n",
      "\n",
      "<p>Cancer is an evolutionary process that typically spans multiple decades before it causes symptoms. The survival probability of patients with a tumor diagnosed early is five to ten times higher than when diagnosed at an advanced stage. The last stage of cancer progression, metastasis, is responsible for 90% of cancer-related deaths. In a tumor with billions of cells virtually any point mutation is expected to be present in a few cells. Hence, at a genetic level, not only is every cancer type different, but also every tumor of the same type and every cell of the same tumor are different. This enormous heterogeneity poses a major barrier to drug development and long-term disease control but also represents a unique opportunity to study the evolutionary principles that govern cancer initiation and progression.&lt;br/&gt;&lt;br/&gt;My research focuses on the stochastic biological processes underlying cancer evolution, in particular those related to the initiation, progression, spread, and treatment of cancer. The goal of this research is to accurately diagnose aggressive cancers as well as to provide new insights about metastatic spread and the development of resistance against therapies. I develop computational methods and design mathematical models to generate novel hypotheses and explain observations on a mechanistic level in close collaboration with many physician‐scientists.</p>\n",
      "\n",
      "[<p>Grant Rotskoff studies the nonequilibrium dynamics of living matter with a particular focus on self-organization from the molecular to the cellular scale. His work involves developing theoretical and computational tools that can probe and predict the properties of physical systems driven away from equilibrium. Recently, he has focused on characterizing and designing physically accurate machine learning techniques for biophysical modeling. Prior to his current position, Grant was a James S. McDonnell Fellow working at the Courant Institute of Mathematical Sciences at New York University. He completed his Ph.D. at the University of California, Berkeley in the Biophysics graduate group supported by an NSF Graduate Research Fellowship. His thesis, which was advised by Phillip Geissler and Gavin Crooks, developed theoretical tools for understanding nonequilibrium control of the small, fluctuating systems, such as those encountered in molecular biophysics. He also worked on coarsegrained models of the hydrophobic effect and self-assembly. Grant received an S.B. in Mathematics from the University of Chicago, where he became interested in biophysics as an undergraduate while working on free energy methods for large-scale molecular dynamics simulations.<br/><br/>Research Summary<br/><br/>My research focuses on theoretical and computational approaches to \"mesoscale\" biophysics. Many of the cellular phenomena that we consider the hallmarks of living systems occur at the scale of hundreds or thousands of proteins. Processes like the self-assembly of organelle-sized structures, the dynamics of cell division, and the transduction of signals from the environment to the machinery of the cell are not macroscopic phenomena—they are the result of a fluctuating, nonequilibrium dynamics. Experimentally probing mesoscale systems remains extremely difficult, though it is continuing to benefit from advances in cryo-electron microscopy and super-resolution imaging, among many other techniques. Predictive and explanatory models that resolve the essential physics at these intermediate scales have the power to both aid and enrich the understanding we are presently deriving from these experimental developments.<br/><br/>Major parts of my research include:<br/><br/>1. Dynamics of mesoscale biophysical assembly and response.— Biophysical processes involve chemical gradients and time-dependent external signals. These inherently nonequilibrium stimuli drive supermolecular organization within the cell. We develop models of active assembly processes and protein-membrane interactions as a foundation for the broad goal of characterizing the properties of nonequilibrium biomaterials.<br/><br/>2. Machine learning and dimensionality reduction for physical models.— Machine learning techniques are rapidly becoming a central statistical tool in all domains of scientific research. We apply machine learning techniques to sampling problems that arise in computational chemistry and develop approaches for systematically coarse-graining physical models. Recently, we have also been exploring reinforcement learning in the context of nonequilibrium control problems.<br/><br/>3. Methods for nonequilibrium simulation, optimization, and control.— We lack well-established theoretical frameworks for describing nonequilibrium states, even seemingly simple situations in which there are chemical or thermal gradients. Additionally, there are limited tools for predicting the response of nonequilibrium systems to external perturbations, even when the perturbations are small. Both of these problems pose key technical challenges for a theory of active biomaterials. We work on optimal control, nonequilibrium statistical mechanics, and simulation methodology, with a particular interest in developing techniques for importance sampling configurations from nonequilibrium ensembles.</p>]\n",
      "\n",
      "[<p>I am a computational neuroscientist and currently focus on understanding brain dynamics at rest as well as during learning. The overarching goal of my research is to develop reliable computational methods that will allow for characterizing and modeling temporal dynamics of brain activity, without averaging data in either space or time. I firmly believe that the spatiotemporal richness in brain activity might hold the key to finding the person- and disorder-centric biomarkers. Funded by a career development award (K99/R00; NIMH) and a young investigator award (NARSAD; Brain &amp; Behavior Foundation), I am currently developing methods to model the temporal dynamics of brain activity in individuals with fragile X syndrome and healthy controls. The application of computational modeling to neuroscience and psychiatry is nascent in its development but holds significant promise to affect public health positively. I have a strong interdisciplinary background in (1) computational sciences, (2) neuroscience as well as (3) psychiatry. Integrating neuroscience, psychiatry, and mathematical modeling represents the new frontier in applications and analysis of large neuroimaging datasets and has the potential to revolutionize our understanding of dynamical brain organization in healthy controls and individuals with psychiatric disorders.</p>]<p>I am a computational neuroscientist and currently focus on understanding brain dynamics at rest as well as during learning. The overarching goal of my research is to develop reliable computational methods that will allow for characterizing and modeling temporal dynamics of brain activity, without averaging data in either space or time. I strongly believe that the spatiotemporal richness in brain activity might hold the key to finding the person- and disorder-centric biomarkers. Funded by a career development award (K99/R00; NIMH) and a young investigator award (NARSAD; Brain &amp;amp; Behavior Foundation), I am currently developing methods to model the temporal dynamics of brain activity in individuals with fragile X syndrome and healthy controls. The application of computational modeling to neuroscience and psychiatry is nascent in its development but holds significant promise to positively affect public health. I have a strong interdisciplinary background in (1) computational sciences, (2) neuroscience as well as (3) psychiatry. Integrating neuroscience, psychiatry, and mathematical modeling represents the new frontier in applications and analysis of large neuroimaging datasets and has the potential to revolutionize our understanding of dynamical brain organization in healthy controls and in individuals with psychiatric disorders.</p>\n",
      "\n",
      "[<p>The Salzman lab is primarily a computational data science laboratory focused on studying RNA biology with a secondary focus on validation and novel experimental design to enable statistical analysis of RNA expression. We develop new data science methods to analyze the mechanism and function gene expression and RNA splicing. We design new methodology aiming to uncover how splicing is regulated at a single cell level and is dysregulated in diseases from viral infections to cancer.  We have active work in the area of analysis of massive single-cell sequencing projects such as Tabula Sapiens, spatial genomics and analysis of host response to SARS-CoV-2.</p>]<p>Circular RNA regulation and function; computational and experimental approaches</p>\n",
      "\n",
      "\n",
      "\n",
      "<p>The long-term goal of our research is to advance experimental paradigms for understanding normal cognitive and disease processes at the level of neural circuits, with emphasis on learning and memory processes. By contrast, much current research on learning and memory concentrates on levels of organization in the nervous system that are either more macroscopic (e.g. in cognitive psychology) or more microscopic (e.g. in synaptic physiology).&lt;br/&gt;&lt;br/&gt;Our approach combines behavioral, electrophysiological, and computational methodologies with high-resolution fluorescence optical imaging that is capable of resolving individual neurons and dendrites. By necessity, we aim to advance imaging methods so that we can examine dynamics of neuronal populations or of dendritic compartments in behaving animals. En route, we are also performing experiments on circuit properties in anesthetized animals, such as the studies that use our newly invented fluorescence endoscopes for examining hippocampal cells and dendrites in vivo.&lt;br/&gt;&lt;br/&gt;We seek explanations that span different levels of organization, from cells to entire circuits. We work with both genetic model organisms, mice and fruit flies, and human subjects. Our research emphasizes understanding the control and learning of motor behaviors, as well as the potential application of our newly developed imaging techniques to clinical use in humans.</p>\n",
      "\n",
      "[<p>Naima Sharaf got her undergraduate degree in Chemistry at the University of North Carolina-Chapel Hill. She carried out her Ph.D. studies at the University of Pittsburgh in the lab of Dr. Angela Gronenborn where she used fluorine solution NMR to understand inhibitor-induced conformational changes with HIV-1 reverse transcriptase. To expand her structural biology skill set, she undertook postdoctoral training at Caltech in the lab of Dr. Doug Rees where she characterized the structure and function of the Neisseria meningitides methionine ABC transport system using x-ray crystallography and single-particle cryo-EM. This research sparked Dr. Sharaf's current interest in lipoproteins, particularly their roles in bacterial physiology and potential in vaccine design.  Research in the Sharaf Lab bridges biochemistry, biology, microbiology, and immunology to translate lipoprotein research into therapeutics.</p>]\n",
      "\n",
      "[<p>My interdisciplinary research group draws on diverse scientific cultures to develop a creative, rigorous and quantitative approach to the fundamental question of how growth drives cell division. Our diverse backgrounds include mathematics, physics, engineering, biochemistry, genetics, and cell, molecular, and systems biology. This reflects my interdisciplinary training (BS Mathematics; BS Physics - MIT 1999; PhD Applied Mathematics - Cambridge 2004; Postdoctoral training Genetics, Cell, and Systems Biology - Rockefeller)</p>]<p>My laboratory’s goal is to understand how cell growth triggers cell division. Linking growth to division is important because it allows cells to maintain a specific size range to best perform their physiological functions. Today, thanks to decades of research, we have an extensive, likely nearly complete parts-list of key regulatory proteins. Deletion, inhibition, or over-expression of these proteins often results in changes to cell size. However, the underlying molecular mechanisms for how growth triggers division are not understood.  How do the regulatory proteins work together to produce a biochemical activity reflecting cell size or growth? Since we now have most of the parts, the next step to solving this fundamental question is to better understand how they work together.&lt;br/&gt; &lt;br/&gt;My laboratory recently made a breakthrough discovery in understanding how growth triggers division in budding yeast. While it was expected that growth would act to increase the activities of the cyclin-dependent kinases (Cdk) known to promote cell division, this is not the case. Rather, we found that cell growth acts in the opposite manner. Cell growth triggers division by diluting a protein that inhibits cell division. We recently discovered an analogous mechanism operating in human cells. &lt;br/&gt; &lt;br/&gt;Our discovery of a mechanism linking cell growth to cell division in budding yeast opens many avenues of research, three of which we are currently pursuing:&lt;br/&gt; &lt;br/&gt;1.     Cell size control results from the dilution of the cell cycle inhibitor Whi5 because its synthesis is independent of cell size. In contrast, most proteins are produced in proportion to cell size. We identified the set of proteins whose expression is largely independent of cell size. We now aim to determine the molecular mechanism(s) through which this occurs and identify the biological processes impacted.&lt;br/&gt;2.    We are addressing how gene expression depends on cell size in human cells. We are working with the Chan Zuckerberg Biohub Cell Atlas Project to establish a workflow so that all their single cell sequencing experiments will include data on cell size. This will allow us to examine cell size dependency of gene expression across an unprecedented number of human cell types.&lt;br/&gt;3.    Our work in yeast led us to the hypothesis that cell growth could trigger division in human cells by diluting a cell cycle inhibitor. We can apply our quantitative single-cell imaging approach because CRISPR-based genome editing allows us to tag cell cycle regulators with fluorescent proteins at their endogenous loci. We are now measuring and manipulating concentration dynamics in live cells to determine how cell growth impacts key regulators of division.&lt;br/&gt; &lt;br/&gt;Our work has fundamental implications for understanding how the most basic aspect of cell morphology, cell size, is controlled. In the next 5 years, we aim to determine how growth triggers division in human cells, which has the potential to revolutionize our understanding of how cell division is regulated in both natural developmental contexts and in disease. Over the 5-10 year time horizon, we intend to pursue both developmental and medical directions.</p>\n",
      "\n",
      "[<p>Professor Edward Solomon’s research spans the fields of physical-inorganic, bioinorganic, and theoretical-inorganic chemistry. His work focuses on spectroscopic elucidation of the electronic structure of transition metal complexes and its contribution to reactivity. He has developed new spectroscopic and electronic structure methods and applied these to active sites in catalysis. He has made significant contributions to our understanding of metal sites involved in electron and oxo transfer, copper sites involved in O2 binding, activation and reduction to water, in structure/function correlations over non-heme iron enzymes, and in the correlation of biological to heterogeneous catalysis.<br/><br/>Edward I. Solomon grew up in North Miami Beach, Florida, received his Ph.D. at Princeton (1972) and was a postdoctoral fellow at The Ørsted Institute in Denmark and at Caltech. He started his career at MIT in late 1975, became a full professor in 1981, and joined the faculty at Stanford in 1982 where he is now the Monroe E. Spaght Professor of Humanities and Sciences and Professor of Photon Science at SLAC National Accelerator Laboratory. He has been a visiting professor in France, Argentina, Japan, China, India, Australia and Brazil. He has received ACS National Awards in Inorganic Chemistry, Distinguished Service in the Advancement of Inorganic Chemistry, the Alfred Bader Award in Bioinorganic or Bioorganic Chemistry, the Ira Remsen Award, and the Kosolapoff Award, the Centenary Medal from the Royal Society of Chemistry (UK), the Wheland Medal from the University of Chicago, the Bailar Medal from the University of Illinois, the Frontiers in Biological Chemistry Award from the Max-Planck- Institute (Mülheim), the Chakravorty Award from the Chemical Research Society of India and the Dean’s Award for Distinguished Teaching at Stanford among others. He is a member of the National Academy of Sciences, the American Academy of Arts and Sciences and a Fellow in American Association for the Advancement of Science and in the American Chemical Society.<br/><br/>The Solomon lab uses both experimental and theoretical techniques to define the electronic and geometric structures of biologically- and catalytically-relevant transition metal sites, with the goal of applying insights into electronic structure to obtain a detailed understanding of reactivity and function. This research utilizes a wide range of spectroscopic, theoretical, and chemical techniques to probe structure/function relationships, gain mechanistic insight, and address fundamental questions of relevance to chemistry and biology. The systems under study can be divided into five general areas:<br/><br/>– Electron Transfer Sites<br/>– Copper Active Sites in Biology<br/>– Mononuclear Non-Heme Iron Enzymes: Structure/Function Correlation<br/>– Binuclear Non-Heme Iron Enzymes: Dioxygen Binding and Activation<br/>– Correlations from Biological to Heterogenous Catalysis</p>]<p>Professor Solomon’s research spans the fields of physical-inorganic and bioinorganic chemistry, emphasizing the application of a wide variety of spectroscopic and computational methods to determine the electronic structure of transition metal complexes. Research is directed toward both high symmetry small molecule complexes to define in detail electronic structure contributions to chemical and physical properties, and metal ion active sites in catalysis to understand their unusual spectral features in terms of electronic and geometric structure and to evaluate these structural contributions to reactivity. Many studies focus on fundamental problems in bioinorganic chemistry. Areas of present interest include: 1) Electronic structure contributions to electron transfer in copper, iron-sulfur and heme sites; 2) O2 binding, activation, and reduction by Cu cluster active sites; 3) Structure/function correlations over non-heme iron enzymes; 4) Development of new spectroscopic and electronic structure methods in bioinorganic chemistry; and 5) Correlation of biological to heterogeneous catalysis.</p>\n",
      "\n",
      "[<p>Theory and Computation of Biological Processes and Complex Materials<br/><br/>The Spakowitz lab is engaged in projects that address fundamental chemical and physical phenomena underlying a range of biological processes and soft-material applications. Current research in our lab focuses on four main research themes: chromosomal organization and dynamics, protein self-assembly, polymer membranes, and charge transport in conducting polymers. These broad research areas offer complementary perspectives on chemical and physical processes, and we leverage this complementarity throughout our research. Our approach draws from a diverse range of theoretical and computational methods, including analytical theory of semiflexible polymers, polymer field theory, continuum elastic mechanics, Brownian dynamics simulation, equilibrium and dynamic Monte Carlo simulations, and analytical theory and numerical simulations of reaction-diffusion phenomena. A common thread in our work is the need to capture phenomena over many length and time scales, and our flexibility in research methodologies provides us with the critical tools to address these complex multidisciplinary problems.</p>]<p>Theory and computation of biological processes and complex materials</p>\n",
      "\n",
      "[<p>James Spudich, Douglass M. and Nola Leishman Professor of Cardiovascular Disease, is in the Department of Biochemistry at Stanford University School of Medicine.  He received his B.S. in chemistry from the University of Illinois in 1963 and his Ph.D. in biochemistry from Stanford in 1968.  He did postdoctoral work in genetics at Stanford and in structural biology at the MRC Laboratory in Cambridge, England.  From 1971 to 1977 he was Assistant, Associate, and Full Professor in the Department of Biochemistry and Biophysics, University of California, San Francisco.  In 1977 he was appointed Professor in the Department of Structural Biology at Stanford University.  Spudich served as Chairman of the Department of Structural Biology from 1979-1984.  Since 1992 he has been Professor in the Department of Biochemistry, and served as Chairman from 1994-1998.  He has held a joint appointment as Professor in the Department of Developmental Biology since 1989.  From 1998 to 2002, he was Co-Founder and first Director of the Stanford Interdisciplinary Program in Bioengineering, Biomedicine and Biosciences called Bio-X. At present he is also an Adjunct Professor at the National Center for Biological Sciences, Tata Institute of Fundamental Research and InStem in Bangalore, India. <br/><br/>Spudich has given more than 40 named lectureships and keynote addresses, including the First Annual Lecture of the series \"The James Spudich AHA Research Committee Lecture,” named in his honor; the Pauling Lecture, Stanford; the Paul Dudley White Lecture, Mass General Hospital of Harvard University; the DeWitt Stetten, Jr. Lecture, NIH; the Meyerhof Lecture, Heidelberg; the Keith R. Porter Lecture, ASCB; the Hans Neurath Lecture, University of Washington; the National Lecture, Biophysical Society; the Mayer Lecture, MIT; Plenary Lecture (shared with Aaron Klug), Madrid International Congress on Cell Biology; the Friday Evening Lecture, Woods Hole; and the Cori Lecture, Washington University.<br/><br/>Spudich was a recipient of a Guggenheim Fellowship in 1978.  He was elected to the National Academy of Sciences in 1991.  He is also a member of the American Academy of Arts and Sciences, and the American Association for the Advancement of Science.  Spudich received the American Heart Association Basic Research Prize, the Alexander von Humboldt Research Award, the Biophysical Society Lifetime Research Career Award, the Lewis S. Rosenstiel Award for Outstanding Research Achievement in the Field of Basic Medical Studies, the American Chemical Society’s Award for the Chemistry of Biological Processes, the Biophysics Society Award for Outstanding Investigator in the Field of Single Molecule Biology, the E.B. Wilson Medal, the Arthur Kornberg and Paul Berg Lifetime Achievement Award in Biomedical Sciences, the Wiley Prize in Biomedical Sciences, the Ahmed H. Zewail Award and the Massry Prize. In 2012, he received the Albert Lasker Basic Medical Research Award.</p>]<p>Our general research interest is the structure and function of molecular motors in vitro and in vivo, with emphasis on understanding the molecular basis of muscle contraction.  Our major areas of specific interest are the molecular basis of energy transduction that leads to ATP-driven myosin movement on actin, the roles of the myosin family of molecular motors in eukaryotic cells, the regulation of actin and myosin interaction and their assembly states, and the biochemistry and regulation of the attachment of molecular motors to their corresponding cargo. &lt;br/&gt;&lt;br/&gt;Our approaches include biochemical, genetic, biophysical and structural studies of actin, myosin, and associated proteins from eukaryotic cells.  We have designed and developed in vitro assays for ATP-dependent movement of purified myosin on filaments reconstituted from purified actin.  We have taken this assay to the single molecule level, using laser traps, total internal reflection fluorescence microscopy, and gold nanoparticle tracking.  Myosin cloning and expression of mutagenized forms that are analyzed for altered functions is routine in our laboratory. &lt;br/&gt;&lt;br/&gt;The detailed understanding we have developed of how myosin transduces the chemical energy of ATP hydrolysis into mechanical movement has led us to our current focus on human hypertrophic cardiomyopathy (HCM) caused by missense mutations in human ß-cardiac myosin.  Our goal is to elucidate the molecular basis of hypercontractility seen clinically resulting from HCM mutations.  We postulated in 2015 that a majority of HCM mutations shift ß-cardiac myosin heads from a sequestered off-state to an active on-state for interaction with actin, resulting in the hypercontractility seen clinically. This hypothesis is different from earlier prevailing views, and this viewing an old disease in a new light is the basis of all of our current research.  We now have extensive evidence for this hypothesis using a combination of the various high-resolution technologies we have developed over the years as well as new approaches.  Our work is now providing possible paths forward for therapeutic intervention for cardiomyopathy patients.</p>\n",
      "\n",
      "[<p>Dr. Svensson completed her Ph.D. at Lund University and her postdoctoral work at the Dana-Farber Cancer Institute and Harvard Medical School, in the lab of Bruce Spiegelman, studying mammalian energy metabolism. The Svensson Laboratory is dedicated to the discovery of new pathways of energy regulation. We are using a combination of secretome proteomics, gene editing and physiology approaches to better understand how to target complex diseases such as aging, metabolic diseases, and cancer.</p>]<p>The Svensson Laboratory is dedicated to the discovery of new ligands that can control metabolism. We are using a combination of omics, gene editing and physiology approaches to better understand how to target complex diseases such as aging, metabolic diseases, and cancer.</p>\n",
      "\n",
      "[<p>Prof. Sindy KY Tang is the Kenneth and Barbara Oshman Faculty Scholar and Associate Professor of Mechanical Engineering and by courtesy of Radiology (Precision Health and Integrated Diagnostics) at Stanford University. She received her Ph.D. from Harvard University in Engineering Sciences under the supervision of Prof. George Whitesides. Her lab at Stanford works on the fundamental understanding of fluid mechanics and mass transport in micro-nano systems, and the application of this knowledge towards problems in biology, rapid diagnostics for health and environmental sustainability. The current areas of focus include the flow physics of confined micro-droplets using experimental and machine learning methods, interfacial mass transport and self-assembly, and ultrahigh throughput opto-microfluidic systems for disease diagnostics, water and energy sustainability, and single-cell wound healing studies. She was a Stanford Biodesign Faculty Fellow in 2018. Dr. Tang’s work has been recognized by multiple awards including the NSF CAREER Award, 3M Nontenured Faculty Award, the ACS Petroleum Fund New Investigator Award, and invited lecture at the Nobel Symposium on Microfluidics in Sweden. Website: http://web.stanford.edu/group/tanglab/</p>]<p>The long-term goal of Dr. Tang's research program is to harness mass transport in microfluidic systems to accelerate precision medicine and material design for a future with better health and environmental sustainability. &lt;br/&gt;&lt;br/&gt;Current research areas include: (I) Physics of droplets in microfluidic systems, (II) Interfacial mass transport and self-assembly, and (III) Applications in food allergy, single-cell wound repair, and the bottom-up construction of synthetic cell and tissues in close collaboration with clinicians and biochemists at the Stanford School of Medicine, UCSF, and University of Michigan.&lt;br/&gt;&lt;br/&gt;For details see https://web.stanford.edu/group/tanglab/</p>\n",
      "\n",
      "<p>The goal of our laboratory is to develop, scale up, and broadly disseminate molecular technologies for mapping cells and functional circuits. At the sub-cellular scale, maps document the spatial organization of proteins, RNA, DNA, and metabolites with nanometer precision and temporal acuity on the order of seconds. Maps also chart the connectivity between these molecules, elucidating the circuits and signaling processes that give rise to function. &lt;br/&gt;&lt;br/&gt;Beyond the single cell, we also strive to map cellular ensembles, such as brain tissue. Can we create tools that contribute to the construction of cell and tissue atlases, and can we map the cellular circuits that give rise to function and behavior? To achieve these ambitious goals, our laboratory has focused on the development of scalable technologies to detect, measure, and manipulate molecules and circuits, both at the sub-cellular level, and at the level of cell populations.</p>\n",
      "\n",
      "[<p>Dr. Vasanawala received his bachelor's degree in mathematics from Caltech, and then completed his medical training and PhD at Stanford.  His research efforts are focused on developing fast and quantitative MRI methods.  He serves as the Director of MRI at Stanford Children's.  He also serves as the division Chief of Pediatric Radiology, Associate Chair of Radiology, and the Radiologist-in-Chief for Pediatric Radiology.</p>]<p>Our group is focused on developing new MRI techniques, and in particular, developing novel applications for children.  We take a comprehensive approach, exploring novel hardware, data acquisition, image reconstruction, and image analysis techniques.  These approaches are then evaluated for cardiovascular, abdominal, and musculoskeletal pediatric MRI exams.  Additionally, we seek to develop quantitative MRI methods, including those for cardiovascular function, renal function, and tumor perfusion.</p>\n",
      "\n",
      "<p>Cognitive neuroscience of memory and cognitive/executive control in young and older adults. Research interests include encoding and retrieval mechanisms; interactions between declarative, nondeclarative, and working memory; forms of cognitive control; neurocognitive aging; functional organization of prefrontal cortex, parietal cortex, and the medial temporal lobe; assessed by functional MRI, scalp and intracranial EEG, and transcranial magnetic stimulation</p>\n",
      "\n",
      "[<p>Soichi Wakatsuki is a Professor of Photon Science at the SLAC National Accelerator Laboratory where he recently initiated the Biociences Division, and Professor of Structural Biology, Stanford School of Medicine.  He received his B.S and M.S. degrees in Chemical Engineering from University of Tokyo, and his Ph.D. degree in Chemistry from Stanford University in 1991.  After postdoctoral studies on time-resolved x-ray crystallography of enzyme reactions in Oxford (1990 to 1994), he moved to Grenoble, France in 1994 to work at the European Synchrotron Radiation Facility (ESRF) where he led Joint Structural Biology Group to develop high-brilliance x-ray crystallography beamlines and instruments, as well as several structural biology projects on protein transport.  In 2000, Soichi moved back to Japan to start a new Structural Biology Research Center at KEK (High Energy Accelerator Research Organization), Tsukuba, Japan, and later served as Director of Photon Factory (national synchrotron radiation facility) from 2006 to 2012.  There he further developed x-ray beamlines and a large scale protein crystallization system, led initiatives to start three national projects on structural proteomics.  Fascinated by new research opportunities in integrative bioimaging at Stanford and the world’s first hard x-ray free electron laser (XFEL) at SLAC, Soichi returned to Stanford in 2013.  Soichi’s research interests include structural biology of post-translational modification and vesicle transport, structural biology of polyubiquitin recognition, synchrotron radiation and XFEL instrumentation, protein crystallography and small angle X-ray scattering, integrative multi-scale bioimaging.</p>]<p>Ubiquitin signaling: structure, function, and therapeutics&lt;br/&gt;      Ubiquitin is a small protein modifier that is ubiquitously produced in the cells and takes part in the regulation of a wide range of cellular activities such as gene transcription and protein turnover. The key to the diversity of the ubiquitin roles in cells is that it is capable of interacting with other cellular proteins either as a single molecule or as different types of chains. Ubiquitin chains are produced through polymerization of ubiquitin molecules via any of their seven internal lysine residues or the N-terminal methionine residue. Covalent interaction of ubiquitin with other proteins is known as ubiquitination which is carried out through an enzymatic cascade composed of the ubiquitin-activating (E1), ubiquitin-conjugating (E2), and ubiquitin ligase (E3) enzymes. The ubiquitin signals are decoded by the ubiquitin-binding domains (UBDs).  These domains often specifically recognize and non-covalently bind to the different ubiquitin species, resulting in distinct signaling outcomes.&lt;br/&gt;      We apply a combination of the structural (including protein crystallography, small angle x-ray scattering, cryo-electron microscopy (Cryo-EM) etc.), biocomputational and biochemical techniques to study the ubiquitylation and deubiquitination processes, and recognition of the ubiquitin chains by the proteins harboring ubiquitin-binding domains. Current research interests including SARS-COV2 proteases and their interactions with polyubiquitin chains and ubiquitin pathways in host cell responses, with an ultimate goal of providing strategies for effective therapeutics with reduced levels of side effects.&lt;br/&gt;&lt;br/&gt;Protein self-assembly processes and applications. &lt;br/&gt;      The Surface layers (S-layers) are crystalline protein coats surrounding microbial cells. S-layer proteins (SLPs) regulate their extracellular, self-assembly by crystallizing when exposed to an environmental trigger. We have demonstrated that the Caulobacter crescentus SLP readily crystallizes into sheets both in vivo and in vitro via a calcium-triggered multistep assembly pathway. Observing crystallization using a time course of Cryo-EM imaging has revealed a crystalline intermediate wherein N-terminal nucleation domains exhibit motional dynamics with respect to rigid lattice-forming crystallization domains. Rate enhancement of protein crystallization by a discrete nucleation domain may enable engineering of kinetically controllable self-assembling 2D macromolecular nanomaterials. In particular, this is inspiring designing robust novel platform for nano-scale protein scaffolds for structure-based drug design and nano-bioreactor design for the carbon-cycling enzyme pathway enzymes. Current research focuses on development of nano-scaffolds for high throughput in vitro assays and structure determination of small and flexible proteins and their interaction partners using Cryo-EM, and applying them to cancer and anti-viral therapeutics.&lt;br/&gt;&lt;br/&gt;Multiscale imaging and technology developments. &lt;br/&gt;      Multimodal, multiscale imaging modalities will be developed and integrated to understand how molecular level events of key enzymes and protein network are connected to cellular and multi-cellular functions through intra-cellular organization and interactions of the key machineries in the cell. Larger scale organization of these proteins will be studied by solution X-ray scattering and Cryo-EM. Their spatio-temporal arrangements in the cell organelles, membranes, and cytosol will be further studied by X-ray fluorescence imaging and correlated with cryoEM and super-resolution optical microscopy. We apply these multiscale integrative imaging approaches to biomedical, and environmental and bioenergy research questions with Stanford, DOE national labs, and other domestic and international collaborators.</p>\n",
      "\n",
      "<p>Overview – The lab concentrates on the invention of molecules and techniques that enable better studies of biological processes.  In short, we invent tools for biology and we are motivated by approaches that enable entirely new experiments with unprecedented control.  These new techniques may also provide a window into mechanisms that cells use to maintain protein homeostasis.  Protein quality control is a particular interest at present.&lt;br/&gt;&lt;br/&gt;New Approaches for Conditional Control of Protein Function – We have developed an experimental system in which the stability of a specific protein depends on the presence or absence of a cell-permeable molecule.  We started with a well-studied protein-ligand pair: the FKBP12 protein and a high-affinity ligand we synthesized called Shield-1.  We screened a library of FKBP sequences to identify mutants that are unstable in the absence of Shield-1 and are stabilized by Shield-1.  Further characterization of these mutants revealed that the most destabilizing mutants caused a 50-fold to 100-fold reduction in the expression levels of the proteins to which they were fused.  Importantly, this instability is transmitted to any fused partner protein, allowing us to regulate the stability of any protein-of-interest using Shield-1.&lt;br/&gt;&lt;br/&gt;These reagents are called destabilizing domains (DDs), and we have developed a broad portfolio of these reagents based on several distinct protein-ligand combinations.  Most of these reagents are available through Addgene.  The systems work well in cultured mammalian cells, a variety of model organisms (e.g., flies, worms, frogs) and in living mice and rats.  This technique allows rapid and reversible elimination of a specific protein in a variety of biological contexts.  We have also engineered a DD system that functions in the opposite sense.  The fusion protein is stable in the absence of the ligand, and administration of the ligand causes the fusion protein to be rapidly degraded.  Similarly, we have engineered a conditional stability system where light is used to regulate protein stability.  Most of these reagents have been reviewed recently (Rakhit et al., Chemistry &amp;amp; Biology 2014, 21, 1238).&lt;br/&gt;&lt;br/&gt;Protein Quality Control in Cells – The DDs can be thought of as model substrates that have the potential to help us understand how cells detect and deal with misfolded or unfolded proteins.  The ability to conditionally regulate the folding state of these domains using high-affinity ligands allows us to correlate specific biophysical properties with biological stability.  One focus of the lab is understanding how these technologies work in cells.  Using purified proteins we have shown that the DD proteins are either unfolded or significantly populate the unfolded state in the absence of the stabilizing ligand (Egeler et al., JBC  2011, 286, 31328).  We have also used a focused RNAi strategy to identify proteins involved in the cellular response to the unfolded DDs (Chu et al., JBC  2013, 288, 34575).&lt;br/&gt;&lt;br/&gt;More recently, we are using the DDs as conditionally folded proteins to create an acute unfolded protein stress by withdrawing the stabilizing ligand from the cell culture media.  This approach allowed us to identify a novel coordinated transcriptional response that mammalian cells trigger when unfolded protein appears (Miyazaki et al., eLIFE  2015).  Interestingly, there is little overlap between this new cellular response and the heat shock response, which conventional wisdom holds as the &amp;quot;cytosolic unfolded protein response&amp;quot;.  Additionally, creating unfolded DD in either the cytosol or nucleus elicits distinct responses, suggesting that mammalian cells maintain different protein quality control surveillance environments in these compartments.  Understanding the molecular mechanisms behind these new cellular responses as well as how they help to maintain protein homeostasis is a major focus of the lab.</p>\n",
      "\n",
      "[<p>We are a discovery-driven research group working at the interface between developmental biology, bioengineering, and statistical physics. We combine quantitative organism-wide fluorescence imaging, functional genomics, and physical modeling to understand the fundamental rules that control collective cell behaviors to optimize tissue regeneration, adaptation, and evolution.</p>]\n",
      "\n",
      "<p>Our laboratory studies molecular interactions that underlie the establishment and maintenance of cell and tissue structure. Our principal areas of interest are the architecture and dynamics of intercellular adhesion junctions, signaling pathways that govern cell fate determination, and determinants of cell polarity. We also have a long-standing interest in carbohydrate-based cellular recognition and adhesion. We take a strongly reductionist approach to these problems by reconstituting macromolecular assemblies with purified components in order to analyze them using biochemical, biophysical and structural methods. Mechanistic models derived from these studies are tested in cell culture systems. &lt;br/&gt;&lt;br/&gt;&lt;br/&gt;Intercellular adhesion&lt;br/&gt;Several distinct intercellular junctions connect epithelial cells. Two of these, the adherens junction and the desmosome, contain cadherin cell adhesion molecules. The extracellular regions of these transmembrane proteins mediate intercellular binding, while their cytoplasmic domains are linked to the actin- (adherens junction) or intermediate filament- (desmosome) based cytoskeletons. In this way the cytoskeletons of cells comprising a tissue are linked, imparting particular morphologies and mechanical strength to the tissue. The dynamics of these complex assemblies underlie changes in cell and tissue architecture that occur during development and in many cancers. &lt;br/&gt;&lt;br/&gt;Our goal is to understand the architecture and dynamics of these junctions.  A major current focus is in understanding how mechanical force regulates these assemblies by altering molecular conformation, how different junctional components alter force responsiveness, and different intercellular junctions communicate. We are also studying the interplay between cell adhesion and the development and maintenance of apical-basal polarity. Finally, we are examining how cell-cell junctional proteins have changed during metazoan evolution as part of the development of more complex tissue architectures. &lt;br/&gt;&lt;br/&gt;&lt;br/&gt;Wnt signaling&lt;br/&gt;The Wnt signaling pathway controls cell fate determination during embryogenesis and in the normal renewal of tissues in the adult. Moreover, dysregulation of the pathway drives progression of many cancers. In the Wnt/beta-catenin pathway, Wnt protein binding to cell surface receptors leads to activation of target genes by beta-catenin. In the absence of a Wnt signal, non-junctional beta-catenin is bound in a multiprotein “destruction complex”, where it is phosphorylated and targeted for degradation by the ubiquitin/proteosome pathway. Binding of a Wnt to cell surface receptors prevents beta-catenin destruction. We are biochemically reconstituting these complexes for mechanistic and structural studies. Our goals are to determine the mechanism of beta-catenin destruction in the absence of a Wnt signal, and how Wnt binding to receptors turns off beta-catenin destruction.</p>\n",
      "\n",
      "[<p>A pioneer in the use of lasers to study chemical reactions at the molecular level, Marguerite Blake Wilbur Professor Richard N. Zare pursues diverse theoretical and experimental interests in physical chemistry and nanoscale chemical analysis. The Zarelab has made a broad impact in analytic chemistry with development of laser-induced fluorescence to study reaction dynamics, and seminal contributions to understanding of molecular collision processes. The group continues to invent tools and measurement techniques to study phenomena from reaction in microdroplets to drug delivery.<br/><br/>Born in 1939 in Cleveland, Ohio, Professor Zare trained in physical and analytical chemistry at Harvard University (B.A. 1961, Ph.D. 1964). His doctoral study under Professor Dudley Herschbach explored photodissociation dynamics. After faculty positions spanning chemistry at the Massachusetts Institute of Technology, chemistry, physics and astrophysics at the University of Colorado, and chemistry at Columbia University, he joined the Stanford chemistry faculty in 1977. He has taught an introductory chemistry class every year since. As a Howard Hughes Medical Institute Professor since 2006, Professor Zare has also developed a course introducing undergraduates to hands-on interdisciplinary research, combining physics, and biology to explore how living systems use molecular interactions with light for vision, photosynthesis and more. Professor Zare served as chair of the Department of Chemistry from 2005 to 2011, and has helped to guide scientific policy as chairman of several national and international science boards. His dedication to research and teaching has been recognized in many awards, including the National Medal of Science, the Wolf Prize in Chemistry, and the Presidential Award for Excellence in Science, Mathematics, and Engineering Mentoring. Among other honors, Professor Zare is a member of the National Academy of Sciences, the American Academy of Arts and Sciences, and the American Philosophical Society. He has also received 11 honorary doctorates.<br/><br/>Current research in the Zarelab explores wide-ranging questions in physical and analytical chemistry, from the study of elementary chemical reactions to chemical analysis of extraterrestrial materials. The major focus of these efforts is chemical analysis on the nanoscale. The team has devised tools and techniques to examine molecules in extremely tiny volumes – the volumes characteristic of what is found in heterogeneous structures in mineral samples or in the contents of cells and subcellular compartments. Group members have also made contributions to the chemical analysis of liquid samples separated using a capillary format by electrophoresis or electrochromatography. Some “firsts” include the use of cavity ring-down spectroscopy to analyze trace species in solution, development of detectors for capillary electrophoresis based on the techniques of laser-induced fluorescence, and CCD imaging, and the use of mass spectrometric imaging of tissue samples by means of desorption electrospray ionization.<br/><br/>Please visit the Zarelab website to learn more: http://web.stanford.edu/group/Zarelab/</p>]<p>My research group is exploring a variety of topics that range from the basic understanding of chemical reaction dynamics to the nature of the chemical contents of single cells.&lt;br/&gt;&lt;br/&gt;Under thermal conditions nature seems to hide the details of how elementary reactions occur through a series of averages over reagent velocity, internal energy, impact parameter, and orientation. To discover the effects of these variables on reactivity, it is necessary to carry out studies of chemical reactions far from equilibrium in which the states of the reactants are more sharply restricted and can be varied in a controlled manner.  My research group is attempting to meet this tough experimental challenge through a number of laser techniques that prepare reactants in specific quantum states and probe the quantum state distributions of the resulting products. It is our belief that such state-to-state information gives the deepest insight into the forces that operate in the breaking of old bonds and the making of new ones.&lt;br/&gt;&lt;br/&gt;Space does not permit a full description of these projects, and I earnestly invite correspondence. The following examples are representative:&lt;br/&gt;&lt;br/&gt;The simplest of all neutral bimolecular reactions is the exchange reaction H H2 -&amp;gt; H2 H. We are studying this system and various isotopic cousins using a tunable UV laser pulse to photodissociate HBr (DBr) and hence create fast H (D) atoms of known translational energy in the presence of H2 and/or D2 and using a laser multiphoton ionization time-of-flight mass spectrometer to detect the nascent molecular products in a quantum-state-specific manner by means of an imaging technique. It is expected that these product state distributions will provide a key test of the adequacy of various advanced theoretical schemes for modeling this reaction.&lt;br/&gt;&lt;br/&gt;Analytical efforts involve the use of capillary zone electrophoresis, two-step laser desorption laser multiphoton ionization mass spectrometry, cavity ring-down spectroscopy, and Hadamard transform time-of-flight mass spectrometry. We believe these methods can revolutionize trace analysis, particularly of biomolecules in cells.</p>\n",
      "\n",
      "[<p>Roseanna N. Zia is an Associate Professor of Chemical Engineering at Stanford University and, by courtesy, Mechanical Engineering.  She received her Ph.D. from the California Institute of Technology in Mechanical Engineering in 2011 with Professor John F. Brady, for development of theory in colloidal hydrodynamics and microrheology.  Zia subsequently conducted post-doctoral study of colloidal gels at Princeton University, in collaboration with Professor William B. Russel.  Zia began her faculty career at Cornell in January 2013, then subsequently moved her research group to Stanford University in 2017. <br/><br/>Dr. Zia’s research includes developing micro-continuum theory for structure-property relationships of flowing suspensions, elucidating the mechanistic origins of the colloidal glass transition, and microscopic modeling of reversibly bonded colloidal gels, which resulted in discovery that gel aging is actually ongoing but very slow phase separation and the finding that mechanical yield of colloidal gels is actually a non-equilibrium phase transition, triggered by changes in osmotic pressure.  Her research group aims to unlock the fundamental connections between colloidal-scale physics and life-essential processes in biological cells using theoretical colloid physics, biological modeling, and high-fidelity computational models.  Her group’s ultimate vision is to create a generalized platform for uncovering disease mechanisms and pathways for physics-based therapeutics.<br/><br/>Dr. Zia’s work has been recognized by several awards, including the PECASE Award, the ONR Director of Research Early Career Award, the Office of Naval Research (ONR) Young Investigator award, the NSF CAREER Award, the NSF BRIGE Award, the Publication Award from the Society of Rheology, and the Engineering Sonny Yau (’72) Teaching Award.  Most recently she was named an Otterson Faculty Fellow at Stanford.<br/><br/>Dr. Zia serves as an Associate Editor for the Journal of Rheology, and on the Advisory Board of the AIChE Journal.</p>]<p>Colloidal gels, glasses, and suspensions form over 95% of biological fluids, most pharmaceutical fluids, and are ubiquitous across personal-care, agricultural, and industrial-coating materials. Despite the pervasiveness of these fluid-suspended, microscopically small particles (colloids), many of their behaviors have defied explanation – such as the sudden collapse of colloidal gels, vitrification that thwarts crystallization, and their physical role in biological cell function.  In addition to these opportunities, the frontier of cellular biology resides at the colloidal scale, requiring a merger between physics and biology, which have evolved almost orthogonally since the 19th century. My lab focuses on unifying mesoscale physics and chemistry with cellular-level biology through novel theoretical modeling and large-scale computational simulations. Our work includes four areas: 1) constructing a micro-continuum theory of complex fluids to predict heterogenous flows of arbitrary composition; 2) mechanistically explaining non-equilibrium phase transitions in colloidal systems; 3) modeling confined and large-scale hydrodynamically-interacting colloidal suspensions; and 4) modeling the physics of living cells. I chose these areas specifically to support my group’s ultimate vision: accurate, physics-based modeling of whole-cell function where physical laws predict biological behavior.&lt;br/&gt;&lt;br/&gt;Core Competencies: My lab’s expertise is in uncovering the physics of far-from-equilibrium behaviors of soft matter and explaining how they emerge from microscopic forces. We have developed three core competencies:&lt;br/&gt;•Micro-continuum theory using energy and momentum methods. Theoretical modeling of colloids using microhydrodynamics is routine, but application to dense or heterogeneous suspensions is an immense challenge. Our energy methods are a powerful tool to encode complex boundary conditions yet preserve structural detail.&lt;br/&gt;•Physics-based computational modeling.  Computational modeling of suspensions is robust for small or unconfined systems, but current methods obtain large system size by sacrificing on the geometry or transport processes they represent. Our novel physics-based and deep-learning algorithms address these limitations.&lt;br/&gt;•Physics-based modeling of biological cell functions. Biochemical models of life cannot explain many growth- or condition-dependent changes in biological cell function. We merge colloidal-scale modeling with biological process chemistry to reveal entirely new physics-based mechanisms of life-essential cell processes.</p>\n",
      "\n",
      "<p>Glia are a frontier of neuroscience, and overwhelming evidence from the last decade shows that they are essential regulators of all aspects of the nervous system. The Zuchero Lab aims to uncover how glial cells regulate neural development and how their dysfunction contributes to diseases like multiple sclerosis (MS) and in injuries like stroke.&lt;br/&gt;&lt;br/&gt;Although glia represent more than half of the cells in the human brain, fundamental questions remain to be answered. How do glia develop their highly specialized morphologies and interact with neurons to powerfully control form and function of the nervous system? How is this disrupted in neurodegenerative diseases and after injury? By bringing cutting-edge cell biology techniques to the study of glia, we aim to uncover how glia help sculpt and regulate the nervous system and test their potential as novel, untapped therapeutic targets for disease and injury.&lt;br/&gt;&lt;br/&gt;We are particularly interested in myelin, the insulating sheath around neuronal axons that is lost in diseases like MS. How do oligodendrocytes- the glial cell that produces myelin in the central nervous system- form and remodel myelin, and why do they fail to regenerate myelin in disease? Our current projects aim to use cell biology and neuroscience approaches to answer these fundamental questions. Ultimately we hope our work will lead to much-needed therapies to promote remyelination in patients.</p>\n",
      "\n",
      "<p>Hanawalt was a productive researcher in the field of DNA repair since his pioneering discovery of repair replication and co-discovery of nucleotide excision repair in E. coli in 1963. In 1982 Hanawalt and his colleagues reported the first example of intragenomic DNA repair heterogeneity: chemical adducts in alpha DNA in African green monkey kidney cells were not as efficiently repaired as in the genome overall. Hanawalt and his colleagues then discovered that repair of some types of damage is selective; active genes are preferentially repaired, and in fact a special repair pathway, termed transcription-coupled repair (TCR), operates on the transcribed strands of expressed genes. TCR was documented in mammalian cells, in E. coli, and in yeast chromosomal and plasmid borne genes. The discovery of TCR in Hanawalt s laboratory has had profound implications for the fields of mutagenesis, environmental carcinogenesis, aging, and risk assessment. &lt;br/&gt;The prototype recQ gene was discovered in E. coli in Hanawalt s laboratory, and we now know of five homologues in humans including the genes mutated in the cancer prone hereditary diseases: Bloom s syndrome, Werner s syndrome, and Rothman Thompson syndrome. &lt;br/&gt;More recent studies focused upon the regulation of TCR and the global genomic nucleotide excision repair (GGR) pathway. Features of the TCR pathway (defective in Cockayne syndrome) include the possibility of &amp;quot;gratuitous TCR&amp;quot; at transcription pause sites in undamaged DNA. The GGR pathway was shown to be controlled through the SOS stress response in E. coli and through the activated product of the p53 tumor suppressor gene in human cells. These regulatory systems particularly affect the efficiency of repair of the predominant UV-induced photoproduct, the cyclobutane pyrimidine dimer, as well as that of chemical carcinogen DNA adducts, such as benzo(a)pyrene diol-epoxide and benzo(g)chrysene. Rodent cells (typically lacking the p53-controlled GGR pathway) are unable to carry out efficient GGR of some lesions. Therefore, caution should be exercised in the interpretation of results from such systems for risk assessment in genetic toxicology.</p>\n",
      "\n",
      "<p>Prof. Smith s laboratory explores the development, structure, function and disorders of the brain s neural circuitry.  The lab s experimental approach has typically begun with the invention of a new optical imaging method followed by applications of that method to attack important but previous untractable experimental challenges.  Early on, Smith invented a novel fiber-optic spectrometer for calcium sensing that enabled the first detection and measurement of calcium transients in vertebrate neurons, the first quantitative measurements of presynaptic Ca transients, and the extraordinarily significant discovery of Ca influx through NMDA receptor channels.  Later Smith lab imaging inventions led to numerous significant neuroscience discoveries, including retrograde actin flow within neuronal growth cones, intracellular Ca waves in astrocytes, the active role of dendritic filopodia in synaptogenesis, and the packeted delivery of synaptic protein components during synaptogenesis, and to the first optical measurements of single synaptic vesicle release, the first in vivo imaging of synaptotropic dendrite growth, and the first in vivo functional imaging measurements of visual receptive field development in a vertebrate animal.  Most recently, they have invented a unique high-resolution proteomic imaging method called  array tomography , and are now working to apply this novel method to explore the molecular architecture of cortical microcircuits in mouse and human.  This work is currently focused on efforts to identify the circuit loci of the specific changes in synaptic connectivity associated with specific memory traces, i.e. the physical  engrams  of experience.  This work is referenced more fully in the Smith  Biosketch  and  CV  documents to be found on this web page.</p>\n",
      "\n",
      "[<p>Norbert Pelc is Professor of Radiology, Emeritus. His primary research interests are in the physics, engineering, and mathematics of diagnostic imaging and the development of applications of this imaging technology. His current work focuses on computed tomography, specifically in methods to improve the information content and image quality and to reduce the radiation dose from these examinations. He holds a doctorate and master degrees in Medical Radiological Physics from Harvard University and a BS from the University of Wisconsin in Madison. He served on the first National Advisory Council of the National Institute of Biomedical Imaging and Bioengineering of the NIH. He is a member of the National Academy of Engineering and a Fellow of the American Association of Physicists in Medicine, the International Society for Magnetic Resonance in Medicine, the American Institute of Medical and Biological Engineering, and of SPIE.</p>]<p>Medical imaging has made enormous strides in recent decades. In clinical medicine, imaging plays an increasingly important role in patient care.  A recent study found that internists rank the development of computed tomography (CT) and magnetic resonance imaging (MRI), together, as the most important innovation in medicine (Health Affairs, Vol 20, p. 30, 2001). At the same time, experts in a completely different scientific field, the National Academy of Engineering, ranks the development of imaging as one of the top 20 greatest engineering achievements of the 20th century (www.greatachievements.org), amazingly at a rank higher than that of household appliances and nuclear technology.  Imaging is also taking on an increasing role in research, improving our understanding of both normal and diseased states and as a surrogate endpoint in the evaluation of therapies.  Imaging allows serial studies in the same individual, thereby increasing statistical power and reducing the number of subjects needed in a study.  Imaging is also a powerful tool to guide minimally invasive therapies.&lt;br/&gt;&lt;br/&gt;The effectiveness of imaging and the powerful impact of visual images have led to a major increase in the utilization of this strategy, a trend that will continue but will evolve in coming years.  Further advances will lead to improved detection, localization, and characterization of disease which should enable more accurate selection of optimized therapies for individual subjects (personalized medicine) as well as treatments that are more effective, less expensive, and less traumatic. Imaging will also play an increasingly important role in the challenges facing biomedical research.&lt;br/&gt;&lt;br/&gt;There are many imaging  modalities , each acquiring data using physical mechanisms such as x-ray transmission, nuclear magnetic resonance, acoustic or optical properties, and signals from radioactive tracers.  Optimal design and utilization of each requires an appreciation of the underlying physical phenomena.  Each modality uses sensors to detect signals and mathematical methods to covert the measured signals to images.  Additional image processing methods are used to extract physiological information from the images.&lt;br/&gt;&lt;br/&gt;My own interests center on the physics, engineering and mathematics of medical imaging.  While I have worked on many imaging modalities over the past decades, my current projects are focused on computed tomography, digital x-ray imaging, and hybrid multimodality systems.  An area of current focus is understanding the potential impact of a new class of x-ray detectors for CT imaging, energy discriminating photon counting detectors. They promise improved tissue characterization and dose efficiency, but currently available detectors have imperfections that detract from their performance. Important questions for this and other new technologies are: what are the real benefits and when is it worthwhile adopting them into clinical systems.&lt;br/&gt;&lt;br/&gt;In addition to these technical projects, I am also interested in the development of new clinical and research applications of medical imaging.  This is highly interdisciplinary research, incorporating not only the latest imaging technology but also fundamental appreciation of anatomy and pathophysiology.</p>\n",
      "\n"
     ]
    }
   ],
   "source": [
    "profile_divs = soup.findAll('div', {\"class\": \"profile-name\"})\n",
    "stanford_faculty_dict = {}\n",
    "for i in profile_divs:\n",
    "    atags = i.findAll('a')\n",
    "    for j in atags:\n",
    "        faculty_url = j['href']\n",
    "        resp = requests.get(faculty_url)\n",
    "        fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "        \n",
    "        bio = fac_soup.findAll('div', {\"class\": \"bio content-section\"})\n",
    "        bio_content = [x.findAll('p') for x in bio]\n",
    "        \n",
    "        if len(bio_content) == 0:\n",
    "            bio_content = [\"\"]\n",
    "            \n",
    "        bio_content = str(bio_content[0])\n",
    "        research_tab = fac_soup.findAll('a', {'id': \"research-and-scholarship\"})\n",
    "        url_tail = research_tab[0]['href'] if len(research_tab) != 0 else \"\"\n",
    "        research_url = faculty_url + url_tail\n",
    "        \n",
    "        research_resp = requests.get(research_url)\n",
    "        research_soup = BeautifulSoup(research_resp.text, \"html.parser\")\n",
    "        research = research_soup.findAll('div', {\"class\": \"interests content-section\"})\n",
    "        research_cont = [x.findAll('p') for x in research]\n",
    "        if len(research_cont) != 0:\n",
    "            test = str(research_cont[0][0])\n",
    "            bio_content += test \n",
    "        print(bio_content)\n",
    "        stanford_faculty_dict[(j.text, faculty_url)] = bio_content      \n",
    "    print()\n",
    "        \n",
    "        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "68c860fd-688f-47f5-8dd4-55e9b19c41be",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 109 faculty\n",
      "Vijay Pande https://med.stanford.edu/profiles/vijay-pande 108\n",
      "Russ B. Altman https://med.stanford.edu/profiles/russ-altman 77\n",
      "Ron Dror https://med.stanford.edu/profiles/ron-dror 60\n",
      "Crystal Mackall https://med.stanford.edu/profiles/crystal-mackall 40\n",
      "Manish Saggar https://med.stanford.edu/profiles/manish-saggar 40\n",
      "Grant M. Rotskoff https://med.stanford.edu/profiles/rotskoff 39\n",
      "Anshul Kundaje https://med.stanford.edu/profiles/anshul-kundaje 35\n",
      "Soichi Wakatsuki https://med.stanford.edu/profiles/soichi-wakatsuki 34\n",
      "Possu Huang https://med.stanford.edu/profiles/possu-huang 32\n",
      "Peter S. Kim https://med.stanford.edu/profiles/peterkim 30\n",
      "Rhiju Das https://med.stanford.edu/profiles/rhiju-das 29\n",
      "Michael Levitt https://med.stanford.edu/profiles/michael-levitt 29\n",
      "Roseanna N. Zia https://med.stanford.edu/profiles/roseanna-zia 27\n",
      "Axel Brunger https://med.stanford.edu/profiles/axel-brunger 26\n",
      "Keren Haroush https://med.stanford.edu/profiles/keren-haroush 26\n",
      "Tom Wandless https://med.stanford.edu/profiles/thomas-wandless 25\n",
      "Zev Bryant https://med.stanford.edu/profiles/zev-bryant 22\n",
      "Todd Martinez https://med.stanford.edu/profiles/todd-martinez 22\n",
      "Chris Garcia https://med.stanford.edu/profiles/chris-garcia 21\n",
      "Shamit Kachru https://med.stanford.edu/profiles/shamit-kachru 20\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(stanford_faculty_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0f25faf2-5f34-4c45-8eaf-599f702cfaf3",
   "metadata": {},
   "source": [
    "# MIT Faculty Analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "149ab74f-d85e-4371-897a-99b3cda20c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = \"https://csbphd.mit.edu/faculty\"\n",
    "response = requests.get(url)\n",
    "soup = BeautifulSoup(response.text, \"html.parser\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "d51e7268-b0ab-4228-afce-0d7def12fad9",
   "metadata": {},
   "outputs": [],
   "source": [
    "mit_faculty_dict = {}\n",
    "profile_trs = soup.findAll('tr')\n",
    "for idx, i in enumerate(profile_trs):\n",
    "    if idx == 0:\n",
    "        continue\n",
    "    name = i.findAll(\"a\")[0]\n",
    "    link_tail = name[\"href\"].replace(\"/faculty\", \"\")\n",
    "    name = name.text\n",
    "    resp = requests.get(url + link_tail)\n",
    "    fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "    data = fac_soup.findAll(\"section\", {\"class\":\"field field-name-field-research-summary field-type-text-with-summary field-label-above\"})\n",
    "    text = \"\"\n",
    "    for i in data:\n",
    "        text += i.text\n",
    "    mit_faculty_dict[(name, url+link_tail)] = text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "a3a1f2a5-d4a3-474b-b8e6-eee446d00028",
   "metadata": {
    "scrolled": true,
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 60 faculty\n",
      "Barzilay, Regina https://csbphd.mit.edu/faculty/regina-barzilay 44\n",
      "Kulik, Heather J. https://csbphd.mit.edu/faculty/heather-j-kulik 39\n",
      "Poggio, Tomaso A. https://csbphd.mit.edu/faculty/tomaso-poggio 32\n",
      "Bathe, Mark https://csbphd.mit.edu/faculty/mark-bathe 31\n",
      "Stultz, Collin M. https://csbphd.mit.edu/faculty/collin-m-stultz 30\n",
      "Gifford, David K. https://csbphd.mit.edu/faculty/david-k-gifford 26\n",
      "Coley, Connor W. https://csbphd.mit.edu/faculty/connor-w-coley 25\n",
      "Zhang, Bin https://csbphd.mit.edu/faculty/bin-zhang 25\n",
      "Sontag, David https://csbphd.mit.edu/faculty/david-sontag 22\n",
      "Burge, Chris https://csbphd.mit.edu/faculty/chris-burge 21\n",
      "Hemann, Michael T. https://csbphd.mit.edu/faculty/michael-t-hemann 19\n",
      "Shalek, Alex K. https://csbphd.mit.edu/faculty/alex-k-shalek 18\n",
      "Wong, Harikesh S. https://csbphd.mit.edu/faculty/harikesh-s-wong 17\n",
      "Keating, Amy https://csbphd.mit.edu/faculty/amy-keating 16\n",
      "Fraenkel, Ernest https://csbphd.mit.edu/faculty/ernest-fraenkel 13\n",
      "Weiss, Ron https://csbphd.mit.edu/faculty/ron-weiss 13\n",
      "Lieberman, Tami  https://csbphd.mit.edu/faculty/tami-lieberman 12\n",
      "Uhler, Caroline https://csbphd.mit.edu/faculty/caroline-uhler 12\n",
      "Fiete, Ila https://csbphd.mit.edu/faculty/ila-fiete 11\n",
      "Young, Richard A. https://csbphd.mit.edu/faculty/richard-young 10\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(mit_faculty_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "300b5b0c-b2e4-456f-820b-6b22f1e36fde",
   "metadata": {},
   "source": [
    "# Pitt-CMU Faculty Analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "2635dba6-e15a-48c9-9375-a48b3e3e6d2f",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = \"https://www.mbsb.pitt.edu/index.php/people/faculty\"\n",
    "response = requests.get(url)\n",
    "soup = BeautifulSoup(response.text, \"html.parser\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "77ca12f5-2c83-484a-accc-253e12d7efa3",
   "metadata": {},
   "outputs": [],
   "source": [
    "pitt_faculty_dict = {}\n",
    "faculty_list = soup.findAll('div', {\"class\": \"nspArt nspCol4\"})\n",
    "for i in faculty_list:\n",
    "    name = i.findAll('a')[0]\n",
    "    link_tail = name['href']\n",
    "    name = name.text\n",
    "    prefix = url.split(\"/index.php\")[0]\n",
    "    faculty_link = prefix + link_tail\n",
    "    resp = requests.get(faculty_link)\n",
    "    fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "    data = fac_soup.findAll(\"div\", {\"class\":\"item-page\"})\n",
    "    text = \"\"\n",
    "    for i in data:\n",
    "        text += i.text\n",
    "    pitt_faculty_dict[(name, faculty_link)] = text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "d810383c-52c9-4166-89f7-d9c7dbbbce49",
   "metadata": {
    "scrolled": true,
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 43 faculty\n",
      "Jacob Durrant https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/277-currant 56\n",
      "Pei Tang https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/188-pei-tang 50\n",
      "Olexandr Isayev https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/315-oleander-isayev 25\n",
      "Yan Xu https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/200-yan-xu 25\n",
      "Jonathan Coleman https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/298-jonathan-coleman 21\n",
      "Markus Deserno https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/152-markus-deserno 18\n",
      "Maria Kurnikova https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/169-maria-kurnikova 17\n",
      "Cheng Zhang https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/296-cheng-zhang 14\n",
      "Jeffrey Brodsky https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/144-jeffrey-brodsky 13\n",
      "Patricia Opresko https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/180-patricia-opresko 12\n",
      "Kabirul Islam https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/263-kislam 11\n",
      "Jean-Pierre Vilardaga https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/275-vilardaga 11\n",
      "Seth Horne https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/162-seth-horne 10\n",
      "Jinwoo Ahn https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/140-jinwoo-ahn 9\n",
      "Thomas Smithgall https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/185-thomas-smithgall 9\n",
      "Ben Van Houten https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/193-ben-van-houten 9\n",
      "William Furey https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/155-william-furey 8\n",
      "Ivet Bahar https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/142-ivet-bahar 6\n",
      "Angela Gronenborn https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/156-angela-gronenborn 6\n",
      "Frederick Lanni https://www.mbsb.pitt.edu/index.php/site-map/articles/81-faculty/170-frederick-lanni 6\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(pitt_faculty_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "188408ed-b75d-4105-b2e1-55996b62af9c",
   "metadata": {},
   "source": [
    "# UCSF (just bioengineering and pharmacology faculty)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f119ac65-ffa4-4b9e-baad-cd1a70f208a5",
   "metadata": {},
   "source": [
    "### Bioengineering Faculty"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "f61e13f8-afaf-443e-a52c-8d226e421421",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = \"https://bts.ucsf.edu/people/faculty\"\n",
    "response = requests.get(url)\n",
    "soup = BeautifulSoup(response.text, \"html.parser\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "195dc5cb-449f-4d34-b45f-a0b9bd574c4b",
   "metadata": {},
   "outputs": [],
   "source": [
    "ucsf_faculty_dict = {}\n",
    "faculty_table = soup.findAll(\"div\", {\"class\":\"person_name\"})\n",
    "for idx, i in enumerate(faculty_table):\n",
    "    if idx == 0:\n",
    "        continue\n",
    "    name = i.findAll(\"a\")\n",
    "    link = name[0]['href']\n",
    "    name = name[0].text\n",
    "    resp = requests.get(link)\n",
    "    fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "    data = fac_soup.findAll(\"div\", {\"class\":\"moscone-column-content-region-inner moscone-content-inner panel-panel-inner\"})\n",
    "    text = \"\"\n",
    "    for i in data:\n",
    "        text += i.text\n",
    "    \n",
    "    ucsf_faculty_dict[(name, link)] = text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "4662c733-1943-49f1-a906-a92528b7fa74",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 21 faculty\n",
      "Tanja Kortemme, PhD https://pharmacy.ucsf.edu/tanja-kortemme 188\n",
      "Kathy Giacomini, PhD https://pharmacy.ucsf.edu/kathy-giacomini 98\n",
      "Leslie Benet, PhD https://pharmacy.ucsf.edu/leslie-benet 65\n",
      "Deanna Kroetz, PhD https://pharmacy.ucsf.edu/deanna-kroetz 55\n",
      "Francis Szoka, PhD https://pharmacy.ucsf.edu/francis-szoka 54\n",
      "Rada Savic, PhD https://pharmacy.ucsf.edu/rada-savic 49\n",
      "Sourav Bandyopadhyay, PhD https://pharmacy.ucsf.edu/sourav-bandyopadhyay 46\n",
      "C. Anthony Hunt, PhD https://pharmacy.ucsf.edu/c-anthony-hunt 43\n",
      "Andrej Sali, PhD https://pharmacy.ucsf.edu/andrej-sali 43\n",
      "Tejal Desai, PhD https://pharmacy.ucsf.edu/tejal-desai 42\n",
      "Ryan Hernandez, PhD https://pharmacy.ucsf.edu/ryan-hernandez 30\n",
      "Shuvo Roy, PhD https://pharmacy.ucsf.edu/shuvo-roy 26\n",
      "James Fraser, PhD https://pharmacy.ucsf.edu/james-fraser 24\n",
      "Seth Shipman, PhD https://pharmacy.ucsf.edu/seth-shipman 23\n",
      "Nancy Sambol, PharmD https://pharmacy.ucsf.edu/nancy-sambol 22\n",
      "Esteban G. Burchard, MD, MPH https://pharmacy.ucsf.edu/esteban-burchard 10\n",
      "Sophie Dumont, PhD https://pharmacy.ucsf.edu/sophie-dumont 10\n",
      "Xin Chen, PhD https://pharmacy.ucsf.edu/xin-chen 7\n",
      "Su Guo, PhD https://pharmacy.ucsf.edu/su-guo 7\n",
      "Nadav Ahituv, PhD https://pharmacy.ucsf.edu/nadav-ahituv 1\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(ucsf_faculty_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d52a877b-e507-4409-b8e8-0d1815c75ca5",
   "metadata": {},
   "source": [
    "### Pharmaceutical Chemistry"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "71e709f4-096c-484a-ac61-22c5eaf8f064",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = \"https://pharmchem.ucsf.edu/people/faculty\"\n",
    "response = requests.get(url)\n",
    "soup = BeautifulSoup(response.text, \"html.parser\")\n",
    "ucsf_pharm_faculty_dict = {}\n",
    "faculty_table = soup.findAll(\"div\", {\"class\":\"person_name\"})\n",
    "for idx, i in enumerate(faculty_table):\n",
    "    if idx == 0:\n",
    "        continue\n",
    "    name = i.findAll(\"a\")\n",
    "    link = name[0]['href']\n",
    "    name = name[0].text\n",
    "    resp = requests.get(link)\n",
    "    fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "    data = fac_soup.findAll(\"div\", {\"class\":\"moscone-column-content-region-inner moscone-content-inner panel-panel-inner\"})\n",
    "    text = \"\"\n",
    "    for i in data:\n",
    "        text += i.text\n",
    "    \n",
    "    ucsf_pharm_faculty_dict[(name, link)] = text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "d3ba703c-d5af-4cb9-a891-ef6f48e3c4cc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 33 faculty\n",
      "Michael Keiser, PhD https://pharmacy.ucsf.edu/michael-keiser 82\n",
      "Thomas James, PhD https://pharmacy.ucsf.edu/thomas-james 70\n",
      "Brian Shoichet, PhD https://pharmacy.ucsf.edu/brian-shoichet 70\n",
      "Lei Wang, PhD https://pharmacy.ucsf.edu/lei-wang 62\n",
      "Tom Ferrin, PhD https://pharmacy.ucsf.edu/tom-ferrin 61\n",
      "Matt Jacobson, PhD https://pharmacy.ucsf.edu/matt-jacobson 61\n",
      "William DeGrado, PhD https://pharmacy.ucsf.edu/william-degrado 55\n",
      "Xiaokun Shu, PhD https://pharmacy.ucsf.edu/xiaokun-shu 53\n",
      "Michelle Arkin, PhD https://pharmacy.ucsf.edu/michelle-arkin 47\n",
      "Jim Wells, PhD https://pharmacy.ucsf.edu/jim-wells 44\n",
      "Paul Ortiz de Montellano, PhD https://pharmacy.ucsf.edu/paul-ortiz-de-montellano 43\n",
      "Charles S. Craik, PhD https://pharmacy.ucsf.edu/charles-s-craik 40\n",
      "Lani Wu, PhD https://pharmacy.ucsf.edu/lani-wu 39\n",
      "Jason Gestwicki, PhD https://pharmacy.ucsf.edu/jason-gestwicki 34\n",
      "Aashish Manglik, MD, PhD https://pharmacy.ucsf.edu/aashish-manglik 33\n",
      "Adam Renslo, PhD https://pharmacy.ucsf.edu/adam-renslo 26\n",
      "Al Burlingame, PhD https://pharmacy.ucsf.edu/al-burlingame 24\n",
      "Bo Huang, PhD https://pharmacy.ucsf.edu/bo-huang 24\n",
      "Michael Grabe, PhD https://pharmacy.ucsf.edu/michael-grabe 20\n",
      "John Gross, PhD https://pharmacy.ucsf.edu/john-gross 18\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(ucsf_pharm_faculty_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d564455e-2f9b-410c-afab-a40289949a0c",
   "metadata": {},
   "source": [
    "# UPenn Faculty Analysis\n",
    "### All of the faculty have different website formats so this just scrapes everything. Much less useful rankings..."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "428b47e4-f677-436b-aa25-f7547a0ad77d",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = \"https://www.med.upenn.edu/bmbgrad/bmb_faculty.html\"\n",
    "response = requests.get(url)\n",
    "soup = BeautifulSoup(response.text, \"html.parser\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "9dfa7878-1846-4820-8c45-32fbfd6f0956",
   "metadata": {},
   "outputs": [],
   "source": [
    "all_content = soup.findAll(\"div\", {\"class\":\"row main\"})\n",
    "found = False\n",
    "list_ = all_content[0].findAll(\"strong\")[7]\n",
    "links = list_.find_all_next(\"a\")\n",
    "penn_faculty_dict = {}\n",
    "for i in links:\n",
    "    try:\n",
    "        link = i['href']\n",
    "        name = i.text\n",
    "        if name == \".\":\n",
    "            continue \n",
    "            \n",
    "        resp = requests.get(link)\n",
    "        fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "        text = \"\"\n",
    "        for i in fac_soup:\n",
    "            text += i.text\n",
    "        penn_faculty_dict[(name, link)] = text\n",
    "        if \"Zhaolan\" in name:\n",
    "            break\n",
    "    except KeyError:\n",
    "        pass"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "23f7244a-1c9e-4f10-9197-6d04f31ed33e",
   "metadata": {
    "scrolled": true,
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 104 faculty\n",
      "\n",
      "Luca Busino, Ph.D https://www.med.upenn.edu/cbio/faculty/#personnel-description5 245\n",
      "Liling Wan, Ph.D. https://www.med.upenn.edu/cbio/faculty/ 245\n",
      "\n",
      "Daniel Kulp, Ph.D., https://wistar.org/our-scientists/daniel-kulp 118\n",
      "\n",
      "Nikolaos Sgourakis, Ph.D  https://www.research.chop.edu/mechanistic-molecular-immunology-lab 104\n",
      "\n",
      "Cesar de la Fuente, Ph.D https://www.seas.upenn.edu/directory/profile.php?ID=299 75\n",
      "\n",
      "Scott L. Diamond, Ph.D. http://www.seas.upenn.edu/directory/profile.php?ID=24 75\n",
      "\n",
      "Amish Patel, Ph.D. https://www.seas.upenn.edu/directory/profile.php?ID=182 75\n",
      "\n",
      "Ravi Radhakrishnan, Ph.D. http://www.seas.upenn.edu/directory/profile.php?ID=76 75\n",
      "Kai Wang, Ph.D. http://pathology.med.upenn.edu/department/people/1160/kai-wang 75\n",
      "\n",
      "Arjun Raj, Ph.D. http://www.seas.upenn.edu/be/dir/details/ArjunRajdetails.html 55\n",
      "\n",
      "Sydney Shaffer, MD, Ph.D. http://pathology.med.upenn.edu/department/people 55\n",
      "\n",
      "James Shorter, Ph.D. http://www.med.upenn.edu/apps/faculty/index.php/g20x000321/p8151447 51\n",
      "\n",
      "Lewis A. Chodosh, M.D., Ph.D. http://www.med.upenn.edu/apps/faculty/index.php/g275/p20564 46\n",
      "\n",
      "Ronen Marmorstein, Ph.D. http://www.med.upenn.edu/apps/faculty/index.php/g20000220/p20660 35\n",
      "\n",
      "David Cormode, Ph.D, https://www.med.upenn.edu/apps/faculty/index.php/g275/p8524204 34\n",
      "\n",
      "Bin Tian, Ph.D.  https://wistar.org/our-scientists/bin-tian 34\n",
      "\n",
      "Kim A. Sharp, Ph.D. http://www.med.upenn.edu/apps/faculty/index.php/g20x000321/p17328 32\n",
      "\n",
      "Maureen E. Murphy, Ph.D. https://www.wistar.org/our-science/scientists/maureen-murphy-phd 31\n",
      "\n",
      "Theresa Busch, Ph.D. http://www.med.upenn.edu/apps/faculty/index.php/g20x000321/p14028 28\n",
      "\n",
      "Roland L. Dunbrack, Jr. Ph.D. http://www.med.upenn.edu/apps/faculty/index.php/g20x000321/p12980 26\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(penn_faculty_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e97c750c-c19c-4822-93f9-534455ca24e9",
   "metadata": {},
   "source": [
    "# Harvard Faculty Analysis (Biophysics)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "9f824032-0993-4ea4-8c63-8dbff542e043",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "url =  f\"https://biophysics.fas.harvard.edu/people/people-terms/faculty\"\n",
    "harvard_faculty_dict = {}\n",
    "for i in range(0, 6):\n",
    "    response = requests.get(url)\n",
    "    soup = BeautifulSoup(response.text, \"html.parser\")\n",
    "    link_list = soup.findAll(\"div\", {\"class\":\"view-content\"})[0].findAll(\"a\")\n",
    "    for link in link_list:\n",
    "        if \"Read more\" in link.text:\n",
    "            name = link.text.split(\"about\")[-1]\n",
    "            link = link['href']\n",
    "            \n",
    "            resp = requests.get(link)\n",
    "            fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "            text = \"\"\n",
    "            for word in fac_soup:\n",
    "                text += word.text\n",
    "            harvard_faculty_dict[(name, link)] = text\n",
    "    url = f\"https://biophysics.fas.harvard.edu/people/people-terms/faculty?page={i+1}\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "e2d548da-53fc-4d7b-a06a-a25b07bc28f9",
   "metadata": {
    "scrolled": true,
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 85 faculty\n",
      " Peng Yin https://biophysics.fas.harvard.edu/people/peng-yin 116\n",
      " Jan Drugowitsch https://biophysics.fas.harvard.edu/people/jan-drugowitsch 115\n",
      " Eugene I. Shakhnovich https://biophysics.fas.harvard.edu/people/eugene-i-shakhnovich 80\n",
      " Adam E. Cohen https://biophysics.fas.harvard.edu/people/adam-cohen 50\n",
      " Conor L. Evans https://biophysics.fas.harvard.edu/people/conor-l-evans 44\n",
      " Andrew C. Kruse https://biophysics.fas.harvard.edu/people/andrew-c-kruse 44\n",
      " Chris Sander https://biophysics.fas.harvard.edu/people/chris-sander 41\n",
      " Shamil R. Sunyaev https://biophysics.fas.harvard.edu/people/shamil-r-sunyaev 41\n",
      " Wei-Chung Allen Lee https://biophysics.fas.harvard.edu/people/wei-chung-allen-lee 40\n",
      " Gary Ruvkun https://biophysics.fas.harvard.edu/people/gary-ruvkun 39\n",
      " Maofu Liao https://biophysics.fas.harvard.edu/people/maofu-liao 38\n",
      " Pamela  A.  Silver https://biophysics.fas.harvard.edu/people/pamela-silver 38\n",
      " James J. Chou https://biophysics.fas.harvard.edu/people/james-j-chou 36\n",
      " Min Dong, Ph.D. https://biophysics.fas.harvard.edu/people/min-dong-phd 35\n",
      " Stephen C. Harrison https://biophysics.fas.harvard.edu/people/stephen-c-harrison 35\n",
      " Gabriel Kreiman https://biophysics.fas.harvard.edu/people/gabriel-kreiman 33\n",
      " Galit  Lahav, Ph.D. https://biophysics.fas.harvard.edu/people/galit-lahav 33\n",
      " William M. Shih https://biophysics.fas.harvard.edu/people/william-m-shih 33\n",
      " Debora S. Marks https://biophysics.fas.harvard.edu/people/debora-s-marks 31\n",
      " Leonid A. Mirny, Ph.D. https://biophysics.fas.harvard.edu/people/leonid-mirny 31\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(harvard_faculty_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f82a179b-3ce0-4272-8286-d20ab8c42db7",
   "metadata": {},
   "source": [
    "# Columbia Faculty Analysis (Doesn't have anything)\n",
    "### Molecular Pharmacology and Therapeutics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "id": "b01e062b-dca5-401f-8bbc-0b8462919d27",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = \"https://www.gsas.cuimc.columbia.edu/faculty-6\"\n",
    "response = requests.get(url)\n",
    "soup = BeautifulSoup(response.text, \"html.parser\")\n",
    "fac_list = soup.findAll(\"div\", {\"class\", \"field field-name-body field-type-text-with-summary field-label-hidden\"})[0]\n",
    "columbia_faculty_dict = {}\n",
    "for i in fac_list.findAll(\"a\"):\n",
    "    fac_url = \"https://www.gsas.cuimc.columbia.edu\"\n",
    "    if \"mailto\" in i['href']:\n",
    "        continue\n",
    "    if \"https://\" in i['href']:\n",
    "        fac_url = i['href']\n",
    "    else:\n",
    "        fac_url = fac_url + i['href']\n",
    "        \n",
    "    resp = requests.get(fac_url)\n",
    "    fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "    text = \"\"\n",
    "    for word in fac_soup:\n",
    "        text += word.text\n",
    "    columbia_faculty_dict[(i.text, fac_url)] = text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "id": "bfe482e2-fc0e-4649-828b-0a31782ccfdb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 38 faculty\n",
      "Robert Kass, PhD https://www.gsas.cuimc.columbia.edu/profile/robert-s-kass-phd 70\n",
      "Kenneth Olive, PhD https://www.gsas.cuimc.columbia.edu/profile/kenneth-p-olive-phd 67\n",
      "Christian Schindler, MD, PhD https://www.gsas.cuimc.columbia.edu/profile/christian-w-schindler-md 38\n",
      "Henry Colecraft, PhD https://www.gsas.cuimc.columbia.edu/profile/henry-colecraft-phd 36\n",
      "Joseph Graziano, PhD https://www.gsas.cuimc.columbia.edu/profile/joseph-h-graziano-phd 26\n",
      "Andrea Califano, PhD https://www.gsas.cuimc.columbia.edu/profile/andrea-califano-dr 25\n",
      "Wayne Hendrickson, PhD https://www.gsas.cuimc.columbia.edu/profile/wayne-hendrickson-phd 25\n",
      "Jonathan Javitch, MD, PhD https://www.gsas.cuimc.columbia.edu/profile/jonathan-javitch-md 19\n",
      "David Sulzer, PhD https://www.gsas.cuimc.columbia.edu/profile/david-sulzer-phd 19\n",
      "Virginia Cornish, PhD https://www.gsas.cuimc.columbia.edu/profile/virginia-w-cornish-phd 17\n",
      "Dritan Agalliu, PhD https://www.gsas.cuimc.columbia.edu/profile/dritan-agalliu-phd 14\n",
      "Penelope Boyden, PhD https://www.gsas.cuimc.columbia.edu/profile/penelope-boyden-phd 11\n",
      "Gordana Vunjak-Novakovic, PhD https://www.gsas.cuimc.columbia.edu/profile/gordana-vunjak-novakovic-phd 11\n",
      "Ronald Liem, PhD https://www.gsas.cuimc.columbia.edu/profile/ronald-k-liem-phd 8\n",
      "Li Qiang, PhD https://www.pathology.columbia.edu/profile/li-qiang-phd 8\n",
      "Susan Steinberg, MD https://www.gsas.cuimc.columbia.edu/profile/susan-f-steinberg-md 8\n",
      "Richard Baer, PhD https://www.gsas.cuimc.columbia.edu/profile/richard-j-baer-phd 7\n",
      "Wendy Chung, MD, PhD https://www.gsas.cuimc.columbia.edu/profile/wendy-k-chung-md 7\n",
      "Tae-Wan Kim, PhD https://www.gsas.cuimc.columbia.edu/profile/tae-wan-kim-phd 7\n",
      "Joachim Scholz, MD https://www.gsas.cuimc.columbia.edu/profile/joachim-scholz-md 7\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(columbia_faculty_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "56b7815d-ebd4-4504-a5b7-8fb05e186689",
   "metadata": {},
   "source": [
    "### Physiology and Cellular Biophysics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "id": "823fabd6-00c8-450a-b106-1150c216bf5d",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = \"https://www.gsas.cuimc.columbia.edu/faculty-7\"\n",
    "response = requests.get(url)\n",
    "soup = BeautifulSoup(response.text, \"html.parser\")\n",
    "fac_list = soup.findAll(\"div\", {\"class\", \"field field-name-body field-type-text-with-summary field-label-hidden\"})[0]\n",
    "biophysics_columbia_faculty_dict = {}\n",
    "for i in fac_list.findAll(\"a\"):\n",
    "    fac_url = \"https://www.gsas.cuimc.columbia.edu\"\n",
    "    if \"mailto\" in i['href']:\n",
    "        continue\n",
    "    if \"https://\" in i['href']:\n",
    "        fac_url = i['href']\n",
    "    else:\n",
    "        fac_url = fac_url + i['href']\n",
    "        \n",
    "    try:\n",
    "        resp = requests.get(fac_url)\n",
    "        fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "        text = \"\"\n",
    "        for word in fac_soup:\n",
    "            text += word.text\n",
    "        biophysics_columbia_faculty_dict[(i.text, fac_url)] = text\n",
    "    except:\n",
    "        pass"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "id": "87f3e788-6b08-414d-8037-896e2d810703",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 55 faculty\n",
      "Nicholas Tatonetti, PhD https://www.gsas.cuimc.columbia.edu/profile/nicholas-p-tatonetti-phd 141\n",
      "Robert Kass, PhD https://www.gsas.cuimc.columbia.edu/profile/robert-s-kass-phd 70\n",
      "Scott Banta, PhD https://cheme.columbia.edu/faculty/scott-banta 68\n",
      "Larry Abbott, PhD https://www.gsas.cuimc.columbia.edu/profile/larry-abbott-phd 45\n",
      "Anne Moscona, MD https://microbiology.columbia.edu/faculty-anne-moscona 40\n",
      "Henry Colecraft, PhD https://www.gsas.cuimc.columbia.edu/profile/henry-colecraft-phd 36\n",
      "Ning Qian, PhD https://www.gsas.cuimc.columbia.edu/profile/ning-qian-phd 36\n",
      "Allie Obermeyer, PhD https://www.engineering.columbia.edu/faculty/allie-obermeyer 31\n",
      "John Hunt, PhD https://biology.columbia.edu/people/hunt 30\n",
      "Wayne Hendrickson, PhD https://www.gsas.cuimc.columbia.edu/profile/wayne-hendrickson-phd 25\n",
      "Jonathan Javitch, MD, PhD https://www.gsas.cuimc.columbia.edu/profile/jonathan-javitch-md 19\n",
      "Matthias Quick, PhD https://www.columbiapsychiatry.org/profile/matthias-quick-phd 18\n",
      "anu Ben-Johny, PhD https://www.ps.columbia.edu/profile/manu-b-johny-phd 12\n",
      "Lawrence Shapiro, PhD https://www.gsas.cuimc.columbia.edu/profile/lawrence-s-shapiro-phd 12\n",
      "ingting Yang, PhD https://www.columbiaeye.org/profile/tyang?profile=researcher 12\n",
      "Richard Vallee, PhD https://www.gsas.cuimc.columbia.edu/profile/richard-vallee-phd 11\n",
      "Gordana Vunjak-Novakovic, PhD https://www.gsas.cuimc.columbia.edu/profile/gordana-vunjak-novakovic-phd 11\n",
      "M https://www.gsas.cuimc.columbia.edu/profile/attila-losonczy-md 10\n",
      "X. Shawn Liu https://www.physiology.columbia.edu/Liu.html 10\n",
      "Attila Losonczy, MD, PhD https://www.gsas.cuimc.columbia.edu/profile/attila-losonczy-md 10\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(biophysics_columbia_faculty_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e9f183a-1fc6-4a64-8408-4382d6ef1a49",
   "metadata": {},
   "source": [
    "# Northwestern Faculty Analysis (Not working as well because they inject the faculty content with javascript so its not in the initial HTML)\n",
    "### Biophysics Department"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "id": "c3ca453b-1e69-4fe0-a3d4-c48e93bfec17",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = \"https://biophysics.northwestern.edu/people/preceptors/index.html\"\n",
    "response = requests.get(url)\n",
    "soup = BeautifulSoup(response.text, \"html.parser\")\n",
    "faculty_list = soup.findAll(\"div\", {\"class\":\"people-wrap\"})\n",
    "northwestern_fac_dict = {}\n",
    "for i in faculty_list:\n",
    "    name = i.findAll(\"a\")[1]\n",
    "    link = name['href']\n",
    "    name = name.text\n",
    "\n",
    "    resp = requests.get(link)\n",
    "    fac_soup = BeautifulSoup(resp.text, \"html.parser\")\n",
    "    \n",
    "    text = \"\"\n",
    "    for word in fac_soup:\n",
    "        text += word.text\n",
    "        \n",
    "    northwestern_fac_dict[(name, link)] = text\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "id": "59c0a8a8-19a3-4abf-b25a-5fceba4c1da2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found and scored 29 faculty\n",
      "Joshua Leonard http://www.mccormick.northwestern.edu/research-faculty/directory/profiles/leonard-josh.html 42\n",
      "Danielle Tullman-Ercek http://www.mccormick.northwestern.edu/research-faculty/directory/profiles/tullman-ercek-danielle.html 22\n",
      "Neha Kamat http://www.mccormick.northwestern.edu/research-faculty/directory/profiles/kamat-neha.html 12\n",
      "Brian Hoffman https://www.chemistry.northwestern.edu/people/core-faculty/profiles/brian-hoffman.html 11\n",
      "Adilson Motter http://www.physics.northwestern.edu/people/faculty/core-faculty/adilson-motter.html 10\n",
      "Julius Lucks http://www.mccormick.northwestern.edu/research-faculty/directory/profiles/lucks-julius.html 8\n",
      "Heather Pinkett http://www.molbiosci.northwestern.edu/people/core-faculty/heather-pinkett.html 8\n",
      "Curt Horvath http://www.molbiosci.northwestern.edu/people/core-faculty/curt-horvath.html 7\n",
      "Michael Jewett http://www.mccormick.northwestern.edu/research-faculty/directory/profiles/jewett-mike.html 7\n",
      "Richard Morimoto http://www.molbiosci.northwestern.edu/people/core-faculty/richard-morimoto.html 6\n",
      "Neil Kelleher https://www.chemistry.northwestern.edu/people/core-faculty/profiles/neil-kelleher.html 5\n",
      "Keith Tyo http://www.mccormick.northwestern.edu/research-faculty/directory/profiles/tyo-keith.html 5\n",
      "John Marko http://www.molbiosci.northwestern.edu/people/core-faculty/john-marko.html 4\n",
      "Ishwar Radhakrishnan http://www.molbiosci.northwestern.edu/people/core-faculty/ishwar-radhakrishnan.html 4\n",
      "Reza Vafabakhsh http://www.molbiosci.northwestern.edu/people/core-faculty/reza-vafbakhsh.html 4\n",
      "Robert Lamb http://www.molbiosci.northwestern.edu/people/core-faculty/robert-lamb.html 3\n",
      "Jason Brickner http://www.molbiosci.northwestern.edu/people/core-faculty/jason-brickner.html 2\n",
      "Lin Chen http://www.chemistry.northwestern.edu/people/core-faculty/profiles/lin-chen.html 2\n",
      "Amy Rosenzweig http://www.molbiosci.northwestern.edu/people/core-faculty/amy-rosenzweig.html 2\n",
      "Yuan He http://www.molbiosci.northwestern.edu/people/core-faculty/Yuan-He.html 1\n",
      "Laura Lackner  http://www.molbiosci.northwestern.edu/people/core-faculty/laura-lackner.html 1\n",
      "Alfonso MondragÃ³n http://www.molbiosci.northwestern.edu/people/core-faculty/alfonso-mondragon.html 1\n",
      "Murali Prakriya http://www.pharm.northwestern.edu/faculty/profile.html?xid=16139 1\n",
      "Karla Satchell https://www.feinberg.northwestern.edu/faculty-profiles/az/profile.html?xid=10285 1\n",
      "Paul DeCaen http://www.pharm.northwestern.edu/faculty/profile.html?xid=34221 0\n",
      "Daniel Foltz https://www.feinberg.northwestern.edu/sites/biochem/faculty/profile.html?xid=33062 0\n",
      "Alfred George http://www.feinberg.northwestern.edu/faculty-profiles/az/profile.html?xid=29140 0\n",
      "Gabriel Rocklin https://www.feinberg.northwestern.edu/faculty-profiles/az/profile.html?xid=43966 0\n",
      "Sadie Wignall http://www.molbiosci.northwestern.edu/people/core-faculty/sadie-wignall.html 0\n"
     ]
    }
   ],
   "source": [
    "print_sorted_faculty(northwestern_fac_dict, False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5a4adb66-1cf1-4878-befe-2c605e4644bc",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
